<!DOCTYPE html>
<html xmlns:og="http://ogp.me/ns#" xmlns:fb="https://www.facebook.com/2008/fbml" class=" js flexbox flexboxlegacy canvas canvastext webgl no-touch geolocation postmessage no-websqldatabase indexeddb hashchange history draganddrop websockets rgba hsla multiplebgs backgroundsize borderimage borderradius boxshadow textshadow opacity cssanimations csscolumns cssgradients no-cssreflections csstransforms csstransforms3d csstransitions fontface generatedcontent video audio localstorage sessionstorage webworkers applicationcache svg inlinesvg smil svgclippaths" lang="en"><head>
<meta http-equiv="content-type" content="text/html; charset=UTF-8"><meta class="foundation-mq-small"><meta class="foundation-mq-small-only"><meta class="foundation-mq-medium"><meta class="foundation-mq-medium-only"><meta class="foundation-mq-large"><meta class="foundation-mq-large-only"><meta class="foundation-mq-xlarge"><meta class="foundation-mq-xlarge-only"><meta class="foundation-mq-xxlarge"><meta class="foundation-data-attribute-namespace">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
<meta content="mdpi" name="sso-service">
<meta content="width=device-width, initial-scale=1.0" name="viewport">
<title>Remote Sensing  | Free Full-Text | Modelling Distributions of Rove Beetles in Mountainous Areas Using Remote Sensing Data | HTML</title><link rel="stylesheet" href="font-awesome.css">
<link rel="stylesheet" href="jquery.css">
<link rel="stylesheet" href="chosen.css">
<link rel="stylesheet" href="main2.css">
<link rel="mask-icon" href="https://www.mdpi.com/img/mask-icon-128.svg?c1c7eca266cd7013" color="#4f5671">
<link rel="apple-touch-icon" sizes="180x180" href="https://www.mdpi.com/icon/apple-touch-icon-180x180.png">
<link rel="apple-touch-icon" sizes="152x152" href="https://www.mdpi.com/icon/apple-touch-icon-152x152.png">
<link rel="apple-touch-icon" sizes="144x144" href="https://www.mdpi.com/icon/apple-touch-icon-144x144.png">
<link rel="apple-touch-icon" sizes="120x120" href="https://www.mdpi.com/icon/apple-touch-icon-120x120.png">
<link rel="apple-touch-icon" sizes="114x114" href="https://www.mdpi.com/icon/apple-touch-icon-114x114.png">
<link rel="apple-touch-icon" sizes="76x76" href="https://www.mdpi.com/icon/apple-touch-icon-76x76.png">
<link rel="apple-touch-icon" sizes="72x72" href="https://www.mdpi.com/icon/apple-touch-icon-72x72.png">
<link rel="apple-touch-icon" sizes="57x57" href="https://www.mdpi.com/icon/apple-touch-icon-57x57.png">
<link rel="apple-touch-icon" href="https://www.mdpi.com/icon/apple-touch-icon-57x57.png">
<link rel="apple-touch-icon-precomposed" href="https://www.mdpi.com/icon/apple-touch-icon-57x57.png">
<link rel="manifest" href="https://www.mdpi.com/manifest.json">
<meta name="theme-color" content="#ffffff">
<meta name="application-name" content="&nbsp;">
<link rel="apple-touch-startup-image" href="https://www.mdpi.com/img/journals/remotesensing-logo-sq.png?933394d88c7dba88">
<link rel="apple-touch-icon" href="https://www.mdpi.com/img/journals/remotesensing-logo-sq.png?933394d88c7dba88">
<meta name="msapplication-TileImage" content="/img/journals/remotesensing-logo-sq.png?933394d88c7dba88">
<style>

h2, #abstract .related_suggestion_title {
    color: rgba(56,112,46,0.75);
}

.batch_articles a {
    color: #000;
}

    a, .batch_articles .authors a, a:focus, a:hover, a:active, .batch_articles a:focus, .batch_articles a:hover, li.side-menu-li a {
        color: rgba(56,112,46,0.75);
    }

    span.label a {
        color: #fff;
    }

    #main-content a.title-link:hover,
    #main-content a.title-link:focus,
    #main-content div.generic-item a.title-link:hover,
    #main-content div.generic-item a.title-link:focus {
        color: rgba(56,112,46,0.75);
    }

    #main-content #middle-column .generic-item.article-item a.title-link:hover,
    #main-content #middle-column .generic-item.article-item a.title-link:focus {
        color: rgba(56,112,46,0.75);
    }

    .art-authors a.toEncode {
        color: #333;
        font-weight: 700;
    }

    #main-content #middle-column ul li::before {
        color: rgba(56,112,46,0.75);
    }

    .accordion-navigation.active a.accordion__title,
    .accordion-navigation.active a.accordion__title::after {
        color: rgba(56,112,46,0.75) !important;
    }

    .accordion-navigation li:hover::before,
    .accordion-navigation li:hover a,
    .accordion-navigation li:focus a {
        color: rgba(56,112,46,0.75) !important;
    }

    .relative-size-container .relative-size-image .relative-size {
        background-color: rgba(56,112,46,0.75);
    }

    .middle-column__help__fixed a:hover i,
        color: rgba(56,112,46,0.75) !important;
    }

    input[type="checkbox"]:checked:after {
        color: rgba(56,112,46,0.75) !important;
    }

    input[type="checkbox"]:not(:disabled):hover:before {
        border-color: rgba(56,112,46,0.75) !important;
    }

    #main-content .bolded-text {
        color: rgba(56,112,46,0.75);
    }


#main-content .hypothesis-count-container {
    background-color: rgba(56,112,46,0.75) !important;
}

#main-content .hypothesis-count-container:before {
    border-bottom-color: rgba(56,112,46,0.75) !important;
}

#main-content .trendmd-widget-cookie-notification__info a {
    color: rgba(56,112,46,0.75) !important;
}

.full-size-menu ul li.menu-item .dropdown-wrapper {
    background-color: #38702E !important;
}

.full-size-menu ul li.menu-item > a.open::after {
    border-bottom: 10px solid #38702E;
}

#title-story .title-story-orbit .orbit-caption {
    #background: url('/img/design/000000_background.png') !important;
    background: url('/img/design/ffffff_background.png') !important;
    color: rgb(51, 51, 51) !important;
}

#main-content .content__container__orbit {
    background-color: #000 !important;
}

.html-article-menu .row span {
    color: rgba(56,112,46,0.75);
}

.html-article-menu .row span.active {
    }

.side-menu-li.active::before,
.side-menu-li.active a {
    color: rgba(56,112,46,0.75) !important;
}

.side-menu-ul li.active a, .side-menu-ul li.active, .side-menu-ul li.active::before {
    color: rgba(56,112,46,0.75) !important;
}

.side-menu-ul li.active a {
    font-weight: 700 !important;
}

.result-selected, .active-result.highlighted, .active-result:hover,
.result-selected, .active-result.highlighted, .active-result:focus {
    background-color: rgba(56,112,46,0.75) !important;
    color: #fff;
}

.search-container.search-container__default-scheme {
    background-color: #38702E;
}

nav.tab-bar .open-small-search.active:after {
    border-bottom-color: #38702E;
}

.search-container.search-container__default-scheme .custom-accordion-for-small-screen-link::after {
    color: #fff;
}

@media only screen and (max-width: 50em) {
    #main-content .content__container.journal-info {
        color: #fff;
        background-color: rgba(56,112,46,0.75);
    }

    #main-content .content__container.journal-info a {
        color: #fff;
    }
} 

.button.button--color {
    color: rgba(56,112,46,0.75);
    border-color: rgba(56,112,46,0.75);
}

.button.button--color:hover,
.button.button--color:focus {
    color: #fff;
    border-color: rgba(56,112,46,0.75);
    background-color: rgba(56,112,46,0.75);
}

.button.button--color-inversed {
    background-color: rgba(56,112,46,0.75);
    border-color: #fff;
    color: #fff;
}

.button.button--color-inversed:hover,
.button.button--color-inversed:focus {
    background-color: rgba(56,112,46,0.75);
    border-color: #fff;
    color: #fff;
}

.button.button--color path {
    fill: rgba(56,112,46,0.75);
}

.button.button--color:hover path {
    fill: #fff;
}

#main-content #search-refinements .ui-slider-horizontal .ui-slider-range {
    background-color: rgba(56,112,46,0.75);
}

.breadcrumb__element:last-of-type a {
    color: rgba(56,112,46,0.75);
}

    
#main-header {
    background-color: rgba(56,112,46,0.75);
}

#full-size-menu .top-bar, #full-size-menu li.menu-item span.user-email {
    background-color:   rgba(56,112,46,0.75)
}

.top-bar-section li:not(.has-form) a:not(.button) {
    background-color:   rgba(56,112,46,0.75)
}

#full-size-menu li.menu-item .dropdown-wrapper li a:hover {
    color:   rgba(56,112,46,0.75)
}

#full-size-menu li.menu-item a:hover, #full-size-menu li.menu.item a:focus, nav.tab-bar a:hover {
    }
#full-size-menu li.menu.item a:active, #full-size-menu li.menu.item a.active {
    background-color: #95BF8F;
}

#full-size-menu li.menu-item a.open-mega-menu.active, #full-size-menu li.menu-item div.mega-menu, a.open-mega-menu.active {
    background-color: #95BF8F;
    color: #000;
}

#full-size-menu li.menu-item div.mega-menu li, #full-size-menu li.menu-item div.mega-menu a {
    background-color: #95BF8F;
    color: #000;
    border-color: #9a9a9a;
}

div.type-section h1 {
    padding-left: 15px !important;
    background-color: rgba(56,112,46,0.75);
    color: #fff;
    font-size: 20px;
    line-height: 26px;
    font-weight: 300;
}

div.type-section h3 {
    margin-left: 15px;
    margin-bottom: 0px;
    font-weight: 300;
}

.journal-tabs .tab-title.active a {
    background-color: rgba(56,112,46,0.75);
    border-color: rgba(56,112,46,0.75);
    color: #fff;
}

</style>
<link rel="stylesheet" href="jquery-ui-1.css">
<link rel="stylesheet" href="magnific-popup.css">
<link rel="stylesheet" href="article-html.css">
<style>
                                </style>
<meta name="title" content="Modelling Distributions of Rove Beetles in Mountainous Areas Using Remote Sensing Data">
<meta name="description" content="Mountain ecosystems are biodiversity hotspots that are increasingly threatened by climate and land use/land cover changes. Long-term biodiversity monitoring programs provide unique insights into resulting adverse impacts on plant and animal species distribution. Species distribution models (SDMs) in combination with satellite remote sensing (SRS) data offer the opportunity to analyze shifts of species distributions in response to these changes in a spatially explicit way. Here, we predicted the presence probability of three different rove beetles in a mountainous protected area (Gran Paradiso National Park, GPNP) using environmental variables derived from Landsat and Aster Global Digital Elevation Model data and an ensemble modelling approach based on five different model algorithms (maximum entropy, random forest, generalized boosting models, generalized additive models, and generalized linear models). The objectives of the study were (1) to evaluate the potential of SRS data for predicting the presence of species dependent on local-scale environmental parameters at two different time periods, (2) to analyze shifts in species distributions between the years, and (3) to identify the most important species-specific SRS predictor variables. All ensemble models showed area under curve (AUC) of the receiver operating characteristics values above 0.7 and true skills statistics (TSS) values above 0.4, highlighting the great potential of SRS data. While only a small proportion of the total area was predicted as highly suitable for each species, our results suggest an increase of suitable habitat over time for the species Platydracus stercorarius and Ocypus ophthalmicus, and an opposite trend for Dinothenarus fossor. Vegetation cover was the most important predictor variable in the majority of the SDMs across all three study species. To better account for intra- and inter-annual variability of population dynamics as well as environmental conditions, a continuation of the monitoring program in GPNP as well as the employment of SRS with higher spatial and temporal resolution is recommended.">
<link rel="image_src" href="remotesensing-logo.png">
<meta name="dc.title" content="Modelling Distributions of Rove Beetles in Mountainous Areas Using Remote Sensing Data">
<meta name="dc.creator" content="Andreas Dittrich">
<meta name="dc.creator" content="Stephanie Roilo">
<meta name="dc.creator" content="Ruth Sonnenschein">
<meta name="dc.creator" content="Cristiana Cerrato">
<meta name="dc.creator" content="Michael Ewald">
<meta name="dc.creator" content="Ramona Viterbi">
<meta name="dc.creator" content="Anna  F. Cord">
<meta name="dc.type" content="Article">
<meta name="dc.source" content="Remote Sensing 2020, Vol. 12, Page 80">
<meta name="dc.date" content="2019-12-24">
<meta name="dc.identifier" content="10.3390/rs12010080">
<meta name="dc.publisher" content="Multidisciplinary Digital Publishing Institute">
<meta name="dc.rights" content="http://creativecommons.org/licenses/by/3.0/">
<meta name="dc.format" content="application/pdf">
<meta name="dc.language" content="en">
<meta name="dc.description" content="Mountain ecosystems are biodiversity hotspots that are increasingly threatened by climate and land use/land cover changes. Long-term biodiversity monitoring programs provide unique insights into resulting adverse impacts on plant and animal species distribution. Species distribution models (SDMs) in combination with satellite remote sensing (SRS) data offer the opportunity to analyze shifts of species distributions in response to these changes in a spatially explicit way. Here, we predicted the presence probability of three different rove beetles in a mountainous protected area (Gran Paradiso National Park, GPNP) using environmental variables derived from Landsat and Aster Global Digital Elevation Model data and an ensemble modelling approach based on five different model algorithms (maximum entropy, random forest, generalized boosting models, generalized additive models, and generalized linear models). The objectives of the study were (1) to evaluate the potential of SRS data for predicting the presence of species dependent on local-scale environmental parameters at two different time periods, (2) to analyze shifts in species distributions between the years, and (3) to identify the most important species-specific SRS predictor variables. All ensemble models showed area under curve (AUC) of the receiver operating characteristics values above 0.7 and true skills statistics (TSS) values above 0.4, highlighting the great potential of SRS data. While only a small proportion of the total area was predicted as highly suitable for each species, our results suggest an increase of suitable habitat over time for the species Platydracus stercorarius and Ocypus ophthalmicus, and an opposite trend for Dinothenarus fossor. Vegetation cover was the most important predictor variable in the majority of the SDMs across all three study species. To better account for intra- and inter-annual variability of population dynamics as well as environmental conditions, a continuation of the monitoring program in GPNP as well as the employment of SRS with higher spatial and temporal resolution is recommended.">
<meta name="dc.subject" content="temporal analysis">
<meta name="dc.subject" content="species distribution model">
<meta name="dc.subject" content="Landsat">
<meta name="dc.subject" content="ASTER GDEM">
<meta name="dc.subject" content="ensemble modelling">
<meta name="dc.subject" content="protected area">
<meta name="dc.subject" content="Italian Alps">
<meta name="dc.subject" content="biodiversity monitoring">
<meta name="prism.publicationName" content="Remote Sensing">
<meta name="prism.publicationDate" content="2019-12-24">
<meta name="prism.volume" content="12">
<meta name="prism.number" content="1">
<meta name="prism.section" content="Article">
<meta name="prism.startingPage" content="80">
<meta name="citation_journal_title" content="Remote Sensing">
<meta name="citation_publisher" content="Multidisciplinary Digital Publishing Institute">
<meta name="citation_title" content="Modelling Distributions of Rove Beetles in Mountainous Areas Using Remote Sensing Data">
<meta name="citation_publication_date" content="2020/1">
<meta name="citation_online_date" content="2019/12/24">
<meta name="citation_volume" content="12">
<meta name="citation_issue" content="1">
<meta name="citation_firstpage" content="80">
<meta name="citation_author" content="Dittrich, Andreas">
<meta name="citation_author" content="Roilo, Stephanie">
<meta name="citation_author" content="Sonnenschein, Ruth">
<meta name="citation_author" content="Cerrato, Cristiana">
<meta name="citation_author" content="Ewald, Michael">
<meta name="citation_author" content="Viterbi, Ramona">
<meta name="citation_author" content="Cord, Anna  F.">
<meta name="citation_doi" content="10.3390/rs12010080">
<meta name="citation_id" content="mdpi-rs12010080">
<meta name="citation_abstract_html_url" content="https://www.mdpi.com/2072-4292/12/1/80">
<meta name="citation_pdf_url" content="https://www.mdpi.com/2072-4292/12/1/80/pdf">
<link rel="alternate" type="application/pdf" title="PDF Full-Text" href="https://www.mdpi.com/2072-4292/12/1/80/pdf">
<meta name="fulltext_pdf" content="https://www.mdpi.com/2072-4292/12/1/80/pdf">
<meta name="citation_fulltext_html_url" content="https://www.mdpi.com/2072-4292/12/1/80/htm">
<link rel="alternate" type="text/html" title="HTML Full-Text" href="https://www.mdpi.com/2072-4292/12/1/80/htm">
<meta name="fulltext_html" content="https://www.mdpi.com/2072-4292/12/1/80/htm">
<link rel="alternate" type="text/xml" title="XML Full-Text" href="https://www.mdpi.com/2072-4292/12/1/80/xml">
<meta name="fulltext_xml" content="https://www.mdpi.com/2072-4292/12/1/80/xml">
<meta name="twitter:card" content="summary">
<meta name="twitter:site" content="@MDPIOpenAccess">
<meta name="twitter:image" content="https://www.mdpi.com/img/journals/remotesensing-logo-sq.png?933394d88c7dba88">
<meta property="fb:app_id" content="131189377574">
<meta property="og:site_name" content="MDPI">
<meta property="og:type" content="article">
<meta property="og:url" content="https://www.mdpi.com/2072-4292/12/1/80">
<meta property="og:title" content="Modelling Distributions of Rove Beetles in Mountainous Areas Using Remote Sensing Data">
<meta property="og:description" content="Mountain ecosystems are biodiversity hotspots that are increasingly threatened by climate and land use/land cover changes. Long-term biodiversity monitoring programs provide unique insights into resulting adverse impacts on plant and animal species distribution. Species distribution models (SDMs) in combination with satellite remote sensing (SRS) data offer the opportunity to analyze shifts of species distributions in response to these changes in a spatially explicit way. Here, we predicted the presence probability of three different rove beetles in a mountainous protected area (Gran Paradiso National Park, GPNP) using environmental variables derived from Landsat and Aster Global Digital Elevation Model data and an ensemble modelling approach based on five different model algorithms (maximum entropy, random forest, generalized boosting models, generalized additive models, and generalized linear models). The objectives of the study were (1) to evaluate the potential of SRS data for predicting the presence of species dependent on local-scale environmental parameters at two different time periods, (2) to analyze shifts in species distributions between the years, and (3) to identify the most important species-specific SRS predictor variables. All ensemble models showed area under curve (AUC) of the receiver operating characteristics values above 0.7 and true skills statistics (TSS) values above 0.4, highlighting the great potential of SRS data. While only a small proportion of the total area was predicted as highly suitable for each species, our results suggest an increase of suitable habitat over time for the species Platydracus stercorarius and Ocypus ophthalmicus, and an opposite trend for Dinothenarus fossor. Vegetation cover was the most important predictor variable in the majority of the SDMs across all three study species. To better account for intra- and inter-annual variability of population dynamics as well as environmental conditions, a continuation of the monitoring program in GPNP as well as the employment of SRS with higher spatial and temporal resolution is recommended.">
<meta property="og:image" content="https://www.mdpi.com/remotesensing/remotesensing-12-00080/article_deploy/html/images/remotesensing-12-00080-ag-550.jpg">
<link rel="alternate" type="application/rss+xml" title="MDPI Publishing - Latest articles" href="https://www.mdpi.com/rss">
<script src="https://connect.facebook.net/signals/config/1073306569680586?v=2.9.18&amp;r=stable" async=""></script><script async="" src="fbevents.js"></script><script type="text/javascript" async="" src="conversion_async.js"></script><script type="text/javascript" async="" src="insight.js"></script><script type="text/javascript" async="" src="analytics.js"></script><script async="" src="gtm.js"></script><script async="" src="analytics.js"></script><script type="text/x-mathjax-config;executed=true">
    MathJax.Hub.processSectionDelay = 0;
      MathJax.Hub.Config({
        menuSettings: {
          CHTMLpreview: false
        },
        "CHTML-preview":{
          disabled: true
        },
        "HTML-CSS": {
          availableFonts: ["TeX"],
          preferredFonts: "TeX",
          webFont:"TeX",
          imageFont:"TeX",
          undefinedFamily:"'Arial Unicode MS',serif"
        },
        "TeX": {
          extensions: ["noErrors.js"],
          noErrors: {
            inlineDelimiters: ["",""],
            multiLine: true,
            style: {
              "font-size":   "90%",
              "text-align":  "left",
              "color":       "black",
              "padding":     "1px 3px",
              "border":      "1px solid"
            }
          }
        }
      });
    </script>
<script type="text/javascript" src="MathJax.js"></script>
<meta name="google-site-verification" content="PxTlsg7z2S00aHroktQd57fxygEjMiNHydKn3txhvwY">
<!--[if lt IE 9]>
            <script>var browserIe8 = true;</script>
            <link rel="stylesheet" href="https://www.mdpi.com/assets/css/ie8foundationfix.css?8ad845af75faa4fd">
            <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
            <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.2/html5shiv.js"></script>
            <script src="//s3.amazonaws.com/nwapi/nwmatcher/nwmatcher-1.2.5-min.js"></script>
            <script src="//html5base.googlecode.com/svn-history/r38/trunk/js/selectivizr-1.0.3b.js"></script>
            <script src="//cdnjs.cloudflare.com/ajax/libs/respond.js/1.1.0/respond.min.js"></script>
            <script src="https://www.mdpi.com/assets/js/ie8/ie8patch.js?9e1d3c689a0471df"></script>
            <script src="https://www.mdpi.com/assets/js/ie8/rem.min.js?94b62787dcd6d2f2"></script>            
                                                        <![endif]-->
<script type="text/javascript">
            (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
            (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
            m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
            })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

            ga('create', 'UA-5824718-2', 'auto');
            ga('require', 'displayfeatures');
            ga('send', 'pageview');
        </script>
<script type="text/javascript">
                (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
                new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
                j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
                'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
                })(window,document,'script','dataLayer','GTM-WT3XWXZ');
            </script>
<link rel="preload" as="script" href="https://www.mdpi.com/assets/js/xmltohtml/articles.js?cf8ded7b859890d1"><link rel="preload" as="script" href="https://www.mdpi.com/assets/js/jquery-ui-1.10.4.custom.min.js?9fb4d59ff745b497"><link rel="preload" as="script" href="https://www.mdpi.com/assets/js/xmltohtml/underscore.js?f893e294cde60c24"><link rel="preload" as="script" href="https://www.mdpi.com/assets/js/xmltohtml/magnific-popup.js?4a09c18460afb26c"><link rel="preload" as="script" href="https://www.mdpi.com/assets/js/xmltohtml/jquery-scrollspy.js?09cbaec0dbb35a67"><link rel="preload" as="script" href="https://www.mdpi.com/assets/js/xmltohtml/storage.js?e9b262d3a3476d25"><link rel="preload" as="script" href="https://www.mdpi.com/assets/js/xmltohtml/affix.js?ac4ea55275297c15"><link rel="preload" as="script" href="https://www.mdpi.com/assets/js/jquery-ui-1.12.0.min.js?4c2c52cc11e5a28d"><link rel="preload" as="script" href="https://www.mdpi.com/assets/js/clipboard.min.js?3f3688138a1b9fc4"><link rel="preload" as="script" href="https://www.mdpi.com/assets/js/xmltohtml/affix.js?ac4ea55275297c15"><link rel="preload" as="script" href="https://www.mdpi.com/assets/js/ifvisible.min.js?c621d19ecb761212"><link rel="preload" as="script" href="https://www.mdpi.com/assets/js/mdpi.js?c5e55b6509d6aab3"><link rel="preload" as="script" href="https://www.mdpi.com/assets/js/lib.js?f0e39afa4d83e329"><link rel="preload" as="script" href="https://www.mdpi.com/assets/js/jquery.cycle2.min.js?63413052928f97ee"><link rel="preload" as="script" href="https://www.mdpi.com/assets/js/jquery.multiselect.js?0edd3998731d1091"><link rel="preload" as="script" href="https://www.mdpi.com/assets/js/foundation-5.5.3.equalizer.min.js?0f6c549b75ec554c"><link rel="preload" as="script" href="https://www.mdpi.com/assets/js/foundation-5.5.3.min.js?6b2ec41c18b29054"><link rel="preload" as="script" href="https://www.mdpi.com/assets/js/jquery-1.12.4.min.js?4f252523d4af0b47"><link rel="preload" as="script" href="modernizr-2.js"><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 5px 0px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 5px; -webkit-border-radius: 5px; -moz-border-radius: 5px; -khtml-border-radius: 5px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 1px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck.RTL {right: .7em; left: auto}
.MathJax_MenuLabel {padding: 1px 2em 3px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #DDDDDD; margin: 4px 3px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: #606872; color: white}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MJX_Assistive_MathML {position: absolute!important; top: 0; left: 0; clip: rect(1px, 1px, 1px, 1px); padding: 1px 0 0 0!important; border: 0!important; height: 1px!important; width: 1px!important; overflow: hidden!important; display: block!important; -webkit-touch-callout: none; -webkit-user-select: none; -khtml-user-select: none; -moz-user-select: none; -ms-user-select: none; user-select: none}
.MJX_Assistive_MathML.MJX_Assistive_MathML_Block {width: 100%!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><script async="" src="hotjar-1635407.js"></script><script src="https://googleads.g.doubleclick.net/pagead/viewthroughconversion/979908000/?random=1588331683572&amp;cv=9&amp;fst=1588331683572&amp;num=1&amp;guid=ON&amp;resp=GooglemKTybQhCsO&amp;u_h=1080&amp;u_w=1920&amp;u_ah=1053&amp;u_aw=1920&amp;u_cd=24&amp;u_his=1&amp;u_tz=120&amp;u_java=false&amp;u_nplug=0&amp;u_nmime=0&amp;gtm=2wg4m0&amp;sendb=1&amp;ig=1&amp;frm=0&amp;url=https%3A%2F%2Fwww.mdpi.com%2F2072-4292%2F12%2F1%2F80%2Fhtm&amp;tiba=Remote%20Sensing%20%7C%20Free%20Full-Text%20%7C%20Modelling%20Distributions%20of%20Rove%20Beetles%20in%20Mountainous%20Areas%20Using%20Remote%20Sensing%20Data%20%7C%20HTML&amp;hn=www.googleadservices.com&amp;async=1&amp;rfmt=3&amp;fmt=4"></script><style></style><meta class="foundation-mq-topbar"><script async="" src="https://script.hotjar.com/modules.64424c84d3fa6c80155d.js" charset="utf-8"></script><style type="text/css">iframe#_hjRemoteVarsFrame {display: none !important; width: 1px !important; height: 1px !important; opacity: 0 !important; pointer-events: none !important;}</style></head>
<body><div id="MathJax_Message" style="display: none;"></div>
<noscript>
            <iframe src="https://www.googletagmanager.com/ns.html?id=GTM-WT3XWXZ" height="0" width="0" style="display:none;visibility:hidden"></iframe>
        </noscript>
<div class="direction direction_right" id="small_right" style="border-right-width: 0px; padding:0;">
<i class="fa fa-caret-right fa-2x"></i>
</div>
<div class="big_direction direction_right" id="big_right" style="border-right-width: 0px;">
<div style="text-align: right;">
Next Article in Journal<br>
<div><a href="https://www.mdpi.com/2072-4292/12/1/81">Generating High-Quality and High-Resolution Seamless Satellite Imagery for Large-Scale Urban Regions</a></div>
Next Article in Special Issue<br>
<div><a href="https://www.mdpi.com/2072-4292/12/3/562">Predicting Microhabitat Suitability for an Endangered Small Mammal Using Sentinel-2 Data</a></div>
</div>
</div>
<div class="direction" id="small_left" style="border-left-width: 0px">
<i class="fa fa-caret-left fa-2x"></i>
</div>
<div class="big_direction" id="big_left" style="border-left-width: 0px;">
<div>
Previous Article in Journal<br>
<div><a href="https://www.mdpi.com/2072-4292/12/1/72">Evaluation of Backscattering Models and Support Vector Machine for the Retrieval of Bare Soil Moisture from Sentinel-1 Data</a></div>
</div>
</div>
<div style="clear: both;"></div>
<div id="menuModal" class="reveal-modal reveal-modal-new reveal-modal-menu" aria-hidden="true" data-reveal="" role="dialog">
<div class="menu-container">
<div class="UI_NavMenu">
<a href="https://www.mdpi.com/about/journals">
<h2>Journals</h2>
</a>
<div class="content__container ">
<div class="custom-accordion-for-small-screen-link ">
<h2>Information</h2>
</div>
<div class="target-item custom-accordion-for-small-screen-content show-for-medium-up">
<div class="menu-container__links">
<div style="width: 100%; max-width: 120px; float: left;">
<a href="https://www.mdpi.com/authors">For Authors</a>
<a href="https://www.mdpi.com/reviewers">For Reviewers</a>
<a href="https://www.mdpi.com/editors">For Editors</a>
<a href="https://www.mdpi.com/librarians">For Librarians</a>
<a href="https://www.mdpi.com/publishing_services">For Publishers</a>
<a href="https://www.mdpi.com/societies">For Societies</a>
</div>
<div style="width: 100%; max-width: 250px; float: left;">
<a href="https://www.mdpi.com/apc">Article Processing Charges</a>
<a href="https://www.mdpi.com/openaccess">Open Access Policy</a>
<a href="https://www.mdpi.com/ioap">Institutional Open Access Program</a>
<a href="https://www.mdpi.com/editorial_process">Editorial Process</a>
<a href="http://blog.mdpi.com/" onclick="if (!window.__cfRLUnblockHandlers) return false; ga('send', 'pageview', 'homeBlog');" target="_blank" rel="noopener noreferrer">MDPI Blog</a>
<a href="https://www.mdpi.com/ethics">Research and Publication Ethics</a>
</div>
</div>
</div>
</div>
<a href="https://www.mdpi.com/authors/english">
<h2>Author Services</h2>
</a>
<div class="content__container ">
<div class="custom-accordion-for-small-screen-link ">
<h2>Initiatives</h2>
</div>
<div class="target-item custom-accordion-for-small-screen-content show-for-medium-up">
<div class="menu-container__links">
<div style="width: 100%; float: left;">
<a href="https://sciforum.net/" target="_blank" rel="noopener noreferrer">Sciforum</a>
<a href="https://www.preprints.org/" target="_blank" rel="noopener noreferrer">Preprints</a>
<a href="https://www.scilit.net/" target="_blank" rel="noopener noreferrer">Scilit</a>
<a href="https://sciprofiles.com/" target="_blank" rel="noopener noreferrer">SciProfiles</a>
<a href="https://www.mdpi.com/books" target="_blank" rel="noopener noreferrer">MDPI Books</a>
<a href="https://encyclopedia.pub/" target="_blank" rel="noopener noreferrer">Encyclopedia</a>
</div>
</div>
</div>
</div>
<a href="https://www.mdpi.com/about">
<h2>About</h2>
</a>
</div>
<div class="menu-container__buttons">
<a class="button UA_SignInUpButton" href="https://www.mdpi.com/user/login">Sign In / Sign Up</a>
</div>
</div>
</div>
<div id="captchaModal" class="reveal-modal reveal-modal-new reveal-modal-new--small" data-reveal="" aria-label="Captcha" aria-hidden="true" role="dialog"></div>
<div id="actionDisabledModal" class="reveal-modal" data-reveal="" aria-labelledby="actionDisableModalTitle" aria-hidden="true" role="dialog" style="width: 300px;">
<h2 id="actionDisableModalTitle">Notice</h2>
<form action="/email/captcha" method="post" id="emailCaptchaForm">
<div class="row">
<div id="js-action-disabled-modal-text" class="small-12 columns">
</div>
<div id="js-action-disabled-modal-submit" class="small-12 columns" style="margin-top: 10px; display: none;">
You can make submissions to other journals
<a href="https://susy.mdpi.com/user/manuscripts/upload" onclick="if (!window.__cfRLUnblockHandlers) return false; ga('send', 'event', 'Submit Paper', 'Send', 'Generic');">here</a>.
</div>
</div>
</form>
<a class="close-reveal-modal" aria-label="Close">
<i class="material-icons">clear</i>
</a>
</div>
<div id="container">
<noscript>
                    <div id="no-javascript">
                        You seem to have javascript disabled. Please note that many of the page functionalities won't work as expected without javascript enabled.
                    </div>
                </noscript>
<div class="fixed">
<nav class="tab-bar show-for-medium-down">
<div class="row full-width collapse">
<div class="medium-3 small-4 columns">
<a href="https://www.mdpi.com/">
<img class="full-size-menu__mdpi-logo" src="mdpi-pub-logo-blue-small4.png" style="max-width: 65px;" title="MDPI Open Access Journals">
</a>
</div>
<div class="medium-3 small-4 columns right-aligned">
<div class="show-for-medium-down">
<a href="#" style="display: none;">
<i class="material-icons" onclick="if (!window.__cfRLUnblockHandlers) return false; $('#menuModal').foundation('reveal', 'close'); return false;">clear</i>
</a>
<a title="Toggle desktop layout" href="https://www.mdpi.com/toggle_desktop_layout_cookie">
<i class="material-icons">zoom_out_map</i>
</a>
<a href="#" class="js-open-small-search open-small-search">
<i class="material-icons show-for-small only">search</i>
</a>
<a title="MDPI main page" class="js-open-menu" data-reveal-id="menuModal" href="#">
<i class="material-icons">menu</i>
</a>
</div>
</div>
</div>
</nav>
</div>
<section class="main-section">
<header>
<div class="full-size-menu show-for-large-up">
<div class="row full-width">
<div class="large-3 columns">
<a href="https://www.mdpi.com/">
<img class="full-size-menu__mdpi-logo" src="mdpi-pub-logo-blue-small4.png" title="MDPI Open Access Journals">
</a>
</div>
<div class="large-6 columns text-center UI_NavMenu">
<ul>
<li class="menu-item">
<a href="https://www.mdpi.com/about/journals">Journals</a>
</li>
<li class="menu-item">
<a href="https://www.mdpi.com/authors" data-dropdown="information-dropdown" aria-controls="information-dropdown" aria-expanded="false" data-options="is_hover:true; hover_timeout:200">Information</a>
<ul id="information-dropdown" class="f-dropdown dropdown-wrapper" data-dropdown-content="" aria-hidden="true" tabindex="-1">
<li>
<div class="row">
<div class="small-4 columns right-border">
<ul>
<li>
<a href="https://www.mdpi.com/authors">For Authors</a>
</li>
<li>
<a href="https://www.mdpi.com/reviewers">For Reviewers</a>
</li>
<li>
<a href="https://www.mdpi.com/editors">For Editors</a>
</li>
<li>
<a href="https://www.mdpi.com/librarians">For Librarians</a>
</li>
<li>
<a href="https://www.mdpi.com/publishing_services">For Publishers</a>
</li>
<li>
<a href="https://www.mdpi.com/societies">For Societies</a>
</li>
</ul>
</div>
<div class="small-8 columns">
<ul>
<li>
<a href="https://www.mdpi.com/apc">Article Processing Charges</a>
</li>
<li>
<a href="https://www.mdpi.com/openaccess">Open Access Policy</a>
</li>
<li>
<a href="https://www.mdpi.com/ioap">Institutional Open Access Program</a>
</li>
<li>
<a href="https://www.mdpi.com/editorial_process">Editorial Process</a>
</li>
<li>
<a href="http://blog.mdpi.com/" onclick="if (!window.__cfRLUnblockHandlers) return false; ga('send', 'pageview', 'homeBlog');" target="_blank" rel="noopener noreferrer">MDPI Blog</a>
</li>
<li>
<a href="https://www.mdpi.com/ethics">Research and Publication Ethics</a>
</li>
</ul>
</div>
</div>
</li>
</ul>
</li>
<li class="menu-item">
<a href="https://www.mdpi.com/authors/english">Author Services</a>
</li>
<li class="menu-item">
<a href="#" data-dropdown="initiatives-dropdown" aria-controls="initiatives-dropdown" aria-expanded="false" data-options="is_hover: true; hover_timeout: 200">Initiatives</a>
<ul id="initiatives-dropdown" class="f-dropdown dropdown-wrapper dropdown-wrapper__small" data-dropdown-content="" aria-hidden="true" tabindex="-1">
<li>
<div class="row">
<div class="small-12 columns">
<ul>
<li>
<a href="https://sciforum.net/" target="_blank" rel="noopener noreferrer">
Sciforum
</a>
</li>
<li>
<a href="https://www.preprints.org/" target="_blank" rel="noopener noreferrer">
Preprints
</a>
</li>
<li>
<a href="https://www.scilit.net/" target="_blank" rel="noopener noreferrer">
Scilit
</a>
</li>
<li>
<a href="https://sciprofiles.com/" target="_blank" rel="noopener noreferrer">
SciProfiles
</a>
</li>
<li>
<a href="https://www.mdpi.com/books" target="_blank" rel="noopener noreferrer">
MDPI Books
</a>
</li>
<li>
<a href="https://encyclopedia.pub/" target="_blank" rel="noopener noreferrer">
Encyclopedia
</a>
</li>
</ul>
</div>
</div>
</li>
</ul>
</li>
<li class="menu-item">
<a href="https://www.mdpi.com/about">About</a>
</li>
</ul>
</div>
<div class="large-3 columns text-right full-size-menu__buttons">
<div>
<a class="button button--default-inversed UA_SignInUpButton" href="https://www.mdpi.com/user/login">Sign In / Sign Up</a>
<a class="button button--default js-journal-active-only-link js-journal-active-only-submit-link UC_NavSubmitButton" href="https://susy.mdpi.com/user/manuscripts/upload?journal=remotesensing" onclick="if (!window.__cfRLUnblockHandlers) return false; ga('send', 'event', 'Submit Paper', 'Send', 'Remote Sensing');" data-disabledmessage="new submissions are not possible.">Submit</a>
</div>
</div>
</div>
</div>
<div class="search-container hide-for-small-down row search-container__default-scheme">
<form id="basic_search" style="background-color: inherit !important;" class="large-12 medium-12 columns " action="/search" method="get">
<div class="row search-container__main-elements">
<div class="large-2 medium-2 small-12 columns text-right1 small-only-text-left">
<div class="show-for-medium-up">
<div class="search-input-label">&nbsp;</div>
</div>
<span class="search-container__title">Search<span class="hide-for-medium"> for Articles</span><span class="hide-for-small">:</span></span>
</div>
<div class="custom-accordion-for-small-screen-content">
<div class="large-2 medium-2 small-6 columns ">
<div class="">
<div class="search-input-label">Title / Keyword</div>
</div>
<input placeholder="Title / Keyword" id="q" tabindex="1" name="q" type="text">
</div>
<div class="large-2 medium-2 small-6 columns ">
<div class="">
<div class="search-input-label">Author / Affiliation</div>
</div>
<input id="authors" placeholder="Author / Affiliation" tabindex="2" name="authors" type="text">
</div>
<div class="large-2 medium-2 small-6 columns ">
<div class="">
<div class="search-input-label">Journal</div>
</div>
<select id="journal" tabindex="3" name="journal" class="chosen-select">
<option value="">All Journals</option>
<option value="acoustics">
Acoustics
</option>
<option value="actuators">
Actuators
</option>
<option value="admsci">
Administrative Sciences
</option>
<option value="aerospace">
Aerospace
</option>
<option value="agriculture">
Agriculture
</option>
<option value="agriengineering">
AgriEngineering
</option>
<option value="agronomy">
Agronomy
</option>
<option value="ai">
AI
</option>
<option value="algorithms">
Algorithms
</option>
<option value="allergies">
Allergies
</option>
<option value="animals">
Animals
</option>
<option value="antibiotics">
Antibiotics
</option>
<option value="antibodies">
Antibodies
</option>
<option value="antioxidants">
Antioxidants
</option>
<option value="applmech">
Applied Mechanics
</option>
<option value="applnano">
Applied Nano
</option>
<option value="applsci">
Applied Sciences
</option>
<option value="asi">
Applied System Innovation (ASI)
</option>
<option value="arts">
Arts
</option>
<option value="atmosphere">
Atmosphere
</option>
<option value="atoms">
Atoms
</option>
<option value="automation">
Automation
</option>
<option value="axioms">
Axioms
</option>
<option value="batteries">
Batteries
</option>
<option value="behavsci">
Behavioral Sciences
</option>
<option value="beverages">
Beverages
</option>
<option value="BDCC">
Big Data and Cognitive Computing (BDCC)
</option>
<option value="bioengineering">
Bioengineering
</option>
<option value="biology">
Biology
</option>
<option value="biomedicines">
Biomedicines
</option>
<option value="biomedinformatics">
BioMedInformatics
</option>
<option value="biomimetics">
Biomimetics
</option>
<option value="biomolecules">
Biomolecules
</option>
<option value="biosensors">
Biosensors
</option>
<option value="bloods">
Bloods
</option>
<option value="brainsci">
Brain Sciences
</option>
<option value="breath">
Breath
</option>
<option value="buildings">
Buildings
</option>
<option value="carbon">
C â€” Journal of Carbon Research (C)
</option>
<option value="cancers">
Cancers
</option>
<option value="catalysts">
Catalysts
</option>
<option value="cells">
Cells
</option>
<option value="ceramics">
Ceramics
</option>
<option value="challenges">
Challenges
</option>
<option value="ChemEngineering">
ChemEngineering
</option>
<option value="chemistry">
Chemistry
</option>
<option value="chemosensors">
Chemosensors
</option>
<option value="children">
Children
</option>
<option value="civileng">
CivilEng
</option>
<option value="cleantechnol">
Clean Technologies (Clean Technol.)
</option>
<option value="climate">
Climate
</option>
<option value="clockssleep">
Clocks &amp; Sleep
</option>
<option value="coatings">
Coatings
</option>
<option value="colloids">
Colloids and Interfaces
</option>
<option value="computation">
Computation
</option>
<option value="computers">
Computers
</option>
<option value="condensedmatter">
Condensed Matter
</option>
<option value="cmd">
Corrosion and Materials Degradation (Corros. Mater. Degrad.)
</option>
<option value="cosmetics">
Cosmetics
</option>
<option value="cryptography">
Cryptography
</option>
<option value="crystals">
Crystals
</option>
<option value="dairy">
Dairy
</option>
<option value="data">
Data
</option>
<option value="dentistry">
Dentistry Journal
</option>
<option value="designs">
Designs
</option>
<option value="diagnostics">
Diagnostics
</option>
<option value="diseases">
Diseases
</option>
<option value="diversity">
Diversity
</option>
<option value="drones">
Drones
</option>
<option value="ecologies">
Ecologies
</option>
<option value="econometrics">
Econometrics
</option>
<option value="economies">
Economies
</option>
<option value="education">
Education Sciences
</option>
<option value="electricity">
Electricity
</option>
<option value="electrochem">
Electrochem
</option>
<option value="electronicmat">
Electronic Materials
</option>
<option value="electronics">
Electronics
</option>
<option value="endocrines">
Endocrines
</option>
<option value="energies">
Energies
</option>
 <option value="eng">
Eng
</option>
<option value="engproc">
Engineering Proceedings
</option>
<option value="entropy">
Entropy
</option>
<option value="environments">
Environments
</option>
<option value="epidemiologia">
Epidemiologia
</option>
<option value="epigenomes">
Epigenomes
</option>
<option value="ejbc">
European Journal of Burn Care (EJBC)
</option>
<option value="ejihpe">
European Journal of Investigation in Health, Psychology and Education (EJIHPE)
</option>
<option value="fermentation">
Fermentation
</option>
<option value="fibers">
Fibers
</option>
<option value="fire">
Fire
</option>
<option value="fishes">
Fishes
</option>
<option value="fluids">
Fluids
</option>
<option value="foods">
Foods
</option>
<option value="forecasting">
Forecasting
</option>
<option value="forests">
Forests
</option>
<option value="fractalfract">
Fractal and Fractional (Fractal Fract)
</option>
<option value="fuels">
Fuels
</option>
<option value="futureinternet">
Future Internet
</option>
<option value="galaxies">
Galaxies
</option>
<option value="games">
Games
</option>
<option value="gastrointestdisord">
Gastrointestinal Disorders (GastrointestDisord)
</option>
<option value="gels">
Gels
</option>
<option value="genealogy">
Genealogy
</option>
<option value="genes">
Genes
</option>
<option value="geography">
Geography
</option>
<option value="geohazards">
GeoHazards
</option>
<option value="geosciences">
Geosciences
</option>
<option value="geriatrics">
Geriatrics
</option>
<option value="healthcare">
Healthcare
</option>
<option value="hearts">
Hearts
</option>
<option value="heritage">
Heritage
</option>
<option value="high-throughput">
High-Throughput
</option>
<option value="horticulturae">
Horticulturae
</option>
<option value="humanities">
Humanities
</option>
<option value="hydrogen">
Hydrogen
</option>
<option value="hydrology">
Hydrology
</option>
<option value="immuno">
Immuno
</option>
<option value="informatics">
Informatics
</option>
<option value="information">
Information
</option>
<option value="infrastructures">
Infrastructures
</option>
<option value="inorganics">
Inorganics
</option>
<option value="insects">
Insects
</option>
<option value="instruments">
Instruments
</option>
<option value="ijd">
International Journal of Diabetology (IJD)
</option>
<option value="ijerph">
International Journal of Environmental Research and Public Health (IJERPH)
</option>
<option value="ijfs">
International Journal of Financial Studies (IJFS)
</option>
<option value="ijms">
International Journal of Molecular Sciences (IJMS)
</option>
<option value="IJNS">
International Journal of Neonatal Screening (IJNS)
</option>
<option value="ijtpp">
International Journal of Turbomachinery, Propulsion and Power (IJTPP)
</option>
<option value="inventions">
Inventions
</option>
<option value="IoT">
IoT
</option>
<option value="ijgi">
ISPRS International Journal of Geo-Information (IJGI)
</option>
<option value="J">
J â€” Multidisciplinary Scientific Journal (J)
</option>
<option value="jcdd">
Journal of Cardiovascular Development and Disease (JCDD)
</option>
<option value="jcm">
Journal of Clinical Medicine (JCM)
</option>
<option value="jcs">
Journal of Composites Science (J. Compos. Sci.)
</option>
<option value="jdb">
Journal of Developmental Biology (JDB)
</option>
<option value="jfb">
Journal of Functional Biomaterials (JFB)
</option>
<option value="jfmk">
Journal of Functional Morphology and Kinesiology (JFMK)
</option>
<option value="jof">
Journal of Fungi (JoF)
</option>
<option value="jimaging">
Journal of Imaging (J. Imaging)
</option>
<option value="jintelligence">
Journal of Intelligence (J. Intell.)
</option>
<option value="jlpea">
Journal of Low Power Electronics and Applications (JLPEA)
</option>
<option value="jmmp">
Journal of Manufacturing and Materials Processing (JMMP)
</option>

<option value="jmse">
Journal of Marine Science and Engineering (JMSE)
</option>
<option value="jne">
Journal of Nuclear Engineering (JNE)
</option>
<option value="JOItmC">
Journal of Open Innovation: Technology, Market, and Complexity (JOItmC)
</option>
<option value="jpm">
Journal of Personalized Medicine (JPM)
</option>
<option value="jrfm">
Journal of Risk and Financial Management (JRFM)
</option>
<option value="jsan">
Journal of Sensor and Actuator Networks (JSAN)
</option>
<option value="land">
Land
</option>
<option value="languages">
Languages
</option>
<option value="laws">
Laws
</option>
<option value="life">
Life
</option>
<option value="livers">
Livers
</option>
<option value="logistics">
Logistics
</option>
<option value="lubricants">
Lubricants
</option>
<option value="make">
Machine Learning and Knowledge Extraction (MAKE)
</option>
<option value="machines">
Machines
</option>
<option value="magnetochemistry">
Magnetochemistry
</option>
<option value="marinedrugs">
Marine Drugs
</option>
<option value="materials">
Materials
</option>
<option value="mca">
Mathematical and Computational Applications (MCA)
</option>
<option value="mathematics">
Mathematics
</option>
<option value="medsci">
Medical Sciences
</option>
<option value="medicina">
Medicina
</option>
<option value="medicines">
Medicines
</option>
<option value="membranes">
Membranes
</option>
<option value="metabolites">
Metabolites
</option>
<option value="metals">
Metals
</option>
<option value="mps">
Methods and Protocols (MPs)
</option>
<option value="micromachines">
Micromachines
</option>
<option value="microorganisms">
Microorganisms
</option>
<option value="minerals">
Minerals
</option>
<option value="modelling">
Modelling
</option>
<option value="molbank">
Molbank
</option>
<option value="molecules">
Molecules
</option>
<option value="mti">
Multimodal Technologies and Interaction (MTI)
</option>
<option value="nanomaterials">
Nanomaterials
</option>
<option value="neurosci">
NeuroSci
</option>
<option value="nitrogen">
Nitrogen
</option>
<option value="ncrna">
Non-Coding RNA (ncRNA)
</option>
<option value="nutrients">
Nutrients
</option>
<option value="oceans">
Oceans
</option>
<option value="optics">
Optics
</option>
<option value="particles">
Particles
</option>
<option value="pathogens">
Pathogens
</option>
<option value="pharmaceuticals">
Pharmaceuticals
</option>
<option value="pharmaceutics">
Pharmaceutics
</option>
<option value="pharmacy">
Pharmacy
</option>
<option value="philosophies">
Philosophies
</option>
<option value="photonics">
Photonics
</option>
<option value="physics">
Physics
</option>
<option value="plants">
Plants
</option>
<option value="plasma">
Plasma
</option>
<option value="pollutants">
Pollutants
</option>
<option value="polymers">
Polymers
</option>
<option value="polysaccharides">
Polysaccharides
</option>
<option value="proceedings">
Proceedings
</option>
<option value="processes">
Processes
</option>
<option value="prosthesis">
Prosthesis
</option>
<option value="proteomes">
Proteomes
</option>
<option value="psych">
Psych
</option>
<option value="psychiatryint">
Psychiatry International
</option>
<option value="publications">
Publications
</option>
<option value="qubs">
Quantum Beam Science (QuBS)
</option>
<option value="quantumrep">
Quantum Reports
</option>
<option value="quaternary">
Quaternary
</option>
<option value="reactions">
Reactions
</option>
<option value="recycling">
Recycling
</option>
<option value="regeneration">
Regeneration
</option>
<option value="religions">
Religions
</option>
<option value="remotesensing" selected="selected">
Remote Sensing
</option>
<option value="reports">
Reports â€” Medical Cases, Images, and Videos (Reports)
</option>
<option value="reprodmed">
Reproductive Medicine (Reprod. Med.)
</option>
<option value="resources">
Resources
</option>
<option value="risks">
Risks
</option>
<option value="robotics">
Robotics
</option>
<option value="safety">
Safety
</option>
<option value="sci">
Sci
</option>
<option value="scipharm">
Scientia Pharmaceutica (Sci. Pharm.)
</option>
<option value="sensors">
Sensors
</option>
<option value="separations">
Separations
</option>
<option value="sexes">
Sexes
</option>
<option value="signals">
Signals
</option>
<option value="sinusitis">
Sinusitis
</option>
<option value="smartcities">
Smart Cities
</option>
<option value="socsci">
Social Sciences
</option>
<option value="societies">
Societies
</option>
<option value="soilsystems">
Soil Systems (Soil Syst.)
</option>
<option value="sports">
Sports
</option>
<option value="stats">
Stats
</option>
<option value="surfaces">
Surfaces
</option>
<option value="surgeries">
Surgeries
</option>
<option value="sustainability">
Sustainability
</option>
<option value="symmetry">
Symmetry
</option>
<option value="systems">
Systems
</option>
<option value="technologies">
Technologies
</option>
<option value="telecom">
Telecom
</option>
<option value="thermo">
Thermo
</option>
<option value="tourismhosp">
Tourism and Hospitality
</option>
<option value="toxics">
Toxics
</option>
<option value="toxins">
Toxins
</option>
<option value="transplantology">
Transplantology
</option>
<option value="tropicalmed">
Tropical Medicine and Infectious Disease (TropicalMed)
</option>
<option value="universe">
Universe
</option>
<option value="urbansci">
Urban Science
</option>
<option value="vaccines">
Vaccines
</option>
<option value="vehicles">
Vehicles
</option>
<option value="vetsci">
Veterinary Sciences
</option>
<option value="vibration">
Vibration
</option>
<option value="viruses">
Viruses
</option>
<option value="vision">
Vision
</option>
<option value="water">
Water
</option>
<option value="women">
Women
</option>
<option value="world">
World
</option>
<option value="wevj">
World Electric Vehicle Journal (WEVJ)
</option>
</select>
</div>
<div class="large-2 medium-2 small-6 columns ">
<div class="">
<div class="search-input-label">Article Type</div>
</div>
<select id="article_type" tabindex="4" name="article_type" class="chosen-select">
<option value="" selected="selected">All Article Types</option>
<option value="research-article">Article</option>
<option value="review-article">Review</option>
<option value="rapid-communication">Communication</option>
<option value="editorial">Editorial</option>
<option value="addendum">Addendum</option>
<option value="book-review">Book Review</option>
<option value="case-report">Case Report</option>
<option value="article-commentary">Comment</option>
<option value="commentary">Commentary</option>
<option value="concept-paper">Concept Paper</option>
<option value="conference-report">Conference Report</option>
<option value="correction">Correction</option>
<option value="creative">Creative</option>
<option value="data-descriptor">Data Descriptor</option>
<option value="discussion">Discussion</option>
<option value="erratum">Erratum</option>
<option value="essay">Essay</option>
<option value="expression-of-concern">Expression of Concern</option>
<option value="interesting-image">Interesting Images</option>
<option value="letter">Letter</option>
<option value="meeting-report">Meeting Report</option>
<option value="books-received">New Book Received</option>
<option value="obituary">Obituary</option>
<option value="opinion">Opinion</option>
<option value="project-report">Project Report</option>
<option value="reply">Reply</option>
<option value="retraction">Retraction</option>
<option value="note">Short Note</option>
<option value="technical-note">Technical Note</option>
<option value="tutorial">Tutorial</option>
</select>
</div>
<div class="large-1 medium-1 small-6 end columns small-push-6 medium-reset-order large-reset-order js-search-collapsed-button-container">
<div class="search-input-label">&nbsp;</div>
<input id="search" value="Search" class="button button--dark button--full-width searchButton1 US_SearchButton" tabindex="12" name="search" type="submit">
</div>
<div class="large-1 medium-1 small-6 end columns large-text-left small-only-text-center small-pull-6 medium-reset-order large-reset-order js-search-collapsed-link-container">
<div class="search-input-label">&nbsp;</div>
<a class="main-search-clear search-container__link" href="#" onclick="if (!window.__cfRLUnblockHandlers) return false; openAdvanced(''); return false;">Advanced<span class="show-for-small-only"> Search</span></a>
</div>
</div>
</div>
<div class="search-container__advanced" style="margin-top: 0; padding-top: 0px; background-color: inherit; color: inherit;">
<div class="row">
<div class="large-2 medium-2 columns show-for-medium-up">&nbsp;</div>
<div class="large-2 medium-2 small-6 columns ">
<div class="">
<div class="search-input-label">Section</div>
</div>
<select id="section" tabindex="5" name="section" class="chosen-select">
<option value="" selected="selected"></option>
</select>
</div>
<div class="large-2 medium-2 small-6 columns ">
<div class="">
<div class="search-input-label">Special Issue</div>
</div>
<select id="special_issue" tabindex="6" name="special_issue" class="chosen-select">
<option value="" selected="selected"></option>
</select>
</div>
<div class="large-1 medium-1 small-6 end columns ">
<div class="search-input-label">Volume</div>
<input id="volume" tabindex="7" name="volume" placeholder="..." value="12" type="text">
</div>
<div class="large-1 medium-1 small-6 end columns ">
<div class="search-input-label">Issue</div>
<input id="issue" tabindex="8" name="issue" placeholder="..." value="1" type="text">
</div>
<div class="large-1 medium-1 small-6 end columns ">
<div class="search-input-label">Number</div>
<input id="number" tabindex="9" name="number" placeholder="..." type="text">
</div>
<div class="large-1 medium-1 small-6 end columns ">
<div class="search-input-label">Page</div>
<input id="page" tabindex="10" name="page" placeholder="..." type="text">
</div>
<div class="large-1 medium-1 small-6 columns small-push-6 medium-reset order large-reset-order medium-reset-order js-search-expanded-button-container"></div>
<div class="large-1 medium-1 small-6 columns large-text-left small-only-text-center small-pull-6 medium-reset-order large-reset-order js-search-expanded-link-container"></div>
</div>
</div>
</form>
<form id="advanced-search" class="large-12 medium-12 columns">
<div class="search-container__advanced">
<div class="row">
<div class="large-2 medium-2 small-12 columns">&nbsp;</div>
<div class="large-8 medium-8 small-12 columns">
<div style="border-top: 1px solid #f5f5f5;">
<div class="info-box">
You can use * to search for partial matches.
</div>
</div>
</div>
<div id="advanced-search-button-hideout" class="large-1 medium-1 show-for-medium-up columns">&nbsp;
<input id="advanced-search-button" class="button button--dark search-submit" value="Search" type="submit">
</div>
<div class="large-1 medium-1 show-for-medium-up columns">&nbsp;</div>
</div>
<div id="advanced-search-template" class="row advanced-search-row">
<div class="large-2 medium-2 small-12 columns show-for-medium-up">&nbsp;</div>
<div class="large-2 medium-2 small-3 columns connector-div">
<div class="search-input-label"><span class="show-for-medium-up">Logical Operator</span><span class="show-for-small">Operator</span></div>
<select class="connector">
<option value="and" selected="selected">AND</option>
<option value="or">OR</option>
</select>
</div>
<div class="large-3 medium-3 small-6 columns search-text-div">
<div class="search-input-label">Search Text</div>
<input class="search-text" placeholder="Search text" type="text">
</div>
<div class="large-2 medium-2 small-6 large-offset-0 medium-offset-0 small-offset-3 columns search-field-div">
<div class="search-input-label">Search Type</div>
<select class="search-field">
<option value="all" selected="selected">All fields</option>
<option value="title">Title</option>
<option value="abstract">Abstract</option>
<option value="keywords">Keywords</option>
<option value="authors">Authors</option>
<option value="affiliations">Affiliations</option>
<option value="doi">Doi</option>
<option value="full_text">Full Text</option>
<option value="references">References</option>
</select>
</div>
<div class="large-1 medium-1 small-3 columns">
<div class="search-input-label">&nbsp;</div>
<div class="search-action-div">
<div class="search-plus">
<i class="material-icons">add_circle_outline</i>
</div>
</div>
<div class="search-action-div">
<div class="search-minus">
<i class="material-icons">remove_circle_outline</i>
</div>
</div>
</div>
<div class="large-1 medium-1 small-6 large-offset-0 medium-offset-0 small-offset-3 end columns">
<div class="search-input-label">&nbsp;</div>
</div>
<div class="large-1 medium-1 small-6 end columns show-for-medium-up"></div>
</div>
</div>
</form>
</div>
<div class="breadcrumb row full-row">
<div class="breadcrumb__element">
<a href="https://www.mdpi.com/about/journals">Journals</a>
</div>
<div class="breadcrumb__element">
<a href="https://www.mdpi.com/journal/remotesensing">Remote Sensing</a>
</div>
<div class="breadcrumb__element">
<a href="https://www.mdpi.com/2072-4292/12">Volume 12</a>
</div>
<div class="breadcrumb__element">
<a href="https://www.mdpi.com/2072-4292/12/1">Issue 1</a>
</div>
<div class="breadcrumb__element">
<a href="#">10.3390/rs12010080</a>
</div>
</div>
</header>
<div id="main-content">
<div class="row full-width">
<div id="left-column" class="content__column large-3 medium-3 small-12 columns">
<div class="content__container">
<a href="https://www.mdpi.com/journal/remotesensing">
<img src="remotesensing-logo.png" alt="remotesensing-logo" title="remotesensing Open Access Journals" style="max-height: 60px; margin: 0 0 0 0;">
</a>
<div class="generic-item no-border">
<a class="button button--color button--full-width js-journal-active-only-link js-journal-active-only-submit-link UC_ArticleSubmitButton" href="https://susy.mdpi.com/user/manuscripts/upload?form%5Bjournal_id%5D%3D16" onclick="if (!window.__cfRLUnblockHandlers) return false; ga('send', 'event', 'Submit Paper', 'Send', 'Remote Sensing');" data-disabledmessage="creating new submissions is not possible.">
Submit to this Journal
</a>
<a class="button button--color button--full-width js-journal-active-only-link UC_ArticleReviewButton" href="https://susy.mdpi.com/volunteer/journals/review" data-disabledmessage="volunteering as journal reviewer is not possible.">
Review for this Journal
</a>
<a class="button button--color button--full-width js-journal-active-only-link UC_ArticleEditIssueButton" href="https://www.mdpi.com/journalproposal/sendproposalspecialissue/remotesensing" data-path="/2072-4292/12/1/80/htm" data-disabledmessage="proposing new special issue is not possible.">
Edit a Special Issue
</a>
</div>
<div class="generic-item link-article-menu show-for-small">
<a href="#" class="link-article-menu show-for-small">
<span class="closed">â–º</span>
<span class="open" style="display: none;">â–¼</span>
Article Menu
</a>
</div>
<div class="hide-small-down-initially UI_ArticleMenu">
<div class="generic-item">
<h2>Article Menu</h2>
</div>
<ul class="accordion accordion__menu" data-accordion="" data-options="multi_expand:false;toggleable: true">
<li class="accordion-navigation">
<a href="#overview" class="accordion__title" aria-expanded="false">Article Overview</a>
<div id="overview" class="content  UI_ArticleMenu_Overview">
<ul>
<li>
<a href="https://www.mdpi.com/2072-4292/12/1/80#abstract">Abstract</a>
</li>
<li>
<a href="https://www.mdpi.com/2072-4292/12/1/80#supplementary">Supplementary Material</a>
</li>
<li>
<a href="https://www.mdpi.com/openaccess">Open Access and Permissions</a>
</li>
<li>
<a href="https://www.mdpi.com/2072-4292/12/1/80#cite">Share and Cite</a>
</li>
<li>
<a href="https://www.mdpi.com/2072-4292/12/1/80#metrics">Article Metrics</a>
</li>
<li>
<a href="https://www.mdpi.com/2072-4292/12/1/80#related_content">Related Articles</a>
</li>
<li>
<a href="https://www.mdpi.com/2072-4292/12/1/80/reprints">Order Article Reprints</a>
</li>
</ul>
</div>
</li>
<li class="accordion-navigation">
<a href="#versions-div" class="accordion__title" aria-expanded="false">Article Versions</a>
<div id="versions-div" class="content  UI_ArticleMenu_Versions">
<ul>
<li>
<a href="https://www.mdpi.com/2072-4292/12/1/80">Abstract</a>
</li>
<li>
<a href="https://www.mdpi.com/2072-4292/12/1/80/notes" onclick="if (!window.__cfRLUnblockHandlers) return false; ga('send', 'pageview', '/2072-4292/12/1/80/notes');">Article Versions Notes</a>
</li>
<li>
<a href="https://www.mdpi.com/2072-4292/12/1/80/htm" onclick="if (!window.__cfRLUnblockHandlers) return false; ga('send', 'pageview', $(this).attr('href'));" id="html_link">Full-Text HTML</a>
</li>
<li class="li-download">
<a href="https://www.mdpi.com/2072-4292/12/1/80/pdf" onclick="if (!window.__cfRLUnblockHandlers) return false; ga('send', 'pageview', $(this).attr('href'));" id="pdf_link">Full-Text PDF</a>
</li>
<li class="li-download">
<a id="js-xml-access-captcha" href="#" data-target="/2072-4292/12/1/80/xml" class="accessCaptcha" onclick="if (!window.__cfRLUnblockHandlers) return false; ga('send', 'pageview', '/2072-4292/12/1/80/xml');">Full-Text XML</a>
</li>
<li class="li-download">
<a href="https://www.mdpi.com/2072-4292/12/1/80/epub" onclick="if (!window.__cfRLUnblockHandlers) return false; ga('send', 'pageview', '/2072-4292/12/1/80/epub');" id="epub_link">Full-Text Epub</a>
</li>
</ul>
</div>
</li>
<li class="accordion-navigation">
<a href="#exports_menu" class="accordion__title" aria-expanded="false">Export Article</a>
<div id="exports_menu" class="content  UI_ArticleMenu_Export">

<form style="margin:0; padding:0; display:inline;" name="export-bibtex" method="POST" action="/export">
<input name="articles_ids[]" value="301594" type="hidden">
<input name="export_format_top" value="bibtex" type="hidden">
<input name="export_submit_top" value="" type="hidden">
</form>

<form style="margin:0; padding:0; display:inline;" name="export-endnote" method="POST" action="/export">
<input name="articles_ids[]" value="301594" type="hidden">
<input name="export_format_top" value="endnote_no_abstract" type="hidden">
<input name="export_submit_top" value="" type="hidden">
</form>

<form style="margin:0; padding:0; display:inline;" name="export-ris" method="POST" action="/export">
<input name="articles_ids[]" value="301594" type="hidden">
<input name="export_format_top" value="ris" type="hidden">
<input name="export_submit_top" value="" type="hidden">
</form>
<ul>
<li class="li-link">
<a href="javascript:window.document.forms['export-bibtex'].submit()">BibTeX</a>
</li>
<li class="li-link">
<a href="javascript:window.document.forms['export-endnote'].submit()">EndNote</a>
 </li>
<li class="li-link">
<a href="javascript:window.document.forms['export-ris'].submit()">RIS</a>
</li>
</ul>
</div>
</li>
<li class="accordion-navigation">
<a href="#related" class="accordion__title" aria-expanded="false">Related Info Links</a>
<div id="related" class="content  UI_ArticleMenu_RelatedLinks">
<ul>
<li class="li-link">
<a href="https://scholar.google.com/scholar?q=Modelling%20Distributions%20of%20Rove%20Beetles%20in%20Mountainous%20Areas%20Using%20Remote%20Sensing%20Data" target="_blank" rel="noopener noreferrer">Google Scholar</a>
</li>
</ul>
</div>
</li>
<li class="accordion-navigation">
<a href="#authors" class="accordion__title" aria-expanded="false">More by Authors Links</a>
<div id="authors" class="content  UI_ArticleMenu_AuthorsLinks">
<ul class="side-menu-ul">
<li>
<a class="expand" onclick='if (!window.__cfRLUnblockHandlers) return false; $(this).closest("li").next("div").toggle(); return false;'>on DOAJ</a>
</li>
<div id="AuthorDOAJExpand" style="display:none;">
<ul class="submenu">
<li class="li-link">
<a href="http://doaj.org/search?source=%7B%22query%22%3A%7B%22query_string%22%3A%7B%22query%22%3A%22Andreas%20Dittrich%22%2C%22default_operator%22%3A%22AND%22%7D%7D%7D" target="_blank" rel="noopener noreferrer">Dittrich, A.</a>
</li><li>
</li>
<li class="li-link">
<a href="http://doaj.org/search?source=%7B%22query%22%3A%7B%22query_string%22%3A%7B%22query%22%3A%22Stephanie%20Roilo%22%2C%22default_operator%22%3A%22AND%22%7D%7D%7D" target="_blank" rel="noopener noreferrer">Roilo, S.</a>
</li><li>
</li>
<li class="li-link">
<a href="http://doaj.org/search?source=%7B%22query%22%3A%7B%22query_string%22%3A%7B%22query%22%3A%22Ruth%20Sonnenschein%22%2C%22default_operator%22%3A%22AND%22%7D%7D%7D" target="_blank" rel="noopener noreferrer">Sonnenschein, R.</a>
</li><li>
</li>
<li class="li-link">
<a href="http://doaj.org/search?source=%7B%22query%22%3A%7B%22query_string%22%3A%7B%22query%22%3A%22Cristiana%20Cerrato%22%2C%22default_operator%22%3A%22AND%22%7D%7D%7D" target="_blank" rel="noopener noreferrer">Cerrato, C.</a>
</li><li>
</li>
<li class="li-link">
<a href="http://doaj.org/search?source=%7B%22query%22%3A%7B%22query_string%22%3A%7B%22query%22%3A%22Michael%20Ewald%22%2C%22default_operator%22%3A%22AND%22%7D%7D%7D" target="_blank" rel="noopener noreferrer">Ewald, M.</a>
</li><li>
</li>
<li class="li-link">
<a href="http://doaj.org/search?source=%7B%22query%22%3A%7B%22query_string%22%3A%7B%22query%22%3A%22Ramona%20Viterbi%22%2C%22default_operator%22%3A%22AND%22%7D%7D%7D" target="_blank" rel="noopener noreferrer">Viterbi, R.</a>
</li><li>
</li>
<li class="li-link">
<a href="http://doaj.org/search?source=%7B%22query%22%3A%7B%22query_string%22%3A%7B%22query%22%3A%22Anna%20%20F.%20Cord%22%2C%22default_operator%22%3A%22AND%22%7D%7D%7D" target="_blank" rel="noopener noreferrer">Cord, A. F.</a>
</li><li>
</li>
</ul>
</div>
<li>
<a class="expand" onclick='if (!window.__cfRLUnblockHandlers) return false; $(this).closest("li").next("div").toggle(); return false;'>on Google Scholar</a>
</li>
<div id="AuthorGoogleExpand" style="display:none;">
<ul class="submenu">
<li class="li-link">
<a href="https://scholar.google.com/scholar?q=Andreas%20Dittrich" target="_blank" rel="noopener noreferrer">Dittrich, A.</a>
</li><li>
</li>
<li class="li-link">
<a href="https://scholar.google.com/scholar?q=Stephanie%20Roilo" target="_blank" rel="noopener noreferrer">Roilo, S.</a>
</li><li>
</li>
<li class="li-link">
<a href="https://scholar.google.com/scholar?q=Ruth%20Sonnenschein" target="_blank" rel="noopener noreferrer">Sonnenschein, R.</a>
</li><li>
</li>
<li class="li-link">
<a href="https://scholar.google.com/scholar?q=Cristiana%20Cerrato" target="_blank" rel="noopener noreferrer">Cerrato, C.</a>
</li><li>
</li>
<li class="li-link">
<a href="https://scholar.google.com/scholar?q=Michael%20Ewald" target="_blank" rel="noopener noreferrer">Ewald, M.</a>
</li><li>
</li>
<li class="li-link">
<a href="https://scholar.google.com/scholar?q=Ramona%20Viterbi" target="_blank" rel="noopener noreferrer">Viterbi, R.</a>
</li><li>
</li>
<li class="li-link">
<a href="https://scholar.google.com/scholar?q=Anna%20%20F.%20Cord" target="_blank" rel="noopener noreferrer">Cord, A. F.</a>
</li><li>
</li>
</ul>
</div>
<li>
<a class="expand" onclick='if (!window.__cfRLUnblockHandlers) return false; $(this).closest("li").next("div").toggle(); return false;'>on PubMed</a>
</li>
<div id="AuthorPubMedExpand" style="display:none;">
<ul class="submenu">
<li class="li-link">
<a href="https://www.pubmed.gov/?cmd=Search&amp;term=Andreas%20Dittrich" target="_blank" rel="noopener noreferrer">Dittrich, A.</a>
</li><li>
</li>
<li class="li-link">
<a href="https://www.pubmed.gov/?cmd=Search&amp;term=Stephanie%20Roilo" target="_blank" rel="noopener noreferrer">Roilo, S.</a>
</li><li>
</li>
<li class="li-link">
<a href="https://www.pubmed.gov/?cmd=Search&amp;term=Ruth%20Sonnenschein" target="_blank" rel="noopener noreferrer">Sonnenschein, R.</a>
</li><li>
</li>
<li class="li-link">
<a href="https://www.pubmed.gov/?cmd=Search&amp;term=Cristiana%20Cerrato" target="_blank" rel="noopener noreferrer">Cerrato, C.</a>
</li><li>
</li>
<li class="li-link">
<a href="https://www.pubmed.gov/?cmd=Search&amp;term=Michael%20Ewald" target="_blank" rel="noopener noreferrer">Ewald, M.</a>
</li><li>
</li>
<li class="li-link">
<a href="https://www.pubmed.gov/?cmd=Search&amp;term=Ramona%20Viterbi" target="_blank" rel="noopener noreferrer">Viterbi, R.</a>
</li><li>
</li>
<li class="li-link">
<a href="https://www.pubmed.gov/?cmd=Search&amp;term=Anna%20%20F.%20Cord" target="_blank" rel="noopener noreferrer">Cord, A. F.</a>
</li><li>
</li>
</ul>
</div>
</ul>
</div>
</li>
<li class="accordion-navigation active">
<a href="#html-links" class="accordion__title" aria-expanded="true">Full Article Text</a>
<div id="html-links" class="content active">
<div class="menu-caption" id="html-quick-links-title"></div>
</div>
</li>
</ul>
<div id="scifeed-modal" class="reveal-modal reveal-modal-new" data-reveal="" aria-labelledby="modalTitle" aria-hidden="true" role="dialog">
</div>
<span style="display:none" id="scifeed_hidden_flag"></span>
<span style="display:none" id="scifeed_subscribe_url">/ajax/scifeed/subscribe</span>
</div>
</div>
</div>
<div id="middle-column" class="content__column large-9 medium-9 small-12 columns end middle-bordered">
<div class="middle-column__help">
<div class="middle-column__help__fixed show-for-medium-up">
<a href="#" class="UA_ShareButton" data-reveal-id="main-share-modal" title="Share">
<i class="material-icons">share</i>
</a>
<a href="#" data-reveal-id="main-help-modal" title="Help">
<i class="material-icons">announcement</i>
</a>
<a href="#" class="" data-hypothesis-trigger-endorses-tab="" title="Endorse">
<i data-hypothesis-endorse-trigger="" class="material-icons">thumb_up</i>
<div data-hypothesis-endorsement-count="" data-hypothesis-trigger-endorses-tab="" class="hypothesis-count-container">
...
</div>
</a>
<a href="#" data-hypothesis-trigger="" class="js-hypothesis-open UI_ArticleAnnotationsButton" title="Comment">
<i class="material-icons">textsms</i>
<div data-hypothesis-annotation-count="" class="hypothesis-count-container">
...
</div>
</a>
</div>
<div id="main-help-modal" class="reveal-modal reveal-modal-new" data-reveal="" aria-labelledby="modalTitle" aria-hidden="true" role="dialog">
<div class="row">
<div class="small-12 columns">
<h2 style="margin: 0;">Need Help?</h2>
</div>
<div class="small-6 columns">
<h3>Support</h3>
<p>
Find support for a specific problem on the support section of our website.
</p>
<a target="_blank" href="https://www.mdpi.com/about/contactform" class="button button--color button--full-width">
Get Support
</a>
</div>
<div class="small-6 columns">
<h3>Feedback</h3>
<p>
Please let us know what you think of our products and services.
</p>
<a target="_blank" href="https://www.mdpi.com/feedback/send" class="button button--color button--full-width">
Give Feedback
</a>
</div>
<div class="small-6 columns end">
<h3>Information</h3>
<p>
Our dedicated information section provides allows you to learn more about MDPI.
</p>
<a target="_blank" href="https://www.mdpi.com/authors" class="button button--color button--full-width">
Get Information
</a>
</div>
</div>
<a class="close-reveal-modal" aria-label="Close">
<i class="material-icons">clear</i>
</a>
</div>
</div>
<div class="middle-column__main ">
<div class="html-content__container content__container content__container__combined-for-large content__container__combined-for-large__first" style="overflow: auto;">
<article>
<div class="html-article-content">
<div itemscope="" itemtype="http://schema.org/ScholarlyArticle" id="abstract" class="abstract_div">
<div class="js-check-update-container"></div>
<div>
<span itemprop="publisher" content="Multidisciplinary Digital Publishing Institute"></span><span itemprop="url" content="https://www.mdpi.com/2072-4292/12/1/80"></span>
<div class="article-icons"><span class="label openaccess">Open Access</span><span class="label articletype">Article</span></div>
<h1 class="title hypothesis_container" itemprop="name">
Modelling Distributions of Rove Beetles in Mountainous Areas Using Remote Sensing Data </h1>
<div class="art-authors hypothesis_container">

by
<span class="inlineblock"><span style="white-space: nowrap"><img src="masked-unknown-user.png" style="width: auto; height: 16px; margin-right: 3px; margin-top: -3px;"><a href="https://sciprofiles.com/profile/author/YkFLNVdNRWtnVS90b1FoOWcyTUVtMGlCUUtBU2JZVHBZbkFEVk1YMmtHWT0=" target="_blank" rel="noopener noreferrer">Andreas Dittrich</a></span><sup> 1,â€ </sup><span style="display: inline; margin-left: 5px;"></span><a class="toEncode emailCaptcha" data-author-id="2375202" href="mailto:please_login"><sup><i class="fa fa-envelope-o"></i></sup></a>, </span><span class="inlineblock"><span style="white-space: nowrap"><img src="masked-unknown-user.png" style="width: auto; height: 16px; margin-right: 3px; margin-top: -3px;"><a href="https://sciprofiles.com/profile/924829" target="_blank" rel="noopener noreferrer">Stephanie Roilo</a></span><sup> 2,â€ </sup><span style="display: inline; margin-left: 5px;"></span><a class="toEncode emailCaptcha" data-author-id="2375203" href="mailto:please_login"><sup><i class="fa fa-envelope-o"></i></sup></a>, </span><span class="inlineblock"><span style="white-space: nowrap"><img src="masked-unknown-user.png" style="width: auto; height: 16px; margin-right: 3px; margin-top: -3px;"><a href="https://sciprofiles.com/profile/author/S1RIcUd1WVRGY2NLMkJURkNqMzVwYzd3YjE2NFF4N3NPeCtObFZyY3AwQT0=" target="_blank" rel="noopener noreferrer">Ruth Sonnenschein</a></span><sup> 2</sup><span style="display: inline; margin-left: 5px;"></span><a class="toEncode emailCaptcha" data-author-id="2375204" href="mailto:please_login"><sup><i class="fa fa-envelope-o"></i></sup></a>, </span><span class="inlineblock"><span style="white-space: nowrap"><img src="masked-unknown-user.png" style="width: auto; height: 16px; margin-right: 3px; margin-top: -3px;"><a href="https://sciprofiles.com/profile/473606" target="_blank" rel="noopener noreferrer">Cristiana Cerrato</a></span><sup> 3</sup><span style="display: inline; margin-left: 5px;"></span><a class="toEncode emailCaptcha" data-author-id="2375205" href="mailto:please_login"><sup><i class="fa fa-envelope-o"></i></sup></a>, </span><span class="inlineblock"><span style="white-space: nowrap"><img src="masked-unknown-user.png" style="width: auto; height: 16px; margin-right: 3px; margin-top: -3px;"><a href="https://sciprofiles.com/profile/author/Nm9XeWNpSmVPa1ZrTWlHVW4xazZwdndrblhHNVRveVBBbm1idW1OSVQyUT0=" target="_blank" rel="noopener noreferrer">Michael Ewald</a></span><sup> 4</sup><span style="display: inline; margin-left: 5px;"></span><a class="toEncode emailCaptcha" data-author-id="2375206" href="mailto:please_login"><sup><i class="fa fa-envelope-o"></i></sup></a>, </span><span class="inlineblock"><span style="white-space: nowrap"><img src="masked-unknown-user.png" style="width: auto; height: 16px; margin-right: 3px; margin-top: -3px;"><a href="https://sciprofiles.com/profile/974111" target="_blank" rel="noopener noreferrer">Ramona Viterbi</a></span><sup> 3</sup><span style="display: inline; margin-left: 5px;"></span><a class="toEncode emailCaptcha" data-author-id="2375207" href="mailto:please_login"><sup><i class="fa fa-envelope-o"></i></sup></a> and </span><span class="inlineblock"><span style="white-space: nowrap"><img src="masked-unknown-user.png" style="width: auto; height: 16px; margin-right: 3px; margin-top: -3px;"><a href="https://sciprofiles.com/profile/41774" target="_blank" rel="noopener noreferrer">Anna F. Cord</a></span><sup> 1,*</sup><span style="display: inline; margin-left: 5px;"></span><a class="toEncode emailCaptcha" data-author-id="2375208" href="mailto:please_login"><sup><i class="fa fa-envelope-o"></i></sup></a><a href="https://orcid.org/0000-0003-3183-8482" target="_blank" rel="noopener noreferrer"><img src="orcid.png" title="OrcID" style="position: relative; width: 13px; margin-left: 3px; max-width: 13px !important; height: auto; top: -5px;"></a></span> 
</div>
<div class="nrm"></div>
<span style="display:block; height:6px;"></span>
<div></div>
<div style="margin: 5px 0 15px 0;" class="hypothesis_container">
<div class="art-affiliations">
<div class="affiliation">
<div class="affiliation-item"><sup>1</sup></div>
<div class="affiliation-name ">Department of Computational Landscape Ecology, Helmholtz Centre for Environmental Researchâ€”UFZ, 04318 Leipzig, Germany</div>
</div>
<div class="affiliation">
<div class="affiliation-item"><sup>2</sup></div>
<div class="affiliation-name ">Eurac Research, Institute for Earth Observation, 39100 Bozen/Bolzano, Italy</div>
</div>
<div class="affiliation">
<div class="affiliation-item"><sup>3</sup></div>
<div class="affiliation-name ">Gran Paradiso National Park, 10135 Turin, Italy</div>
</div>
<div class="affiliation">
<div class="affiliation-item"><sup>4</sup></div>
<div class="affiliation-name ">Karlsruhe Institute of Technology (KIT), Institute of Geography and Geoecology, 76131 Karlsruhe, Germany</div>
</div>
<div class="affiliation">
<div class="affiliation-item"><sup>*</sup></div>
<div class="affiliation-name ">Author to whom correspondence should be addressed. </div>
</div>
<div class="affiliation">
<div class="affiliation-item"><sup>â€ </sup></div>
<div class="affiliation-name ">Shared first authorship.</div>
</div>
</div>
</div>
<div class="bib-identity" style="margin-bottom: 10px;">
<em>Remote Sens.</em> <b>2020</b>, <em>12</em>(1), 80; <a href="https://doi.org/10.3390/rs12010080">https://doi.org/10.3390/rs12010080</a>
</div>
<div class="pubhistory" style="font-weight: bold; padding-bottom: 10px;">
Received: 1 December 2019 / Revised: 20 December 2019 / Accepted: 22 December 2019 / Published: 24 December 2019
</div>
<div class="belongsTo" style="margin-bottom: 10px;">
(This article belongs to the Special Issue <a href="https://www.mdpi.com/journal/remotesensing/special_issues/biodiversity_mapping_monitoring">Remote Sensing for Biodiversity Mapping and Monitoring</a>)
</div>
<div class="highlight-box1">
<div class="download">
<a class="button button--color-inversed margin-bottom-10 UD_ArticlePDF" href="https://www.mdpi.com/2072-4292/12/1/80/pdf" onclick="if (!window.__cfRLUnblockHandlers) return false; ga('send', 'pageview', '/2072-4292/12/1/80/pdf');">Download PDF</a>
<div class="js-browse-figures" style="display: inline-block;">
<a href="#" class="button button--color-inversed margin-bottom-10 openpopupgallery UI_BrowseArticleFigures" data-target="article-popup">Browse Figures</a>
</div>
<div id="article-popup" class="popupgallery" style="display: inline; line-height: 200%">
<a href="https://www.mdpi.com/remotesensing/remotesensing-12-00080/article_deploy/html/images/remotesensing-12-00080-ag.png" title="
                        &lt;strong&gt;Graphical abstract&lt;/strong&gt;&lt;br/&gt;
                                                    
                                                ">
</a>
<a href="https://www.mdpi.com/remotesensing/remotesensing-12-00080/article_deploy/html/images/remotesensing-12-00080-g001.png" title="
                        &lt;strong&gt;Figure 1&lt;/strong&gt;&lt;br/&gt;
                                                    &lt;p&gt;Study area and sampling design for long-term biodiversity monitoring efforts. (&lt;bold&gt;a&lt;/bold&gt;) Location of Gran Paradiso National Park. A buffer of 1 km was added to the official boundaries of the park to avoid edge effects when modelling species distributions. (&lt;bold&gt;b&lt;/bold&gt;) Locations of transects within the park boundaries. (&lt;bold&gt;c&lt;/bold&gt;) Schematic representation of the distribution of plots along a transect (following contour lines). (&lt;bold&gt;d&lt;/bold&gt;) Locations of pitfall traps within each plot.&lt;/p&gt;
                                                ">
</a>
<a href="https://www.mdpi.com/remotesensing/remotesensing-12-00080/article_deploy/html/images/remotesensing-12-00080-g002.png" title="
                        &lt;strong&gt;Figure 2&lt;/strong&gt;&lt;br/&gt;
                                                    &lt;p&gt;Comparison of modelled presence probabilities (ensemble models) at presence (traps in which individuals of each target species were captured) and absence locations (traps without captured individuals, not used for model training) for the time periods 2006â€“2007 and 2012â€“2013.&lt;/p&gt;
                                                ">
</a>
<a href="https://www.mdpi.com/remotesensing/remotesensing-12-00080/article_deploy/html/images/remotesensing-12-00080-g003.png" title="
                        &lt;strong&gt;Figure 3&lt;/strong&gt;&lt;br/&gt;
                                                    &lt;p&gt;Species distribution likelihood for each species in 2006â€“2007 and 2012â€“2013 based on the ensemble models. Locations of the plots used as presence points for model training are shown to emphasize the variability of input data. Coordinates shown are based ETRS (European Terrestrial Reference System) 1989 LAEA (Lambert Azimuthal Equal-Area) projection.&lt;/p&gt;
                                                ">
</a>
</div>
<a class="button button--color-inversed" href="#cite">Cite This Paper</a>
</div>
</div>
<div class="responsive-moving-container small hidden" data-id="article-counters" style="margin-top: 15px;"></div>
</div>
<div class="html-abstract-title">
<a name="abstractc"></a>
<h2>Abstract</h2>
</div>
<div class="art-abstract in-tab hypothesis_container">
Mountain ecosystems are biodiversity hotspots that are increasingly 
threatened by climate and land use/land cover changes. Long-term 
biodiversity monitoring programs provide unique insights into resulting 
adverse impacts on plant and animal species distribution. Species 
distribution models (SDMs) in combination with satellite remote sensing 
(SRS) data offer the opportunity to analyze shifts of species 
distributions in response to these changes in a spatially explicit way. 
Here, we predicted the presence probability of three different rove 
beetles in a mountainous protected area (Gran Paradiso National Park, 
GPNP) using environmental variables derived from Landsat and Aster 
Global Digital Elevation Model data and an ensemble modelling approach 
based on five different model algorithms (maximum entropy, random 
forest, generalized boosting models, generalized additive models, and 
generalized linear models). The objectives of the study were (1) to 
evaluate the potential of SRS data for predicting the presence of 
species dependent on local-scale environmental parameters at two 
different time periods, (2) to analyze shifts in species distributions 
between the years, and (3) to identify the most important 
species-specific SRS predictor variables. All ensemble models showed 
area under curve (AUC) of the receiver operating characteristics values 
above 0.7 and true skills statistics (TSS) values above 0.4, 
highlighting the great potential of SRS data. While only a small 
proportion of the total area was predicted as highly suitable for each 
species, our results suggest an increase of suitable habitat over time 
for the species <span class="html-italic">Platydracus stercorarius</span> and <span class="html-italic">Ocypus ophthalmicus</span>, and an opposite trend for <span class="html-italic">Dinothenarus fossor</span>.
 Vegetation cover was the most important predictor variable in the 
majority of the SDMs across all three study species. To better account 
for intra- and inter-annual variability of population dynamics as well 
as environmental conditions, a continuation of the monitoring program in
 GPNP as well as the employment of SRS with higher spatial and temporal 
resolution is recommended.
</div>
<div class="art-keywords in-tab hypothesis_container"><em>Keywords: </em>
<span itemprop="keywords" style="display:none">temporal analysis; 
species distribution model; Landsat; ASTER GDEM; ensemble modelling; 
protected area; Italian Alps; biodiversity monitoring</span>
<span> <a href="https://www.mdpi.com/search?q=temporal%20analysis">temporal analysis</a>; <a href="https://www.mdpi.com/search?q=species%20distribution%20model">species distribution model</a>; <a href="https://www.mdpi.com/search?q=Landsat">Landsat</a>; <a href="https://www.mdpi.com/search?q=ASTER%20GDEM">ASTER GDEM</a>; <a href="https://www.mdpi.com/search?q=ensemble%20modelling">ensemble modelling</a>; <a href="https://www.mdpi.com/search?q=protected%20area">protected area</a>; <a href="https://www.mdpi.com/search?q=Italian%20Alps">Italian Alps</a>; <a href="https://www.mdpi.com/search?q=biodiversity%20monitoring">biodiversity monitoring</a></span>
</div>
</div>
</div> <div class="hypothesis_container">
<ul class="menu html-nav" data-prev-node="#html-quick-links-title">
</ul>
<div class="html-body">
<section id="sec1-remotesensing-12-00080" type="intro"><h2 data-nested="1"> 1. Introduction</h2><div class="html-p">Mountain
 ecosystems are biodiversity hotspots with higher species richness and 
levels of endemism than adjacent lowlands as a result of steep 
environmental gradients over short distances that lead to topographic, 
geologic, and climatic heterogeneity [<a href="#B1-remotesensing-12-00080" class="html-bibr">1</a>,<a href="#B2-remotesensing-12-00080" class="html-bibr">2</a>].
 Such physiographic complexity creates a mosaic of habitats and, hence, a
 multitude of ecological niches. Furthermore, spatial isolation of 
mountain ridges by environmentally very different lower elevations 
enhances the segregation of populations and, potentially, speciation. 
Mountains also play a key role in providing ecosystem services [<a href="#B3-remotesensing-12-00080" class="html-bibr">3</a>], e.g., they supply half of the global human population with fresh water [<a href="#B4-remotesensing-12-00080" class="html-bibr">4</a>].
 At the same time, mountain regions are especially sensitive to the 
impacts of a changing climate as seen by the shrinking of glaciers, 
changes in water provisioning, and the increase of extreme events such 
as avalanches and landslides [<a href="#B5-remotesensing-12-00080" class="html-bibr">5</a>].</div><div class="html-p">To
 understand the impacts of these changes on plant and animal species 
distributions, long-term biodiversity monitoring initiatives are needed 
to develop adapted conservation measures [<a href="#B6-remotesensing-12-00080" class="html-bibr">6</a>].
 While data collection is a time-consuming endeavor by itself, 
accessibility is an additional major challenge in mountains. However, 
such in situ data are indispensable for predicting species occurrence 
probability at larger scale using, e.g., species distribution models 
(SDMs, see [<a href="#B7-remotesensing-12-00080" class="html-bibr">7</a>]
 for detailed information). SDMs based on species data stemming from 
standardized biodiversity monitoring schemes instead of opportunistic 
observations have been shown to provide higher model accuracy even with a
 lower sample size [<a href="#B8-remotesensing-12-00080" class="html-bibr">8</a>].
 However, not only reliable species presence data, but also geospatial 
information on ecologically relevant environmental factors is needed. 
Since ecological niches in mountains are largely influenced by 
micro-topography [<a href="#B9-remotesensing-12-00080" class="html-bibr">9</a>], geospatial data created by employing spatial interpolation techniques (e.g., for climatic data, [<a href="#B10-remotesensing-12-00080" class="html-bibr">10</a>]) is often not capable of capturing such small-scale differences and heterogeneity.</div><div class="html-p">Besides
 climatic data, which governs species distribution at coarse resolution,
 land use/land cover (LULC) is a main determinant of species 
presence/absence at finer resolutions [<a href="#B11-remotesensing-12-00080" class="html-bibr">11</a>].
 However, existing LULC maps are often not thematically detailed enough 
and the temporal resolution may be inadequate to improve predictions of 
species distributions [<a href="#B12-remotesensing-12-00080" class="html-bibr">12</a>,<a href="#B13-remotesensing-12-00080" class="html-bibr">13</a>,<a href="#B14-remotesensing-12-00080" class="html-bibr">14</a>].
 In this study, we therefore used satellite remote sensing (SRS) data 
offering great potential to mirror spatially explicit habitat 
characteristics for use in SDMs [<a href="#B15-remotesensing-12-00080" class="html-bibr">15</a>].
 Furthermore, SRS data is often the only consistent source of 
information on environmental gradients in space and time allowing also 
retrospective analyses. However, the use of SRS data in mountain regions
 is challenging, e.g., due to high and persistent cloud and snow cover, 
strong illumination effects, and often short vegetation periods (see [<a href="#B16-remotesensing-12-00080" class="html-bibr">16</a>]
 for more detailed information and possible ways forward). Therefore, 
only a limited number of studies so far has employed satellite-derived 
parameters for SDMs in mountainous regions (but see [<a href="#B17-remotesensing-12-00080" class="html-bibr">17</a>,<a href="#B18-remotesensing-12-00080" class="html-bibr">18</a>]).</div><div class="html-p">The
 use of SRS data for assessing taxonomic, structural, and functional 
biodiversity research has generally been increasing in recent years [<a href="#B19-remotesensing-12-00080" class="html-bibr">19</a>],
 driven by a free and open access data policy together with rapid sensor
 developments offering increasing spatial, temporal, and spectral 
resolution of satellite imagery [<a href="#B20-remotesensing-12-00080" class="html-bibr">20</a>].
 Most importantly, the recently available high-resolution and 
high-frequent data of the Sentinel-2 mission allows a much more detailed
 characterization of vegetation phenology [<a href="#B21-remotesensing-12-00080" class="html-bibr">21</a>], improved mapping of fine-scaled habitat types [<a href="#B22-remotesensing-12-00080" class="html-bibr">22</a>], and better assessment of nature conservation status [<a href="#B23-remotesensing-12-00080" class="html-bibr">23</a>].
 However, despite these promising developments, studies employing SRS 
data for analyzing temporal trends in biodiversity face a number of 
limitations in data availability. Due to the availability of a 
consistent and cross-calibrated multi-decadal data record and a 
long-term systematic data acquisition strategy, Landsat satellite 
imagery is a unique monitoring tool for biodiversity assessments over 
time at a high spatial scale [<a href="#B24-remotesensing-12-00080" class="html-bibr">24</a>,<a href="#B25-remotesensing-12-00080" class="html-bibr">25</a>].</div><div class="html-p">The
 objective of this study was to evaluate the potential of Landsat data 
for predicting the presence of different rove beetles (order Coleoptera,
 family Staphylinidae) in the Gran Paradiso National Park (GPNP), a 
mountainous protected area in northwestern Italy. Specifically, we 
analyzed species data from two different field campaigns (2006â€“2007 and 
2012â€“2013) to track changes in time and to test model stability. 
Staphylinids are key organisms in various terrestrial ecosystems and, 
together with carabid beetles, ants, and spiders, constitute the vast 
majority of epigeic mesoarthropods (i.e., invertebrate animals that live
 on the soil surface) [<a href="#B26-remotesensing-12-00080" class="html-bibr">26</a>].
 They are abundant, taxonomically and trophically diverse, occupy a 
variety of ecological niches, and can be easily sampled; all these 
characteristics make them an attractive taxon for use as bioindicators [<a href="#B27-remotesensing-12-00080" class="html-bibr">27</a>,<a href="#B28-remotesensing-12-00080" class="html-bibr">28</a>]. Moreover, many rove beetle species respond to and depend on local-scale environmental parameters and resources [<a href="#B27-remotesensing-12-00080" class="html-bibr">27</a>,<a href="#B28-remotesensing-12-00080" class="html-bibr">28</a>,<a href="#B29-remotesensing-12-00080" class="html-bibr">29</a>],
 representing an important challenge for the application of SRS data. In
 the present study, we focused on three species with fairly different 
habitat requirements. To collate a well-constructed set of predictor 
variables relevant to the biology of the species [<a href="#B30-remotesensing-12-00080" class="html-bibr">30</a>],
 we derived a set of selected topographic and environmental features 
addressing the peculiarities of the mountainous region studied. We used a
 state-of-the-art ensemble modelling framework (i.e., an approach that 
integrated multiple individual model predictions to obtain a final, more
 accurate one; [<a href="#B31-remotesensing-12-00080" class="html-bibr">31</a>]) that combined two regression-based methods (generalized linear models (GLM) and generalized additive models (GAM) [<a href="#B32-remotesensing-12-00080" class="html-bibr">32</a>,<a href="#B33-remotesensing-12-00080" class="html-bibr">33</a>]) and three machine-learning methods (random forest (RF), generalized boosting models (GBM), and maximum entropy (MAXENT) [<a href="#B34-remotesensing-12-00080" class="html-bibr">34</a>,<a href="#B35-remotesensing-12-00080" class="html-bibr">35</a>,<a href="#B36-remotesensing-12-00080" class="html-bibr">36</a>])
 by means of weighted average, an approach suggested to have higher 
accuracy than single-algorithm models and to perform best among 
consensus methods [<a href="#B37-remotesensing-12-00080" class="html-bibr">37</a>]. In particular, we addressed the following questions:</div><div class="html-p"><ul class="html-bullet"><li><div class="html-p">How well can habitat suitability be modelled based on SRS data for rove beetles in mountainous ecosystems?</div></li><li><div class="html-p">How and why do SRS-based model predictions differ among years?</div></li><li><div class="html-p">What are the most important SRS predictors? How do they differ among species?</div></li></ul></div></section><section id="sec2-remotesensing-12-00080" type=""><h2 data-nested="1"> 2. Materials and Methods </h2><section id="sec2dot1-remotesensing-12-00080" type=""><h4 class="html-italic" data-nested="2"> 2.1. Study Area</h4><div class="html-p">The study region encompassed the territory of the Gran Paradiso National Park (GPNP, total area of 847.4 km<sup>2</sup>) located in northwestern Italy (<a href="#remotesensing-12-00080-f001" class="html-fig">Figure 1</a>a).
 GPNP is the oldest national park in Italy that gained official 
conservation status in 1922 and provides habitat for the threatened 
alpine ibex (<span class="html-italic">Capra ibex</span>). The park is 
part of the Long-Term Ecological Research (LTER, site code: 
LTER_EU_IT_109) and of the NATURA 2000 (site code: IT1201000) networks 
and has been subject to intensive long-term monitoring efforts and 
research [<a href="#B38-remotesensing-12-00080" class="html-bibr">38</a>].</div><div class="html-p">GPNP
 is characterized by complex topography with elevation ranging from 670 
to 4061 m above sea level (a.s.l.) including different altitudinal 
vegetation belts (montane, subalpine, alpine, and nival) [<a href="#B39-remotesensing-12-00080" class="html-bibr">39</a>]. Woodlands (20.2% of the territory) are dominated by larch (<span class="html-italic">Larix decidua</span>) and Norway spruce (<span class="html-italic">Picea abies</span>)
 while species-rich alpine grasslands and pastures (17% of the 
territory) can be found above the treeline. About 60% of the territory 
is covered with sparse or no vegetation, bare rocks, or glaciers [<a href="#B38-remotesensing-12-00080" class="html-bibr">38</a>,<a href="#B40-remotesensing-12-00080" class="html-bibr">40</a>].
 The climate is alpine-continental, characterized by low annual mean 
temperatures (ca. 5 Â°C), high seasonal differences, and a general 
paucity of precipitation (990 mm/year on average). However, remarkable 
dissimilarities can be observed among different valleys due to 
variations in altitude, slope, and aspect. Snow cover is usually present
 from Novemberâ€“December to Marchâ€“April [<a href="#B41-remotesensing-12-00080" class="html-bibr">41</a>,<a href="#B42-remotesensing-12-00080" class="html-bibr">42</a>].</div></section><section id="sec2dot2-remotesensing-12-00080" type=""><h4 class="html-italic" data-nested="2"> 2.2. Sampling Design for Biodiversity Data and Species Characteristics</h4><div class="html-p">Species
 occurrence data for this study were collected in the context of the 
long-term monitoring project Monitoring of Animal Biodiversity in 
Mountain Ecosystems, which is an initiative of the GPNP and the first 
attempt to develop a protocol for long-term monitoring of multiple taxa 
in the Italian Alps [<a href="#B39-remotesensing-12-00080" class="html-bibr">39</a>].
 The main purpose of this long-term monitoring initiative is to collect 
information on changes in community composition for multiple taxa [<a href="#B39-remotesensing-12-00080" class="html-bibr">39</a>]
 to allow analyzing biodiversity changes in time and along elevational 
gradients. The first monitoring campaign started in 2006â€“2007 and 
campaigns (carried out over a two-year period) will be continued every 
six years.</div><div class="html-p">The monitoring scheme encompasses 
five representative altitudinal transects to cover three vegetation 
belts (high montane, subalpine, alpine; elevational range 1200â€“2600 m 
a.s.l.; <a href="#remotesensing-12-00080-f001" class="html-fig">Figure 1</a>b).
 Sampling units are plots (circular areas with 100 m radius, between 
five and seven per transect, resulting in a total of 30 plots for GPNP, <a href="#remotesensing-12-00080-f001" class="html-fig">Figure 1</a>c),
 which are separated by an altitudinal difference of approximately 200 m
 to allow for data independence. Pitfall traps in each plot were set up 
along one diameter of the plot with a distance of 50 m from each other (<a href="#remotesensing-12-00080-f001" class="html-fig">Figure 1</a>d).
 Species presence/absence data were collected using pitfall traps 
(plastic cups, diameter of 7 cm, filled with ca. 10 cc of white vinegar 
and some drops of detergent to break surface tension). The traps were 
emptied and refilled every two weeks from May to September. Sampled 
adult specimens were identified at the species level by taxonomic 
experts who registered presence/absence for each taxon.</div><div class="html-p">Here,
 we focus on rove beetles (Coleoptera: Staphylinidae) using data from 
the two monitoring campaigns in 2006â€“2007 and 2012â€“2013. Rove beetles 
have been previously shown to be sensitive to ecosystem modification and
 anthropogenic impacts [<a href="#B43-remotesensing-12-00080" class="html-bibr">43</a>,<a href="#B44-remotesensing-12-00080" class="html-bibr">44</a>].
 Pitfall traps (when being active for a whole season) can be considered 
suitable for obtaining estimates of presence/absence as well as relative
 population density and for quantifying population fluctuations in 
epigeal Coleoptera and allow a standardized and repeatable sampling 
through space and time [<a href="#B45-remotesensing-12-00080" class="html-bibr">45</a>,<a href="#B46-remotesensing-12-00080" class="html-bibr">46</a>]. Among the 178 species present in the database, we here selected three (<span class="html-italic">Dinothenarus fossor</span>, <span class="html-italic">Platydracus stercorarius</span>, and <span class="html-italic">Ocypus ophthalmicus</span>)
 for which a sufficiently high number of sampling locations was 
available, and which were not highly localized within our study area. We
 chose rather common species for which taxonomy, general habitat 
requirements, and ecological traits are reasonably well known [<a href="#B27-remotesensing-12-00080" class="html-bibr">27</a>].
 From those, we focused on species with taxonomic proximity (same 
subfamily and tribe, Staphylininae Staphylinini), but characterized by 
quite different ecological and functional specialization, in particular 
in terms of macro- and micro-habitat affinity and altitudinal range. For
 detailed species characteristics see <a href="#remotesensing-12-00080-box001" class="html-boxed-text">Box 1</a> and description in <a href="#app1-remotesensing-12-00080" class="html-app">Section S1</a> in the <a href="#app1-remotesensing-12-00080" class="html-app">supplementary</a>.</div><div class="html-boxed" id="remotesensing-12-00080-box001">
<div class="html-boxed-title"><div class="html-boxed-title html-p"><b>Box 1.</b> Characteristics of the Staphylinidae species used for species distribution modelling according to Tagliapietra and Zanetti [<a href="#B47-remotesensing-12-00080" class="html-bibr">47</a>] and Zanetti et al. [<a href="#B48-remotesensing-12-00080" class="html-bibr">48</a>]. Source of photographs: Iconographia Coleopterorum Poloniae by Lech Borowiec (<a href="http://baza.biomap.pl/" target="_blank" rel="noopener noreferrer">http://baza.biomap.pl</a>).</div></div>
<div class="html-boxed-body" style=""><div class="html-p"><table class="html-array_table"><tbody><tr><td style="border-top:solid thin;border-left:solid thin;border-bottom:solid thin" class="html-align-center" valign="top" align="center"> </td><td style="border-top:solid thin;border-bottom:solid thin" class="html-align-center" valign="top" align="center"><span class="html-italic">Dinothenarus fossor</span><br><span class="html-fig-inline" id="remotesensing-12-00080-i001"> <img alt="Remotesensing 12 00080 i001" src="remotesensing-12-00080-i001.png"></span></td><td style="border-top:solid thin;border-bottom:solid thin" class="html-align-center" valign="top" align="center"><span class="html-italic">Platydracus stercorarius</span><br><span class="html-fig-inline" id="remotesensing-12-00080-i002"> <img alt="Remotesensing 12 00080 i002" src="remotesensing-12-00080-i002.png"></span></td><td style="border-top:solid thin;border-right:solid thin;border-bottom:solid thin" class="html-align-center" valign="top" align="center"><span class="html-italic">Ocypus ophthalmicus</span><br><span class="html-fig-inline" id="remotesensing-12-00080-i003"> <img alt="Remotesensing 12 00080 i003" src="remotesensing-12-00080-i003.png"></span></td></tr><tr><td style="border-left:solid thin;border-bottom:solid thin" class="html-align-center" valign="middle" align="center">Size</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">15â€“20 mm</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">12â€“15 mm</td><td style="border-bottom:solid thin;border-right:solid thin" class="html-align-center" valign="middle" align="center">15â€“23 mm</td></tr><tr><td style="border-left:solid thin;border-bottom:solid thin" class="html-align-center" valign="middle" align="center">Macro-habitat</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">Sylvicolous (mainly)</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">Eurytopic</td><td style="border-bottom:solid thin;border-right:solid thin" class="html-align-center" valign="middle" align="center">Often in open areas (eurytopic)</td></tr><tr><td style="border-left:solid thin;border-bottom:solid thin" class="html-align-center" valign="middle" align="center">Micro-habitat</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">Phytodetriticolous (mainly)</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">Saprophilous (often close to dung)</td><td style="border-bottom:solid thin;border-right:solid thin" class="html-align-center" valign="middle" align="center">Phytodetriticolous</td></tr><tr><td style="border-left:solid thin;border-bottom:solid thin" class="html-align-center" valign="middle" align="center">Nutrition</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">Predator<br>(mainly myrmecophagous)</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">Generalist predator (other insects)</td><td style="border-bottom:solid thin;border-right:solid thin" class="html-align-center" valign="middle" align="center">Generalist predator<br>(other insects, larvae, nematodes, snails)</td></tr><tr><td style="border-left:solid thin;border-bottom:solid thin" class="html-align-center" valign="middle" align="center">Micro-climate</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">Xerophilus</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">NA</td><td style="border-bottom:solid thin;border-right:solid thin" class="html-align-center" valign="middle" align="center">Xerophilus, heliophilous</td></tr><tr><td style="border-left:solid thin;border-bottom:solid thin" class="html-align-center" valign="middle" align="center">Altitude</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">Montane (mainly)</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">Lowland to montane</td><td style="border-bottom:solid thin;border-right:solid thin" class="html-align-center" valign="middle" align="center">Lowland to subalpine-alpine</td></tr><tr><td style="border-left:solid thin;border-bottom:solid thin" class="html-align-center" valign="middle" align="center">Chorotype</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">Central European</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">Turano-European</td><td style="border-bottom:solid thin;border-right:solid thin" class="html-align-center" valign="middle" align="center">Palearctic</td></tr></tbody></table></div>
</div>
</div></section><section id="sec2dot3-remotesensing-12-00080" type=""><h4 class="html-italic" data-nested="2"> 2.3. Species Presence/Pseudo-Absence Data for Model Training </h4><div class="html-p">To
 train the SDMs, we used the environmental conditions at the trap 
locations in which individuals of each target species were captured 
(true presence data), separately for each species. The number of traps 
and plots in which individuals of each species were captured varied 
among species and years (<a href="#remotesensing-12-00080-t001" class="html-table">Table 1</a>).
 Furthermore, a few samples were excluded from the analysis as located 
in areas covered by clouds and snow in the analyzed SRS data. For all 
species and both monitoring campaigns, we pooled the data collected in 
2006â€“2007 and 2012â€“2013, respectively, thereby increasing the number of 
traps and plots in which individuals were captured (<a href="#remotesensing-12-00080-t001" class="html-table">Table 1</a>).
 The use of data from consecutive years is a common approach in species 
distribution modelling to account for inter-annual differences in 
species abundance and occurrence [<a href="#B49-remotesensing-12-00080" class="html-bibr">49</a>].
 We initially used all traps where the study species were captured as 
presence points for fitting the models, but, due to the close proximity 
of traps within plots, then decided to model each species on plot-pooled
 data by subsampling only one trap per plot as presence point. For each 
plot, we selected the trap with the highest number of captured 
individuals per species, thus ensuring that the site was recurrently 
frequented by the study species. Subsampling is an effective way of 
removing pseudo-replicates (i.e., spatially correlated observations) in 
the dataset. Although this approach further reduced the number of 
presences available for modelling, previous studies [<a href="#B50-remotesensing-12-00080" class="html-bibr">50</a>,<a href="#B51-remotesensing-12-00080" class="html-bibr">51</a>] showed that accurate SDMs can be fitted on very small numbers of training records.</div><div class="html-p">Due
 to the limited size of our dataset, we decided not to use traps without
 captured individuals as absence information but to generate 
pseudo-absence data. Pseudo-absences are commonly used in distribution 
modelling [<a href="#B52-remotesensing-12-00080" class="html-bibr">52</a>],
 and they should not be interpreted as sites at which the species was 
truly absent, but rather as a sample of the environmental variation in 
the study area [<a href="#B53-remotesensing-12-00080" class="html-bibr">53</a>,<a href="#B54-remotesensing-12-00080" class="html-bibr">54</a>]. Moreover, this approach allowed us to use the true absences as an independent dataset for subsequent model evaluation (see <a href="#sec2dot7-remotesensing-12-00080" class="html-sec">Section 2.7</a>).
 Specifically, we created 500 stratified random point locations as 
pseudo-absence data, which were allocated according to the sampling 
design in terms of trap density at specific altitudinal ranges (<a href="#remotesensing-12-00080-f001" class="html-fig">Figure 1</a>c;
 e.g., 6.7% of the traps were located at 1300 m a.s.l. thus 33 out of 
500 random points were assigned to the range 1300 Â± 50 m). Additional 
constraints for the locations of these stratified random points were a 
minimum distance of 30 m between points to match the spatial scale of 
the analyzed satellite remote sensing data (see <a href="#sec2dot4-remotesensing-12-00080" class="html-sec">Section 2.4</a>.) and the non-existence of points within a buffer area with 500 m diameter around trap locations (for detailed information see <a href="#app1-remotesensing-12-00080" class="html-app">Figure S1</a>).
 This buffer acknowledged the necessity to use reasonable 
pseudo-absences that are adequately environmentally distant from the 
presence data (but not outside the area of interest) in order to avoid 
inflation of AUC (area under curve of the receiver operating 
characteristics) values [<a href="#B55-remotesensing-12-00080" class="html-bibr">55</a>]. Areas covered by clouds and snow were excluded for allocation of pseudo-absence data.</div></section><section id="sec2dot4-remotesensing-12-00080" type=""><h4 class="html-italic" data-nested="2"> 2.4. Satellite Remote Sensing Data</h4><div class="html-p">We
 used multispectral Landsat surface reflectance data provided by the 
Earth Resources Observation and Science Centre Processing Architecture [<a href="#B56-remotesensing-12-00080" class="html-bibr">56</a>].
 Landsat imagery has proven suitable to derive geospatial information on
 environmental characteristics that affect species distributions in 
multiple studies [<a href="#B18-remotesensing-12-00080" class="html-bibr">18</a>,<a href="#B57-remotesensing-12-00080" class="html-bibr">57</a>] and has been used to model changes in biodiversity between different time periods [<a href="#B58-remotesensing-12-00080" class="html-bibr">58</a>].
 Two Landsat images acquired on 22 September 2006 (Landsat 5-TM) and 9 
September 2013 (Landsat 8-OLI) that were consistent with the time period
 of in situ species data sampling (between May and September) were 
downloaded. These images were selected to minimize overall cloud cover 
over the study area (41.6% in 2006 and 25.3% in 2013) and differences 
between acquisition dates. We topographically corrected the images and 
calculated the tasseled cap transformation to derive â€œbrightnessâ€, 
â€œgreennessâ€, and â€œwetnessâ€ using sensor-specific coefficients [<a href="#B59-remotesensing-12-00080" class="html-bibr">59</a>,<a href="#B60-remotesensing-12-00080" class="html-bibr">60</a>]. Additionally, we derived the Normalized Difference Vegetation Index (NDVI, [<a href="#B61-remotesensing-12-00080" class="html-bibr">61</a>]) and downloaded the NDVI-based land surface temperature (LST) for each acquisition date [<a href="#B62-remotesensing-12-00080" class="html-bibr">62</a>].</div><div class="html-p">In
 order to represent altitudinal and topographic gradients, the ASTER 
Global Digital Elevation Model (GDEM, cell size: 30 Ã— 30 m, [<a href="#B63-remotesensing-12-00080" class="html-bibr">63</a>])
 was aligned to match pixel locations of the Landsat imagery. While the 
use of altitude information has been subject to debate in ecology in 
general [<a href="#B1-remotesensing-12-00080" class="html-bibr">1</a>] and in species distribution modelling in particular [<a href="#B9-remotesensing-12-00080" class="html-bibr">9</a>,<a href="#B64-remotesensing-12-00080" class="html-bibr">64</a>],
 we here used altitude as a proxy for climatic parameters since 
spatially explicit estimates of climate were not available at 
sufficiently high spatial resolution. In addition, based on the ASTER 
GDEM, the Terrain Ruggedness Index (TRI, based on 8 adjacent cells 
within a quadratic array around the center cell [<a href="#B65-remotesensing-12-00080" class="html-bibr">65</a>]),
 sine of aspect (eastness), cosine of aspect (northness), and slope were
 calculated. Areas covered by clouds, cloud shadows, and snow were 
masked out (<a href="#app1-remotesensing-12-00080" class="html-app">Figure S2</a>).
 To avoid collinearity of explanatory variables in the subsequent 
analyses, a threshold of |r| &gt; 0.7 was used to exclude variables [<a href="#B66-remotesensing-12-00080" class="html-bibr">66</a>].
 As a result, NDVI was not used in the modelling framework due to its 
strong correlation with altitude. Likewise, brightness and land surface 
temperature were also excluded as they were highly correlated with 
northness and wetness (<a href="#app1-remotesensing-12-00080" class="html-app">Figure S3</a>). An overview of the SRS variables used and their ecological relevance is given in <a href="#remotesensing-12-00080-t002" class="html-table">Table 2</a>.</div></section><section id="sec2dot5-remotesensing-12-00080" type=""><h4 class="html-italic" data-nested="2"> 2.5. Ensemble Modelling and Model Parametrization</h4><div class="html-p">Since the choice of model algorithm is a major component of prediction uncertainty in SDMs [<a href="#B37-remotesensing-12-00080" class="html-bibr">37</a>,<a href="#B75-remotesensing-12-00080" class="html-bibr">75</a>],
 several algorithms were employed and combined in order to estimate and 
account for prediction uncertainty (ensemble modelling, [<a href="#B31-remotesensing-12-00080" class="html-bibr">31</a>]).
 We used five modelling methods that are representative for different 
classes of model algorithms, namely generalized linear models and 
generalized additive models (GLM and GAM, regression-based methods, [<a href="#B32-remotesensing-12-00080" class="html-bibr">32</a>,<a href="#B33-remotesensing-12-00080" class="html-bibr">33</a>]), random forest, generalized boosting models, and maximum entropy (RF, GBM, and MAXENT; machine-learning methods; [<a href="#B34-remotesensing-12-00080" class="html-bibr">34</a>,<a href="#B35-remotesensing-12-00080" class="html-bibr">35</a>,<a href="#B36-remotesensing-12-00080" class="html-bibr">36</a>]) as implemented in the biomod2 package version 3.3 [<a href="#B76-remotesensing-12-00080" class="html-bibr">76</a>] using R version 3.6.0 [<a href="#B77-remotesensing-12-00080" class="html-bibr">77</a>].
 To account for the within-algorithm model variation when different sets
 of data were used for model fitting, we computed distribution models 
for each species using 50 repetitions where 70% of the data were used 
for model training and 30% for model testing. To avoid spatial sorting 
bias in the cross-validation procedure [<a href="#B78-remotesensing-12-00080" class="html-bibr">78</a>],
 we used a spatially stratified approach to split the training and the 
testing data sets, to prevent selecting testing-presence sites close to 
training-presence sites. Specifically, the full dataset was divided into
 50 clusters (one for each model repetition), using k-means clustering 
based on the spatial coordinates of presence/pseudo-absence points. For 
each model run, 30% of the points closest to the center coordinates of 
the respective cluster were used as testing dataset and the remaining as
 training dataset. Model outputs were scaled between 0 and 1000 to 
ensure comparability of model predictions across the different model 
algorithms. A binomial link function was used to fit GLMs, allowing 
linear and quadratic terms as well as linear interactions of the 
explanatory variables. To simplify the full model (i.e., to reduce the 
number of environmental predictors by removing less important ones), a 
stepwise backward selection was applied to select final models, based on
 the Bayesian Information Criterion (BIC) [<a href="#B79-remotesensing-12-00080" class="html-bibr">79</a>].
 GAMs were also fitted with a binomial link function, including all 
seven explanatory variables and using the algorithm as implemented in 
the mgcv package [<a href="#B80-remotesensing-12-00080" class="html-bibr">80</a>].
 The basis dimension of the smooth functions was set to k = 4 to avoid 
overfitting. For MAXENT, the maximum number of iterations was set to 500
 and the â€˜Auto featureâ€™ settings were used while excluding threshold and
 hinge features to avoid over-parameterization [<a href="#B81-remotesensing-12-00080" class="html-bibr">81</a>]. The regularization parameter Î² was set to 0.002 to optimize model performance [<a href="#B82-remotesensing-12-00080" class="html-bibr">82</a>]. For RF models we used 500 trees, following previous studies [<a href="#B75-remotesensing-12-00080" class="html-bibr">75</a>],
 while for GBMs we used the default settings (number of trees = 100, 
interaction depth parameter = 1, learning rate = 0.1, and subsampling 
fraction = 0.5) suggested by the gbm package [<a href="#B83-remotesensing-12-00080" class="html-bibr">83</a>].
 The AUC was used as evaluation criteria to generate the ensemble 
predictions from the five different algorithms (see biomod2 
documentation for further details). To obtain a relevant combination of 
several unbiased (i.e., with fair accuracy) models, all individual 
models with an AUC value &lt;0.7 were discarded, and the ensemble models
 were constructed for each species and time period by computing the 
weighted average of all remaining model predictions [<a href="#B52-remotesensing-12-00080" class="html-bibr">52</a>].
 The weights were based on the AUC scores of each model, so that better 
performing models had a higher influence in the final ensemble. This 
technique has been shown to be one of the best performing consensus 
methods [<a href="#B37-remotesensing-12-00080" class="html-bibr">37</a>] and is widely used in ensemble applications [<a href="#B52-remotesensing-12-00080" class="html-bibr">52</a>].
 To assess shifts in suitable habitat between the two time periods, we 
defined five classes of habitat suitability based on the presence 
probability scores extracted from the SDM maps. The five classes were 
defined by the following thresholds of presence probability scores: 
0â€“200, &gt;200â€“400, &gt;400â€“600, &gt;600â€“800, and &gt;800â€“1000, ordered 
from low to high habitat suitability. To compare changes over time, we 
calculated the percentage of area in each class for the common area not 
masked by cloud, shadow, and snow coverage in either of the two time 
periods. Moreover, we calculated Spearman rank correlation coefficients 
between the SDMs, to assess the correlation of species distributions 
between time periods and across different species.</div></section><section id="sec2dot6-remotesensing-12-00080" type=""><h4 class="html-italic" data-nested="2"> 2.6. Assessing Variable Importance of SRS Data</h4><div class="html-p">Variable
 importance was assessed by comparing the model prediction derived from 
the original dataset and predictions derived from permuted datasets [<a href="#B76-remotesensing-12-00080" class="html-bibr">76</a>,<a href="#B84-remotesensing-12-00080" class="html-bibr">84</a>].
 Permuted datasets were created by randomizing one environmental 
variable resulting in one dataset for each predictor variable. The 
predictions of occurrence probabilities resulting from permuted datasets
 and the original dataset were compared by calculating the Pearson 
correlation coefficient; a small correlation coefficient thus indicated 
high importance of the permuted variable. This procedure was repeated 
three times for each model run, and mean correlation values for each 
predictor variable were derived across the 50 model repetitions. Values 
of variable importance were finally calculated by subtracting the 
correlation coefficient from 1. To ensure comparability between 
different model algorithms, the relative variable importance was 
calculated for every variable per algorithm. To understand the effect of
 single variables on the SDM predictions, response curves showing the 
sensibility of the model to each variable were plotted for all variables
 in the ensemble models [<a href="#B76-remotesensing-12-00080" class="html-bibr">76</a>].</div></section><section id="sec2dot7-remotesensing-12-00080" type=""><h4 class="html-italic" data-nested="2"> 2.7. Model Evaluation</h4><div class="html-p">We
 used a cross-validation procedure to assess model performance and 
calculated two evaluation metrics commonly used in species distribution 
modelling, AUC and the true skills statistics (TSS) [<a href="#B54-remotesensing-12-00080" class="html-bibr">54</a>]. AUC values for SDMs indicate how well the model discriminated between species presence and absence/pseudo-absence data [<a href="#B85-remotesensing-12-00080" class="html-bibr">85</a>]. According to Swets [<a href="#B86-remotesensing-12-00080" class="html-bibr">86</a>],
 models with AUC values below 0.7 are considered as poorly accurate, 
those with AUC between 0.7 and 0.9 as useful, and those with values 
above 0.9 as highly accurate. TSS is a threshold-dependent metric, not 
affected by the size of the validation set nor by species prevalence [<a href="#B54-remotesensing-12-00080" class="html-bibr">54</a>,<a href="#B87-remotesensing-12-00080" class="html-bibr">87</a>].
 TSS is calculated as sensitivity (i.e., true positive rate) + 
specificity (i.e., true negative rate) âˆ’1, and therefore ranges from âˆ’1 
to +1, where values of zero or less indicate a performance no better 
than random guessing [<a href="#B54-remotesensing-12-00080" class="html-bibr">54</a>]. According to GonzÃ¡lez-Ferreras et al. [<a href="#B88-remotesensing-12-00080" class="html-bibr">88</a>],
 model accuracy is poor when TSS values are below 0.4, fair when TSS is 
between 0.4 and 0.6, good when TSS is between 0.6 and 0.8, and excellent
 for values above 0.8. Mean AUC and TSS scores were calculated based on 
the 50 model repetitions for all model algorithms and for the ensemble 
models. In addition to AUC and TSS, we analyzed and compared model 
predictions at presence locations and at independent true absence data 
(i.e., traps in which no individuals of each target species were 
captured) for each species to examine how well derived maps represented 
in situ data. Finally, as uncertainty maps are an important tool for 
communicating model reliability in a spatially explicit way [<a href="#B89-remotesensing-12-00080" class="html-bibr">89</a>],
 we calculated the difference between the 97.5th percentile and the 
2.5th percentile of cell values derived from the prediction maps for 
each species, based on 50 model repetitions.</div></section></section><section id="sec3-remotesensing-12-00080" type="results"><h2 data-nested="1"> 3. Results</h2><section id="sec3dot1-remotesensing-12-00080" type=""><h4 class="html-italic" data-nested="2"> 3.1. Model Performance</h4><div class="html-p">AUC
 and TSS values were highly concordant in assessing model performance, 
and the majority of species prediction models showed satisfactory 
accuracy levels based on the two evaluation metrics (i.e., AUC &gt; 0.7 
and TSS &gt; 0.4, <a href="#remotesensing-12-00080-t003" class="html-table">Table 3</a>), and can thus be categorized as useful [<a href="#B86-remotesensing-12-00080" class="html-bibr">86</a>]. <span class="html-italic">Dinothenarus fossor</span>
 showed the highest number of well-performing models, probably because 
of the larger number of presence points in both time periods, compared 
to the other two species (<a href="#remotesensing-12-00080-t001" class="html-table">Table 1</a>). On the contrary, for <span class="html-italic">Platydracus stercorarius,</span> only 5 out of 12 models could be classified as useful, based on the evaluation metrics scores. In the case of <span class="html-italic">Ocypus ophthalmicus</span>,
 5 out of the 6 model types built for the time period 2006â€“2007 
performed well, whereas the number decreased to 4 out of 6 for the 
2012â€“2013. Model performance varied significantly across model 
algorithms, with the machine-learning algorithms GBM, RF, and MAXENT 
producing the highest AUC and TSS scores. In contrast, GAM and GLM 
models performed poorly. Moreover, the evaluation metrics values of GLM 
models showed strong variation for different model repetitions, 
reflecting the instability of the models. The ensemble models always 
outperformed all other model types, returning accurate models for all 
species in both time periods. Contrasting patterns were found when 
comparing the different time periods 2006â€“2007 and 2012â€“2013: For <span class="html-italic">Dinothenarus fossor</span>, we observed higher evaluation metrics scores for the models referring to 2012â€“2013 across all algorithms. In contrast, for <span class="html-italic">Ocypus ophthalmicus</span> model performance was higher for 2006â€“2007. No clear pattern was found for <span class="html-italic">Platydracus stercorarius</span> when comparing model accuracy between time periods.</div><div class="html-p">In
 addition to the cross-validation results, the comparison between 
modelled presence probabilities at presence locations and at absence 
records not used for model training revealed that the majority of 
ensemble models had fair discriminatory power in the prediction of 
suitable vs. unsuitable habitat. For <span class="html-italic">Dinothenarus fossor</span>,
 the modelled presence probabilities were significantly higher in the 
ensemble predictions at presence sites compared to absence sites, in 
both time periods (<a href="#remotesensing-12-00080-f002" class="html-fig">Figure 2</a>).
 However, for the other two species, the ensemble models built for 
2006â€“2007 had higher discriminatory power than those built for 
2012â€“2013.</div></section><section id="sec3dot2-remotesensing-12-00080" type=""><h4 class="html-italic" data-nested="2"> 3.2. Effects of Inter-Annual Variability of Species Records on SRS-Based Training Data</h4><div class="html-p">The inter-annual variability of the plots in which individuals of the three species were captured (see <a href="#sec2dot3-remotesensing-12-00080" class="html-sec">Section 2.3</a>) together with the inter-annual differences in the SRS data themselves (see <a href="#sec2dot4-remotesensing-12-00080" class="html-sec">Section 2.4</a>) had clear effects on the values and range of the SRS-based model training data (<a href="#app1-remotesensing-12-00080" class="html-app">Figure S4</a>).
 Variability was generally lower for the variables derived from the 
ASTER GDEM (altitude, eastness, northness, slope, and ruggedness), which
 were static data for both time periods. On the contrary, the largest 
differences were found for greenness and wetness, due to variations in 
ecosystem conditions and seasonal changes as a result of the differences
 in acquisition dates (the 2006 imagery was acquired 20 days later than 
the 2013 data). However, variability cannot solely be attributed to 
differences between the two Landsat data sets. For example, <span class="html-italic">Ocypus ophthalmicus</span>
 showed considerable variation also for the ASTER GDEM-based variables 
between 2006â€“2007 and 2012â€“2013. Similar differences can be seen for <span class="html-italic">Dinothenarus fossor</span>
 with respect to eastness. Overall, the subsampling of the full dataset 
by retaining only one trap per plot did not significantly reduce the 
variance and range of the environmental variables, apart from few 
exceptions (<a href="#app1-remotesensing-12-00080" class="html-app">Figure S4</a>).</div></section><section id="sec3dot3-remotesensing-12-00080" type=""><h4 class="html-italic" data-nested="2"> 3.3. Variable Importance</h4><div class="html-p">Across
 species, model algorithms, and years, greenness generally had the 
highest mean variable importance according to correlation analyses 
between model predictions based on permuted and original datasets, 
followed by ruggedness (<a href="#remotesensing-12-00080-t004" class="html-table">Table 4</a>).
 This trend was evident also in the ensemble models, for which greenness
 and ruggedness were always the most important variables, with the only 
exception being the model built for <span class="html-italic">Platydracus stercorarius</span>
 in 2006â€“2007, in which wetness was the second most important variable 
after greenness. Slope was of only very minor importance in all models. 
However, slight differences in variable importance between species and 
years could be detected. For example, topographic variables like 
ruggedness and northness were more important for <span class="html-italic">Ocypus ophthalmicus</span> than for the other two species, whereas greenness was less important than for <span class="html-italic">Dinothenarus fossor</span> and <span class="html-italic">Platydracus stercorarius</span>. Wetness was found to be an important variable in the ensemble model built for <span class="html-italic">Platydracus stercorarius</span>
 for the time period 2006â€“2007, whereas it played only a minor role in 
all other models. The most consistent pattern across the two time 
periods considered was found for <span class="html-italic">Dinothenarus fossor</span>, whereas <span class="html-italic">Ocypus ophthalmicus</span>
 showed the highest variations across model algorithms and between time 
periods. However, values of importance scores differed, being generally 
higher for MAXENT and GBM for variables identified as highly important 
than in the other algorithms. On the other hand, GLMs had the smallest 
range of variable importance values across variables. The response 
curves of modelled presence probability relative to each environmental 
variable showed a strong and positive effect of greenness and a negative
 effect of ruggedness on species presence for all three species (<a href="#app1-remotesensing-12-00080" class="html-app">Figure S5</a>). High wetness values decreased the probability of occurrence of <span class="html-italic">Platydracus stercorarius</span> in the model built for 2006â€“2007, and a similar but milder effect could be seen for <span class="html-italic">Dinothenarus fossor</span>
 in 2012â€“2013. Northness, eastness, and slope had negligible effects on 
species presence in all six ensemble models. In addition, altitude had 
weak effects on the SDMs, although some slight differences in 
altitudinal range between species could be observed in the response 
curves. Specifically, the models built for <span class="html-italic">Platydracus stercorarius</span> and <span class="html-italic">Ocypus ophthalmicus</span>
 showed a slight increase of predicted presence probability at higher 
altitudes (above 2000 m), whereas the trend was reversed in the 
2006â€“2007 <span class="html-italic">Dinothenarus fossor</span>Â´s model.</div></section><section id="sec3dot4-remotesensing-12-00080" type=""><h4 class="html-italic" data-nested="2"> 3.4. Species Distribution Maps</h4><div class="html-p">The
 area of GPNP for which species distributions could be modelled was 
larger in 2012â€“2013 due to a smaller proportion of areas covered by 
clouds, cloud shadows, and snow in the respective Landsat data set. 
While the regions predicted as potentially suitable varied to a 
different extent between the species, valleys generally had higher 
suitability scores than hillsides or high-altitude plateaus (<a href="#remotesensing-12-00080-f003" class="html-fig">Figure 3</a>).
 According to our results, the southern and eastern parts of the study 
area were generally more suitable compared to the remaining regions. 
While the six SDMs had high correlation coefficients (<a href="#app1-remotesensing-12-00080" class="html-app">Table S1</a>),
 indicating large similarities in the modelled distributions of the 
three species, we also observed some differences: Ensemble models built 
for <span class="html-italic">Ocypus ophthalmicus</span> resulted in a 
higher predicted occurrence probability in the northeastern part of the 
study area compared to the other species. The southwestern valleys in 
the GPNP were also modelled as highly suitable for <span class="html-italic">Ocypus ophthalmicus</span>, whereas their modelled presence probability was intermediate for <span class="html-italic">Platydracus stercorarius</span> and low for <span class="html-italic">Dinothenarus fossor</span> (<a href="#remotesensing-12-00080-f003" class="html-fig">Figure 3</a>).</div><div class="html-p">Visual comparison of the final SDM maps revealed that modelled distribution patterns across time periods were more similar for <span class="html-italic">Ocypus ophthalmicus</span> than for <span class="html-italic">Dinothenarus fossor</span> and <span class="html-italic">Platydracus stercorarius</span>, which is supported by higher correlation values between the correspondent SDM maps (<a href="#app1-remotesensing-12-00080" class="html-app">Table S1</a>).
 High habitat suitability scores (i.e., above 600) were only obtained 
for a small proportion of GPNP across all species (below 20% of the 
study area, <a href="#app1-remotesensing-12-00080" class="html-app">Table S2</a>). For <span class="html-italic">Dinothenarus fossor</span>, the area with high suitability scores (&gt;600 to 1000) decreased from 2006â€“2007 to 2012â€“2013 (16.4% vs. 8.3%, <a href="#app1-remotesensing-12-00080" class="html-app">Table S2</a>). Contrarily, the highly suitable area increased from 8.9% to 16.4% for <span class="html-italic">Platydracus stercorarius</span> and from 11.3% to 19.1% for <span class="html-italic">Ocypus ophthalmicus</span>
 between the two time periods. Model uncertainty was generally higher in
 areas with medium to high presence probability and low in areas with 
lowest predicted suitability (see <a href="#remotesensing-12-00080-f003" class="html-fig">Figure 3</a> and <a href="#app1-remotesensing-12-00080" class="html-app">Figure S6</a>), showing a similar spatial pattern for the three species. When comparing time periods, overall uncertainty was higher for <span class="html-italic">Dinothenarus fossor</span> and <span class="html-italic">Platydracus stercorarius</span> in the 2012â€“2013 models, whereas the trend was opposite for <span class="html-italic">Ocypus ophthalmicus</span>.</div></section></section><section id="sec4-remotesensing-12-00080" type="discussion"><h2 data-nested="1"> 4. Discussion</h2><div class="html-p">The
 objective of this study was to examine the potential of SRS data to 
produce species distributions maps of rove beetles with different 
ecological and functional specialization. A large set of studies 
predicting species distribution of invertebrates relies on habitat 
parameters recorded in the field as well as bioclimatic variables [<a href="#B90-remotesensing-12-00080" class="html-bibr">90</a>,<a href="#B91-remotesensing-12-00080" class="html-bibr">91</a>,<a href="#B92-remotesensing-12-00080" class="html-bibr">92</a>]. Others also include SRS-derived information on land use/land cover [<a href="#B93-remotesensing-12-00080" class="html-bibr">93</a>,<a href="#B94-remotesensing-12-00080" class="html-bibr">94</a>,<a href="#B95-remotesensing-12-00080" class="html-bibr">95</a>,<a href="#B96-remotesensing-12-00080" class="html-bibr">96</a>] or variables related to topography (e.g., slope, northness, and eastness, [<a href="#B97-remotesensing-12-00080" class="html-bibr">97</a>]).
 However, to our knowledge the present study is the first attempt to 
predict species distribution of beetles in a mountainous region entirely
 based on SRS data.</div><section id="sec4dot1-remotesensing-12-00080" type=""><h4 class="html-italic" data-nested="2"> 4.1. Model Performances to Predict Invertebrate Species Distribution </h4><div class="html-p">The presented SDMs can be rated as useful according to Swets [<a href="#B86-remotesensing-12-00080" class="html-bibr">86</a>] and GonzÃ¡lez-Ferreras et al. [<a href="#B88-remotesensing-12-00080" class="html-bibr">88</a>]
 based on mean AUC and TSS values obtained in the cross-validation step.
 When comparing different algorithms, regression-based methods (GLM and 
GAM) had the lowest mean performance, whereas machine-learning methods 
(RF, GBM, and MAXENT) were found to be the most accurate. GLMs have been
 widely used in SDM studies [<a href="#B53-remotesensing-12-00080" class="html-bibr">53</a>],
 but their inability to capture nonlinear responses might be one reason 
for their poor performance in our study. Otherwise, our results 
confirmed the robustness and effectiveness of more recently developed 
machine-learning techniques: Elith et al. [<a href="#B53-remotesensing-12-00080" class="html-bibr">53</a>]
 showed better performances of GBM and MAXENT compared to GAM and GLM 
across different species (plants, birds, mammals, and reptiles) and 
regions; similarly, Marmion et al. [<a href="#B37-remotesensing-12-00080" class="html-bibr">37</a>] and Grenouillet et al. [<a href="#B98-remotesensing-12-00080" class="html-bibr">98</a>]
 found RF to have the highest accuracy among several tested single 
algorithm models (among which GLM and GAM) in predicting plant and fish 
species distributions, respectively. AraÃºjo and New [<a href="#B31-remotesensing-12-00080" class="html-bibr">31</a>]
 list RF, GBM, and MAXENT as techniques that already incorporate the 
notion of ensemble forecasting, which could be one of the underlying 
factors explaining their higher predictive accuracy over regression 
methods. While the performance of all model algorithms was negatively 
affected by the small number of presence points used for model fitting, 
the selection method of pseudo-absences impacted regression and 
machine-learning techniques differently: Wisz and Guisan [<a href="#B99-remotesensing-12-00080" class="html-bibr">99</a>] and Barbet-Massin et al. [<a href="#B100-remotesensing-12-00080" class="html-bibr">100</a>]
 showed that a large number (e.g., 10,000) of randomly selected 
pseudo-absences improved the performance of GLMs and GAMs, hence our 
spatially stratified selection approach may have influenced the 
performance of these model types.</div><div class="html-p">Overall, the 
variability among the five algorithms confirmed the suitability of the 
chosen ensemble modelling approach, and the evaluation metrics values 
identified the ensembles as the highest performing models compared to 
single algorithm models, as also shown by Marmion et al. [<a href="#B37-remotesensing-12-00080" class="html-bibr">37</a>] and Grenouillet et al. [<a href="#B98-remotesensing-12-00080" class="html-bibr">98</a>].
 Nonetheless, the comparison of modelled presence probability scores at 
true absence and presence points revealed that not all ensemble models 
had high discriminatory power (<a href="#remotesensing-12-00080-f002" class="html-fig">Figure 2</a>). In particular, the models built for <span class="html-italic">Platydracus stercorarius</span> and <span class="html-italic">Ocypus ophthalmicus</span>
 in 2012â€“2013 tended to overestimate distribution ranges, predicting 
high occurrence probability also at absence locations. This could be due
 to the small number of presence points available for model training (18
 and 12, respectively, <a href="#remotesensing-12-00080-t001" class="html-table">Table 1</a>).
 In fact, models trained using all traps where beetles were captured as 
presence points (instead of plot-pooled data) had a consistently higher 
accuracy in discriminating presences from true absence sites. Moreover, 
distributions of widespread species, such as <span class="html-italic">Platydracus stercorarius, Ocypus ophthalmicus</span> (see description in <a href="#app1-remotesensing-12-00080" class="html-app">Section S1</a>)<span class="html-italic">,</span> were more difficult to predict compared to those of rare species [<a href="#B50-remotesensing-12-00080" class="html-bibr">50</a>,<a href="#B51-remotesensing-12-00080" class="html-bibr">51</a>].
 Nevertheless, the obtained species distribution maps can be considered 
as quite reliable based on local ecological knowledge, which is becoming
 increasingly important in ecological modelling in general [<a href="#B101-remotesensing-12-00080" class="html-bibr">101</a>,<a href="#B102-remotesensing-12-00080" class="html-bibr">102</a>]. In particular, the SDMs met our expectations of more similar distribution patterns for <span class="html-italic">Platydracus stercorarius</span> and <span class="html-italic">Dinothenarus fossor</span>, which are both macro-habitat generalists and strongly dependent on vegetated areas, compared to the predicted distribution of <span class="html-italic">Ocypus ophthalmicus</span>, which is less dependent on vegetation (see description in <a href="#app1-remotesensing-12-00080" class="html-app">Section S1</a>).</div><div class="html-p">Regarding
 the spatial resolution of the employed Landsat images and GDEM (30 Ã— 30
 m), we assumed adequacy to predict species distribution of 
invertebrates as other studies employing SRS data were successfully 
conducted on an even broader spatial scale so far [<a href="#B93-remotesensing-12-00080" class="html-bibr">93</a>,<a href="#B94-remotesensing-12-00080" class="html-bibr">94</a>] (but see <a href="#sec4dot4-remotesensing-12-00080" class="html-sec">Section 4.4</a>). Moreover, despite general recognition of the importance of scale selection in SDMs [<a href="#B103-remotesensing-12-00080" class="html-bibr">103</a>], studies addressing scale directly have found equivocal results: Guisan et al. [<a href="#B104-remotesensing-12-00080" class="html-bibr">104</a>]
 showed that a 10 times lower resolution of environmental layers used in
 modelling did not severely affect predictions from SDMs, despite a 
general decrease in model performance when using explanatory variables 
at coarser grain size. Nevertheless, as ecological processes are also 
highly dependent on scale, more research in this direction is needed to 
better understand the complex relationships between the temporal and 
spatial variability of species presence records and representation of 
this variability in SRS data.</div></section><section id="sec4dot2-remotesensing-12-00080" type=""><h4 class="html-italic" data-nested="2"> 4.2. Temporal Variability of Species Presence Records, SRS Data, and Modelled Distribution Ranges</h4><div class="html-p">Inter-annual
 fluctuations in the number of epigeic beetles are a quite common 
phenomenon found in many works while not completely understood. Indeed, 
the number of specimens caught by pitfall traps, and consequently the 
detectability of the species, are linked not only to animal abundance 
but also to their activity [<a href="#B105-remotesensing-12-00080" class="html-bibr">105</a>,<a href="#B106-remotesensing-12-00080" class="html-bibr">106</a>].
 Movement patterns are in turn highly dependent on different factors 
like temperature and general weather conditions as well as food supply 
and individual fitness [<a href="#B107-remotesensing-12-00080" class="html-bibr">107</a>].
 Furthermore, variations at the micro-habitat scale, including changes 
in soil moisture, humidity, and in the occurrence of ephemeral resources
 like prey availability, significantly affect beetlesâ€™ abundance, 
activity, and ultimately detectability [<a href="#B28-remotesensing-12-00080" class="html-bibr">28</a>,<a href="#B108-remotesensing-12-00080" class="html-bibr">108</a>,<a href="#B109-remotesensing-12-00080" class="html-bibr">109</a>].
 To reduce yearly variability, we selected species known to be (1) not 
directly depending on ephemeral resources and (2) nonspecific predators,
 feeding on various soil arthropods.</div><div class="html-p">The 
variability of species abundances at trap locations was one reason which
 led to differences in the values of the SRS-based model predictors at 
the presence locations between the two time periods (2006â€“2007 and 
2012â€“2013). The second reason for the observed variations was in the 
inter-annual differences in environmental conditions, as greenness and 
wetness were based on different Landsat images for the two time periods.
 Specifically, the Landsat image was acquired 20 days later in 2006 
compared to 2013, explaining the lower values of greenness (<a href="#app1-remotesensing-12-00080" class="html-app">Figure S4</a>) associated with differences in vegetation phenology [<a href="#B110-remotesensing-12-00080" class="html-bibr">110</a>].
 Such differences in the vegetation-related environmental variables 
between 2006â€“2007 and 2012â€“2013 likely had a comparatively stronger 
effect on the SDMs built for <span class="html-italic">Dinothenarus fossor</span> and <span class="html-italic">Platydracus stercorarius</span>,
 for which greenness was highly important. The variation in greenness 
values could explain the larger differences in predicted species 
distribution of the two species between time periods when compared to <span class="html-italic">Ocypus ophthalmicus.</span></div><div class="html-p">In
 addition, the 2006 Landsat image was also more strongly affected by 
clouds, shadows, and snow, which further reduced the total area 
available for modelling in both time periods and limited comparisons 
about temporal changes in species distributions. The effect of 
acquisition date of SRS imagery on analyzing species patterns was also 
shown by other studies [<a href="#B111-remotesensing-12-00080" class="html-bibr">111</a>,<a href="#B112-remotesensing-12-00080" class="html-bibr">112</a>],
 highlighting the drawbacks of using single SRS imagery representing 
temporal snapshots and not accounting for intra-annual differences of 
environmental variables. Analyzing multi-temporal SRS data may be a way 
to overcome these limitations and has been shown to improve SDMs for 
marsh bird species [<a href="#B113-remotesensing-12-00080" class="html-bibr">113</a>].
 SDMs based on longer composite periods (i.e., multiple years) also 
showed less deviance from observed species presence/absence field data [<a href="#B114-remotesensing-12-00080" class="html-bibr">114</a>].
 While being aware of these problems, the number of suitable SRS imagery
 was unfortunately restricted in our study and presented SDMs could not 
be improved in this regard (but see <a href="#sec5-remotesensing-12-00080" class="html-sec">Section 5</a>).
 Furthermore, radiometric differences between Landsat 5 TM (Thematic 
Mapper) and Landsat 8 OLI (Operational Land Imager) images may also have
 caused differences in species predictions [<a href="#B115-remotesensing-12-00080" class="html-bibr">115</a>].
 Nevertheless, radiometric differences were overall small in relation to
 other influencing factors, i.e., phenological differences between the 
two acquisition dates, and did not affect our overall outcome.</div></section><section id="sec4dot3-remotesensing-12-00080" type=""><h4 class="html-italic" data-nested="2"> 4.3. Importance of SRS Predictors </h4><div class="html-p">For
 all species, SRS-derived greenness was very important. This indicates 
that vegetated areas are generally preferred by our study species (which
 is well known at least for <span class="html-italic">Dinothenarus fossor</span> and <span class="html-italic">Platydracus stercorarius,</span> whereas <span class="html-italic">Ocypus ophthalmicus</span> is less dependent on vegetation [<a href="#B47-remotesensing-12-00080" class="html-bibr">47</a>,<a href="#B48-remotesensing-12-00080" class="html-bibr">48</a>,<a href="#B116-remotesensing-12-00080" class="html-bibr">116</a>]). This trend was clearly visible in the variable response curves of all models (<a href="#app1-remotesensing-12-00080" class="html-app">Figure S5</a>),
 in which the modelled probability of species presence increased at 
higher greenness values. Moreover, ruggedness was identified as the 
second most important variable in almost all models, negatively 
affecting the probability of species occurrence. This indicates that 
penetrability of the terrain determines species distribution: rugged 
terrain with large variations in elevation, which in turn affect soil 
thickness and moisture, was less suitable for rove beetles. While known 
macro- and micro-habitat requirements are rather different (see <a href="#remotesensing-12-00080-box001" class="html-boxed-text">Box 1</a> and description in <a href="#app1-remotesensing-12-00080" class="html-app">Section S1</a>),
 the patterns of variable importance and their response curves were 
fairly similar for the three study species, with greenness and 
ruggedness being the two most important variables in all ensemble models
 but one (the 2006â€“2007 model for <span class="html-italic">Platydracus stercorarius</span>, in which greenness and wetness scored the highest importance values, see <a href="#remotesensing-12-00080-t004" class="html-table">Table 4</a>).
 This indicates that the SRS variables employed in the present study 
were not entirely capable of depicting the subtle differences in terms 
of habitat requirements of invertebrate species.</div><div class="html-p">Apart
 from ruggedness, other topographic-related variables, such as slope, 
eastness, and northness, had relatively low importance in all models. 
The effect of wetness in the SDMs is hard to interpret, as it showed 
contrasting results in the modelsÂ´ variables response curves. Wetness 
had a strong negative effect in the 2006â€“2007 model built for <span class="html-italic">Platydracus stercorarius</span>,
 but not in 2012â€“2013. For the other two species, wetness had lower 
importance scores and a seemingly negative effect on species occurrence,
 as should be expected in xerophilous species (see <a href="#remotesensing-12-00080-box001" class="html-boxed-text">Box 1</a>).
 Altitude, which is typically inversely related to temperature, affected
 the SDMs very weakly but in different ways for the three species: While
 we observed negative effects for <span class="html-italic">Dinothenarus fossor</span> (although only for the model referring to 2006â€“2007), the tendency was opposite for <span class="html-italic">Ocypus ophthalmicus</span> in both time periods, and for <span class="html-italic">Platydracus stercorarius</span> in 2006â€“2007. These results are in line with our expectations, as the heliophilous species <span class="html-italic">Ocypus ophthalmicus</span>
 is known to occupy higher altitudes with sunny places above the tree 
line and to have a broader altitudinal range compared to the other 
species (see <a href="#remotesensing-12-00080-box001" class="html-boxed-text">Box 1</a> and <a href="#app1-remotesensing-12-00080" class="html-app">Section S1</a>).</div><div class="html-p">It
 is difficult to reflect our findings regarding variable importance with
 respect to other studies, as only very little research has focused on 
the analysis of spatial patterns of Staphylinidae, either at the 
community or at the species level. Indeed, the highest number of 
bibliographic sources is represented by faunistic works, followed by 
articles considering the role of Staphylinidae as bioindicators, in 
forest or in agricultural ecosystems [<a href="#B47-remotesensing-12-00080" class="html-bibr">47</a>,<a href="#B48-remotesensing-12-00080" class="html-bibr">48</a>,<a href="#B117-remotesensing-12-00080" class="html-bibr">117</a>,<a href="#B118-remotesensing-12-00080" class="html-bibr">118</a>].
 In any case, some general patterns have been found. In particular, it 
emerged that Staphylinidae are usually more influenced by local factors 
than landscape-scale features [<a href="#B119-remotesensing-12-00080" class="html-bibr">119</a>].
 Community composition and the presence of single species have often 
been observed as being determined by micro-climate (soil moisture and 
temperature) and by habitat-related characteristics (vegetation cover 
and micro-habitat characteristics like decaying material or soil 
texture) [<a href="#B27-remotesensing-12-00080" class="html-bibr">27</a>,<a href="#B117-remotesensing-12-00080" class="html-bibr">117</a>,<a href="#B120-remotesensing-12-00080" class="html-bibr">120</a>].</div></section><section id="sec4dot4-remotesensing-12-00080" type=""><h4 class="html-italic" data-nested="2"> 4.4. Suggestions for Model Improvement</h4><div class="html-p">While
 the ensemble SDMs outperformed the single algorithm models for all 
species and years and were thus rated as useful based on the 
cross-validation results, some suggestions for model improvement can be 
formulated.</div><div class="html-p">A continuation of the monitoring 
program in GPNP is recommended (and currently conducted) to assess 
trends in biodiversity and the impact of environmental changes on 
species [<a href="#B6-remotesensing-12-00080" class="html-bibr">6</a>,<a href="#B121-remotesensing-12-00080" class="html-bibr">121</a>].
 The generation of new species presence and absence information will 
also improve the performance of SDMs as inter-annual variability in 
species trapping success can be balanced out. Our decision to train the 
SDMs on plot-pooled data to avoid issues of pseudo-replication and of 
spatial correlation reduced our sample size significantly, consequently 
affecting the discriminatory power of the fitted models. An in-depth 
analysis of the spectral reflectance characteristics of presence and 
pseudo-absence data may have further improved model performance as 
recently shown by Remelgado et al. [<a href="#B122-remotesensing-12-00080" class="html-bibr">122</a>].</div><div class="html-p">The
 inclusion of additional predictors, such as detailed land cover and 
grazing intensity maps, could surely improve the models and possibly 
increase their discernment between the distribution maps of the three 
species. In our case, these data were unfortunately not available at the
 right spatial and temporal scale. The availability of environmental 
variables at finer spatial scale could reduce the problem of spatial 
autocorrelation between near presence points by enabling the detection 
of changes in environmental characteristics at a higher resolution, 
namely within the 50 m of inter-trap distance.</div><div class="html-p">While
 we here focused on cost-effective explanatory SRS data, recent advances
 in remote sensing technologies provide the possibility to better 
account for intra-annual differences in habitat characteristics which 
likely improve SDMs for our study species in the future. Specifically, 
Sentinel-1 radar images and Sentinel-2 optical images allow to monitor 
soil moisture content [<a href="#B123-remotesensing-12-00080" class="html-bibr">123</a>] and vegetation status and composition [<a href="#B124-remotesensing-12-00080" class="html-bibr">124</a>,<a href="#B125-remotesensing-12-00080" class="html-bibr">125</a>]
 at high spatial resolution and high temporal frequency, respectively. 
Likewise, high resolution satellite imagery (e.g., PlanetScope) or 
unmanned aerial vehicles (UAVs) equipped with multi- and hyperspectral 
sensors as well as laser scanner systems are very promising in this 
context by providing high resolution data on micro-habitat, vegetation 
structure, and topography [<a href="#B126-remotesensing-12-00080" class="html-bibr">126</a>,<a href="#B127-remotesensing-12-00080" class="html-bibr">127</a>].</div></section></section><section id="sec5-remotesensing-12-00080" type="conclusions"><h2 data-nested="1"> 5. Conclusions</h2><div class="html-p">The
 presented findings have important implications for future studies on 
species distribution modelling. Our results suggest that single time 
interval studies (as currently often done in SRS-based SDMs) may lead to
 false assumptions regarding underlying environmental drivers of species
 distributions. However, a continuation of monitoring is needed to not 
only track changes in species distribution patterns but also to better 
account for inter-annual variations due to natural population dynamics 
and differences in activity patterns. While in the present study, freely
 available Landsat images were employed, we detected shortages and 
corresponding implications for species distribution modelling due to the
 restricted temporal resolution of this data in mountainous regions 
characterized by high and persistent cloud cover. The limited number of 
suitable SRS imagery severely restricted our ability to account for 
intra-annual and inter-annual environmental variability and in turn 
limited the explanatory power regarding habitat suitability trends over 
time. However, recent advances in remote sensing technology can solve 
these problems in future works. Finally, the derived species 
distribution maps provide valuable information for conservation 
endeavors in GPNP, e.g., to identify species hot spots and potential 
connectivity paths inside the protected area. Furthermore, they support 
environmental change scenarios to identify potential biodiversity 
losses.</div></section>
</div>
<div class="html-back">
<section><section id="app1-remotesensing-12-00080" type=""><h2 data-nested="1"> Supplementary Materials</h2><div class="html-p">The following are available online at <a href="https://www.mdpi.com/2072-4292/12/1/80/s1" target="_blank" rel="noopener noreferrer">https://www.mdpi.com/2072-4292/12/1/80/s1</a>,
 Section S1: Detailed characteristics of analyzed species. Figure S1: 
Random points to obtain pseudo-absence data for species distribution 
modelling. Figure S2: Satellite remote sensing input data. Figure S3: 
Spearman rank correlations of explanatory variable values. Figure S4: 
Values and ranges of the SRS-based model training data. Figure S5: 
Variable response curves of ensemble models predictions. Table S1: 
Spearman rank correlation of species distribution maps. Table S2: 
Categorization of predicted species distribution. Figure S6: Uncertainty
 maps of predicted species distribution based on aggregated true 
presence data.</div></section></section><section class="html-notes"><h2>Author Contributions</h2><div class="html-p">Conceptualization:
 A.D., S.R., R.S., and A.F.C. designed the study and methodology. 
Project administration: A.F.C. Investigation: C.C. and R.V. conducted 
species sampling in the field, R.S. acquired and processed satellite 
images. Software: M.E., A.D., S.R., and A.F.C. wrote and adapted the R 
code for ensemble modelling. Formal analysis: A.D. and S.R. applied the R
 code to analyze the data; A.D., S.R., and A.F.C. prepared figures and 
tables. Validation: C.C. and R.V. evaluated species distribution maps 
based on local knowledge. Funding acquisition: A.F.C., C.C., R.S., and 
R.V. Resources: A.D., S.R., A.F.C., C.C., R.S., and R.V. Writing, 
original draft: All authors wrote the manuscript. All authors have read 
and agree to the published version of the manuscript.</div></section><section class="html-notes"><h2>Funding</h2><div class="html-p">This
 work has been carried out within the H2020 project â€˜ECOPOTENTIAL: 
Improving Future Ecosystem Benefits Through Earth Observationsâ€™ (<a href="http://www.ecopotential-project.eu/" target="_blank" rel="noopener noreferrer">http://www.ecopotential-project.eu</a>).
 The project has received funding from the European Unionâ€™s Horizon 2020
 research and innovation program under grant agreement No 641762. Parts 
of the R code used in this study were developed in the context of the 
project â€˜Improving species distribution models of endangered plants in 
Mexico by utilizing remote sensing data and spatial measures of model 
uncertaintyâ€™ funded by the National Commission for the Knowledge and Use
 of Biodiversity of Mexico (CONABIO).</div></section><section id="html-ack" class="html-ack"><h2>Acknowledgments</h2><div class="html-p">We
 are grateful to all park wardens of GPNP and students for their 
fundamental help during the field work and to the taxonomists A. Zanetti
 and G. Degasperi for the identification of the collected rove beetle 
specimens. E. Kasch (Fichtel Mountain Nature Park) provided important 
comments on the manuscript. We thank S. Steger for his supervision and 
fruitful discussions on the modelling techniques.</div></section><section class="html-notes"><h2>Conflicts of Interest</h2><div class="html-p">The authors declare no conflict of interest.</div></section><section id="html-references_list"><h2>References</h2><ol class="html-xxx"><li id="B1-remotesensing-12-00080" class="html-x" data-content="1.">KÃ¶rner,
 C. Why Are There Global Gradients in Species Richness? Mountains Might 
Hold the Answer Rapoport Effect and Speciation/Extinction Rates. <span class="html-italic">Trends Ecol. Evol.</span> <b>2000</b>, <span class="html-italic">15</span>, 513â€“514. [<a href="https://scholar.google.com/scholar_lookup?title=Why+Are+There+Global+Gradients+in+Species+Richness?+Mountains+Might+Hold+the+Answer+Rapoport+Effect+and+Speciation/Extinction+Rates&amp;author=K%C3%B6rner,+C.&amp;publication_year=2000&amp;journal=Trends+Ecol.+Evol.&amp;volume=15&amp;pages=513%E2%80%93514&amp;doi=10.1016/S0169-5347(00)02004-8" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1016/S0169-5347(00)02004-8" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B2-remotesensing-12-00080" class="html-x" data-content="2.">KÃ¶rner,
 C.; Paulsen, J.; Spehn, E.M. A Definition of Mountains and Their 
Bioclimatic Belts for Global Comparisons of Biodiversity Data. <span class="html-italic">Alp. Bot.</span> <b>2011</b>, <span class="html-italic">121</span>, 73â€“78. [<a href="https://scholar.google.com/scholar_lookup?title=A+Definition+of+Mountains+and+Their+Bioclimatic+Belts+for+Global+Comparisons+of+Biodiversity+Data&amp;author=K%C3%B6rner,+C.&amp;author=Paulsen,+J.&amp;author=Spehn,+E.M.&amp;publication_year=2011&amp;journal=Alp.+Bot.&amp;volume=121&amp;pages=73%E2%80%9378&amp;doi=10.1007/s00035-011-0094-4" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1007/s00035-011-0094-4" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B3-remotesensing-12-00080" class="html-x" data-content="3.">DÃ­az,
 S.; Pascual, U.; Stenseke, M.; MartÃ­n-LÃ³pez, B.; Watson, R.T.; MolnÃ¡r, 
Z.; Hill, R.; Chan, K.M.A.; Baste, I.A.; Brauman, K.A.; et al. Assessing
 Natureâ€™s Contributions to People. <span class="html-italic">Science</span> <b>2018</b>, <span class="html-italic">359</span>, 270â€“272. [<a href="https://scholar.google.com/scholar_lookup?title=Assessing+Nature%E2%80%99s+Contributions+to+People&amp;author=D%C3%ADaz,+S.&amp;author=Pascual,+U.&amp;author=Stenseke,+M.&amp;author=Mart%C3%ADn-L%C3%B3pez,+B.&amp;author=Watson,+R.T.&amp;author=Moln%C3%A1r,+Z.&amp;author=Hill,+R.&amp;author=Chan,+K.M.A.&amp;author=Baste,+I.A.&amp;author=Brauman,+K.A.&amp;publication_year=2018&amp;journal=Science&amp;volume=359&amp;pages=270%E2%80%93272&amp;doi=10.1126/science.aap8826&amp;pmid=29348221" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1126/science.aap8826" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>] [<a href="https://www.ncbi.nlm.nih.gov/pubmed/29348221" class="cross-ref" data-typ="pmid" target="_blank" rel="noopener noreferrer">PubMed</a>]</li><li id="B4-remotesensing-12-00080" class="html-x" data-content="4.">Viviroli, D.; Weingartner, R.; Messerli, B. Assessing the Hydrological Significance of the Worldâ€™s Mountains. <span class="html-italic">Mt. Res. Dev.</span> <b>2003</b>, <span class="html-italic">23</span>, 32â€“40. [<a href="https://scholar.google.com/scholar_lookup?title=Assessing+the+Hydrological+Significance+of+the+World%E2%80%99s+Mountains&amp;author=Viviroli,+D.&amp;author=Weingartner,+R.&amp;author=Messerli,+B.&amp;publication_year=2003&amp;journal=Mt.+Res.+Dev.&amp;volume=23&amp;pages=32%E2%80%9340&amp;doi=10.1659/0276-4741(2003)023[0032:ATHSOT]2.0.CO;2" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1659/0276-4741(2003)023[0032:ATHSOT]2.0.CO;2" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B5-remotesensing-12-00080" class="html-x" data-content="5.">Kohler, T.; Wehrli, A.; Jurek, M. <span class="html-italic">Mountains and Climate Change: A Global Concern. Sustainable Mountain Development Series</span>;
 Centre for Development and Environment (CDE), Swiss Agency for 
Development and Cooperation (SDC) and Geographica Bernensia: Bern, 
Switzerland, 2014. [<a href="https://dx.doi.org/10.1016/0360-8352(95)00014-R" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B6-remotesensing-12-00080" class="html-x" data-content="6.">Pocock,
 M.J.O.; Newson, S.E.; Henderson, I.G.; Peyton, J.; Sutherland, W.J.; 
Noble, D.G.; Ball, S.G.; Beckmann, B.C.; Biggs, J.; Brereton, T.; et al.
 Developing and Enhancing Biodiversity Monitoring Programmes: A 
Collaborative Assessment of Priorities. <span class="html-italic">J. Appl. Ecol.</span> <b>2015</b>, <span class="html-italic">52</span>, 686â€“695. [<a href="https://scholar.google.com/scholar_lookup?title=Developing+and+Enhancing+Biodiversity+Monitoring+Programmes:+A+Collaborative+Assessment+of+Priorities&amp;author=Pocock,+M.J.O.&amp;author=Newson,+S.E.&amp;author=Henderson,+I.G.&amp;author=Peyton,+J.&amp;author=Sutherland,+W.J.&amp;author=Noble,+D.G.&amp;author=Ball,+S.G.&amp;author=Beckmann,+B.C.&amp;author=Biggs,+J.&amp;author=Brereton,+T.&amp;publication_year=2015&amp;journal=J.+Appl.+Ecol.&amp;volume=52&amp;pages=686%E2%80%93695&amp;doi=10.1111/1365-2664.12423" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1111/1365-2664.12423" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B7-remotesensing-12-00080" class="html-x" data-content="7.">Franklin, J. <span class="html-italic">Mapping Species Distributions: Spatial Inference and Prediction</span>; Cambridge University Press: Cambridge, UK, 2009. [<a href="https://scholar.google.com/scholar_lookup?title=Mapping+Species+Distributions:+Spatial+Inference+and+Prediction&amp;author=Franklin,+J.&amp;publication_year=2009" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li><li id="B8-remotesensing-12-00080" class="html-x" data-content="8.">SardÃ -Palomera,
 F.; Brotons, L.; Villero, D.; Sierdsema, H.; Newson, S.E.; Jiguet, F. 
Mapping from Heterogeneous Biodiversity Monitoring Data Sources. <span class="html-italic">Biodivers. Conserv.</span> <b>2012</b>, <span class="html-italic">21</span>, 2927â€“2948. [<a href="https://scholar.google.com/scholar_lookup?title=Mapping+from+Heterogeneous+Biodiversity+Monitoring+Data+Sources&amp;author=Sard%C3%A0-Palomera,+F.&amp;author=Brotons,+L.&amp;author=Villero,+D.&amp;author=Sierdsema,+H.&amp;author=Newson,+S.E.&amp;author=Jiguet,+F.&amp;publication_year=2012&amp;journal=Biodivers.+Conserv.&amp;volume=21&amp;pages=2927%E2%80%932948&amp;doi=10.1007/s10531-012-0347-6" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1007/s10531-012-0347-6" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B9-remotesensing-12-00080" class="html-x" data-content="9.">Oke, O.A.; Thompson, K.A. Distribution Models for Mountain Plant Species: The Value of Elevation. <span class="html-italic">Ecol. Model.</span> <b>2015</b>, <span class="html-italic">301</span>, 72â€“77. [<a href="https://scholar.google.com/scholar_lookup?title=Distribution+Models+for+Mountain+Plant+Species:+The+Value+of+Elevation&amp;author=Oke,+O.A.&amp;author=Thompson,+K.A.&amp;publication_year=2015&amp;journal=Ecol.+Model.&amp;volume=301&amp;pages=72%E2%80%9377&amp;doi=10.1016/j.ecolmodel.2015.01.019" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1016/j.ecolmodel.2015.01.019" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B10-remotesensing-12-00080" class="html-xx" data-content="10.">Hijmans,
 R.J.; Cameron, S.E.; Parra, J.L.; Jones, P.G.; Jarvis, A. Very High 
Resolution Interpolated Climate Surfaces for Global Land Areas. <span class="html-italic">Int. J. Climatol.</span> <b>2005</b>, <span class="html-italic">25</span>, 1965â€“1978. [<a href="https://scholar.google.com/scholar_lookup?title=Very+High+Resolution+Interpolated+Climate+Surfaces+for+Global+Land+Areas&amp;author=Hijmans,+R.J.&amp;author=Cameron,+S.E.&amp;author=Parra,+J.L.&amp;author=Jones,+P.G.&amp;author=Jarvis,+A.&amp;publication_year=2005&amp;journal=Int.+J.+Climatol.&amp;volume=25&amp;pages=1965%E2%80%931978&amp;doi=10.1002/joc.1276" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1002/joc.1276" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B11-remotesensing-12-00080" class="html-xx" data-content="11.">Luoto, M.; Virkkala, R.; Heikkinen, R.K. The Role of Land Cover in Bioclimatic Models Depends on Spatial Resolution. <span class="html-italic">Glob. Ecol. Biogeogr.</span> <b>2007</b>, <span class="html-italic">16</span>, 34â€“42. [<a href="https://scholar.google.com/scholar_lookup?title=The+Role+of+Land+Cover+in+Bioclimatic+Models+Depends+on+Spatial+Resolution&amp;author=Luoto,+M.&amp;author=Virkkala,+R.&amp;author=Heikkinen,+R.K.&amp;publication_year=2007&amp;journal=Glob.+Ecol.+Biogeogr.&amp;volume=16&amp;pages=34%E2%80%9342&amp;doi=10.1111/j.1466-8238.2006.00262.x" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1111/j.1466-8238.2006.00262.x" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B12-remotesensing-12-00080" class="html-xx" data-content="12.">Bradley, B.A.; Fleishman, E. Can Remote Sensing of Land Cover Improve Species Distribution Modelling? <span class="html-italic">J. Biogeogr.</span> <b>2008</b>, <span class="html-italic">35</span>, 1158â€“1159. [<a href="https://scholar.google.com/scholar_lookup?title=Can+Remote+Sensing+of+Land+Cover+Improve+Species+Distribution+Modelling?&amp;author=Bradley,+B.A.&amp;author=Fleishman,+E.&amp;publication_year=2008&amp;journal=J.+Biogeogr.&amp;volume=35&amp;pages=1158%E2%80%931159&amp;doi=10.1111/j.1365-2699.2008.01928.x" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1111/j.1365-2699.2008.01928.x" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B13-remotesensing-12-00080" class="html-xx" data-content="13.">Cord,
 A.F.; Klein, D.; Mora, F.; Dech, S. Comparing the Suitability of 
Classified Land Cover Data and Remote Sensing Variables for Modeling 
Distribution Patterns of Plants. <span class="html-italic">Ecol. Model.</span> <b>2014</b>, <span class="html-italic">272</span>, 129â€“140. [<a href="https://scholar.google.com/scholar_lookup?title=Comparing+the+Suitability+of+Classified+Land+Cover+Data+and+Remote+Sensing+Variables+for+Modeling+Distribution+Patterns+of+Plants&amp;author=Cord,+A.F.&amp;author=Klein,+D.&amp;author=Mora,+F.&amp;author=Dech,+S.&amp;publication_year=2014&amp;journal=Ecol.+Model.&amp;volume=272&amp;pages=129%E2%80%93140&amp;doi=10.1016/j.ecolmodel.2013.09.011" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1016/j.ecolmodel.2013.09.011" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B14-remotesensing-12-00080" class="html-xx" data-content="14.">Li,
 X.; Ling, F.; Foody, G.M.; Ge, Y.; Zhang, Y.; Du, Y. Generating a 
Series of Fine Spatial and Temporal Resolution Land Cover Maps by Fusing
 Coarse Spatial Resolution Remotely Sensed Images and Fine Spatial 
Resolution Land Cover Maps. <span class="html-italic">Remote Sens. Environ.</span> <b>2017</b>, <span class="html-italic">196</span>, 293â€“311. [<a href="https://scholar.google.com/scholar_lookup?title=Generating+a+Series+of+Fine+Spatial+and+Temporal+Resolution+Land+Cover+Maps+by+Fusing+Coarse+Spatial+Resolution+Remotely+Sensed+Images+and+Fine+Spatial+Resolution+Land+Cover+Maps&amp;author=Li,+X.&amp;author=Ling,+F.&amp;author=Foody,+G.M.&amp;author=Ge,+Y.&amp;author=Zhang,+Y.&amp;author=Du,+Y.&amp;publication_year=2017&amp;journal=Remote+Sens.+Environ.&amp;volume=196&amp;pages=293%E2%80%93311&amp;doi=10.1016/j.rse.2017.05.011" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1016/j.rse.2017.05.011" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B15-remotesensing-12-00080" class="html-xx" data-content="15.">He,
 K.S.; Bradley, B.A.; Cord, A.F.; Rocchini, D.; Tuanmu, M.N.; 
Schmidtlein, S.; Turner, W.; Wegmann, M.; Pettorelli, N. Will Remote 
Sensing Shape the next Generation of Species Distribution Models? <span class="html-italic">Remote Sens. Ecol. Conserv.</span> <b>2015</b>, <span class="html-italic">1</span>, 4â€“18. [<a href="https://scholar.google.com/scholar_lookup?title=Will+Remote+Sensing+Shape+the+next+Generation+of+Species+Distribution+Models?&amp;author=He,+K.S.&amp;author=Bradley,+B.A.&amp;author=Cord,+A.F.&amp;author=Rocchini,+D.&amp;author=Tuanmu,+M.N.&amp;author=Schmidtlein,+S.&amp;author=Turner,+W.&amp;author=Wegmann,+M.&amp;author=Pettorelli,+N.&amp;publication_year=2015&amp;journal=Remote+Sens.+Ecol.+Conserv.&amp;volume=1&amp;pages=4%E2%80%9318&amp;doi=10.1002/rse2.7" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1002/rse2.7" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B16-remotesensing-12-00080" class="html-xx" data-content="16.">Weiss, D.J.; Walsh, S.J. Remote Sensing of Mountain Environments. <span class="html-italic">Geogr. Compass</span> <b>2009</b>, <span class="html-italic">3</span>, 1â€“21. [<a href="https://scholar.google.com/scholar_lookup?title=Remote+Sensing+of+Mountain+Environments&amp;author=Weiss,+D.J.&amp;author=Walsh,+S.J.&amp;publication_year=2009&amp;journal=Geogr.+Compass&amp;volume=3&amp;pages=1%E2%80%9321&amp;doi=10.1111/j.1749-8198.2008.00200.x" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1111/j.1749-8198.2008.00200.x" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B17-remotesensing-12-00080" class="html-xx" data-content="17.">Liang,
 L.; Li, X.; Huang, Y.; Qin, Y.; Huang, H. Integrating Remote Sensing, 
GIS and Dynamic Models for Landscape-Level Simulation of Forest Insect 
Disturbance. <span class="html-italic">Ecol. Model.</span> <b>2017</b>, <span class="html-italic">354</span>, 1â€“10. [<a href="https://scholar.google.com/scholar_lookup?title=Integrating+Remote+Sensing,+GIS+and+Dynamic+Models+for+Landscape-Level+Simulation+of+Forest+Insect+Disturbance&amp;author=Liang,+L.&amp;author=Li,+X.&amp;author=Huang,+Y.&amp;author=Qin,+Y.&amp;author=Huang,+H.&amp;publication_year=2017&amp;journal=Ecol.+Model.&amp;volume=354&amp;pages=1%E2%80%9310&amp;doi=10.1016/j.ecolmodel.2017.03.007" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1016/j.ecolmodel.2017.03.007" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B18-remotesensing-12-00080" class="html-xx" data-content="18.">Dudov,
 S.V. Modeling of Species Distribution with the Use of Topography and 
Remote Sensing Data on the Example of Vascular Plants of the Tukuringra 
Ridge Low Mountain Belt (Zeya State Nature Reserve, Amur Oblast). <span class="html-italic">Biol. Bull. Rev.</span> <b>2017</b>, <span class="html-italic">7</span>, 246â€“257. [<a href="https://scholar.google.com/scholar_lookup?title=Modeling+of+Species+Distribution+with+the+Use+of+Topography+and+Remote+Sensing+Data+on+the+Example+of+Vascular+Plants+of+the+Tukuringra+Ridge+Low+Mountain+Belt+(Zeya+State+Nature+Reserve,+Amur+Oblast)&amp;author=Dudov,+S.V.&amp;publication_year=2017&amp;journal=Biol.+Bull.+Rev.&amp;volume=7&amp;pages=246%E2%80%93257&amp;doi=10.1134/S2079086417030021" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1134/S2079086417030021" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B19-remotesensing-12-00080" class="html-xx" data-content="19.">Lausch,
 A.; Bannehr, L.; Beckmann, M.; Boehm, C.; Feilhauer, H.; Hacker, J.M.; 
Heurich, M.; Jung, A.; Klenke, R.; Neumann, C.; et al. Linking Earth 
Observation and Taxonomic, Structural and Functional Biodiversity: Local
 to Ecosystem Perspectives. <span class="html-italic">Ecol. Indic.</span> <b>2016</b>, <span class="html-italic">70</span>, 317â€“339. [<a href="https://scholar.google.com/scholar_lookup?title=Linking+Earth+Observation+and+Taxonomic,+Structural+and+Functional+Biodiversity:+Local+to+Ecosystem+Perspectives&amp;author=Lausch,+A.&amp;author=Bannehr,+L.&amp;author=Beckmann,+M.&amp;author=Boehm,+C.&amp;author=Feilhauer,+H.&amp;author=Hacker,+J.M.&amp;author=Heurich,+M.&amp;author=Jung,+A.&amp;author=Klenke,+R.&amp;author=Neumann,+C.&amp;publication_year=2016&amp;journal=Ecol.+Indic.&amp;volume=70&amp;pages=317%E2%80%93339&amp;doi=10.1016/j.ecolind.2016.06.022" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1016/j.ecolind.2016.06.022" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B20-remotesensing-12-00080" class="html-xx" data-content="20.">Turner,
 W.; Rondinini, C.; Pettorelli, N.; Mora, B.; Leidner, A.K.; Szantoi, 
Z.; Buchanan, G.; Dech, S.; Dwyer, J.; Herold, M.; et al. Free and 
Open-Access Satellite Data Are Key to Biodiversity Conservation. <span class="html-italic">Biol. Conserv.</span> <b>2015</b>, <span class="html-italic">182</span>, 173â€“176. [<a href="https://scholar.google.com/scholar_lookup?title=Free+and+Open-Access+Satellite+Data+Are+Key+to+Biodiversity+Conservation&amp;author=Turner,+W.&amp;author=Rondinini,+C.&amp;author=Pettorelli,+N.&amp;author=Mora,+B.&amp;author=Leidner,+A.K.&amp;author=Szantoi,+Z.&amp;author=Buchanan,+G.&amp;author=Dech,+S.&amp;author=Dwyer,+J.&amp;author=Herold,+M.&amp;publication_year=2015&amp;journal=Biol.+Conserv.&amp;volume=182&amp;pages=173%E2%80%93176&amp;doi=10.1016/j.biocon.2014.11.048" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1016/j.biocon.2014.11.048" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B21-remotesensing-12-00080" class="html-xx" data-content="21.">Vrieling,
 A.; Meroni, M.; Darvishzadeh, R.; Skidmore, A.K.; Wang, T.; 
Zurita-Milla, R.; Oosterbeek, K.; Oâ€™Connor, B.; Paganini, M. Vegetation 
Phenology from Sentinel-2 and Field Cameras for a Dutch Barrier Island. <span class="html-italic">Remote Sens. Environ.</span> <b>2018</b>, <span class="html-italic">215</span>, 517â€“529. [<a href="https://scholar.google.com/scholar_lookup?title=Vegetation+Phenology+from+Sentinel-2+and+Field+Cameras+for+a+Dutch+Barrier+Island&amp;author=Vrieling,+A.&amp;author=Meroni,+M.&amp;author=Darvishzadeh,+R.&amp;author=Skidmore,+A.K.&amp;author=Wang,+T.&amp;author=Zurita-Milla,+R.&amp;author=Oosterbeek,+K.&amp;author=O%E2%80%99Connor,+B.&amp;author=Paganini,+M.&amp;publication_year=2018&amp;journal=Remote+Sens.+Environ.&amp;volume=215&amp;pages=517%E2%80%93529&amp;doi=10.1016/j.rse.2018.03.014" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1016/j.rse.2018.03.014" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B22-remotesensing-12-00080" class="html-xx" data-content="22.">Kovacs,
 E.; Roelfsema, C.; Lyons, M.; Zhao, S.; Phinn, S. Seagrass Habitat 
Mapping: How Do Landsat 8 OLI, Sentinel-2, ZY-3A, and Worldview-3 
Perform? <span class="html-italic">Remote Sens. Lett.</span> <b>2018</b>, <span class="html-italic">9</span>, 686â€“695. [<a href="https://scholar.google.com/scholar_lookup?title=Seagrass+Habitat+Mapping:+How+Do+Landsat+8+OLI,+Sentinel-2,+ZY-3A,+and+Worldview-3+Perform?&amp;author=Kovacs,+E.&amp;author=Roelfsema,+C.&amp;author=Lyons,+M.&amp;author=Zhao,+S.&amp;author=Phinn,+S.&amp;publication_year=2018&amp;journal=Remote+Sens.+Lett.&amp;volume=9&amp;pages=686%E2%80%93695&amp;doi=10.1080/2150704X.2018.1468101" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1080/2150704X.2018.1468101" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B23-remotesensing-12-00080" class="html-xx" data-content="23.">Schmidt,
 J.; Fassnacht, F.E.; FÃ¶rster, M.; Schmidtlein, S. Synergetic Use of 
Sentinel-1 and Sentinel-2 for Assessments of Heathland Conservation 
Status. <span class="html-italic">Remote Sens. Ecol. Conserv.</span> <b>2018</b>, <span class="html-italic">4</span>, 225â€“239. [<a href="https://scholar.google.com/scholar_lookup?title=Synergetic+Use+of+Sentinel-1+and+Sentinel-2+for+Assessments+of+Heathland+Conservation+Status&amp;author=Schmidt,+J.&amp;author=Fassnacht,+F.E.&amp;author=F%C3%B6rster,+M.&amp;author=Schmidtlein,+S.&amp;publication_year=2018&amp;journal=Remote+Sens.+Ecol.+Conserv.&amp;volume=4&amp;pages=225%E2%80%93239&amp;doi=10.1002/rse2.68" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1002/rse2.68" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B24-remotesensing-12-00080" class="html-xx" data-content="24.">Wulder,
 M.A.; Masek, J.G.; Cohen, W.B.; Loveland, T.R.; Woodcock, C.E. Opening 
the Archive: How Free Data Has Enabled the Science and Monitoring 
Promise of Landsat. <span class="html-italic">Remote Sens. Environ.</span> <b>2012</b>, <span class="html-italic">122</span>, 2â€“10. [<a href="https://scholar.google.com/scholar_lookup?title=Opening+the+Archive:+How+Free+Data+Has+Enabled+the+Science+and+Monitoring+Promise+of+Landsat&amp;author=Wulder,+M.A.&amp;author=Masek,+J.G.&amp;author=Cohen,+W.B.&amp;author=Loveland,+T.R.&amp;author=Woodcock,+C.E.&amp;publication_year=2012&amp;journal=Remote+Sens.+Environ.&amp;volume=122&amp;pages=2%E2%80%9310&amp;doi=10.1016/j.rse.2012.01.010" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1016/j.rse.2012.01.010" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B25-remotesensing-12-00080" class="html-xx" data-content="25.">Wulder,
 M.A.; Loveland, T.R.; Roy, D.P.; Crawford, C.J.; Masek, J.G.; Woodcock,
 C.E.; Allen, R.G.; Anderson, M.C.; Belward, A.S.; Cohen, W.B.; et al. 
Current Status of Landsat Program, Science, and Applications. <span class="html-italic">Remote Sens. Environ.</span> <b>2019</b>, <span class="html-italic">225</span>, 127â€“147. [<a href="https://scholar.google.com/scholar_lookup?title=Current+Status+of+Landsat+Program,+Science,+and+Applications&amp;author=Wulder,+M.A.&amp;author=Loveland,+T.R.&amp;author=Roy,+D.P.&amp;author=Crawford,+C.J.&amp;author=Masek,+J.G.&amp;author=Woodcock,+C.E.&amp;author=Allen,+R.G.&amp;author=Anderson,+M.C.&amp;author=Belward,+A.S.&amp;author=Cohen,+W.B.&amp;publication_year=2019&amp;journal=Remote+Sens.+Environ.&amp;volume=225&amp;pages=127%E2%80%93147&amp;doi=10.1016/j.rse.2019.02.015" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1016/j.rse.2019.02.015" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B26-remotesensing-12-00080" class="html-xx" data-content="26.">Grimaldi, D.; Engel, M.S. <span class="html-italic">Evolution of the Insect</span>; Cambridge University Press: New York, NY, USA, 2005. [<a href="https://scholar.google.com/scholar_lookup?title=Evolution+of+the+Insect&amp;author=Grimaldi,+D.&amp;author=Engel,+M.S.&amp;publication_year=2005" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li><li id="B27-remotesensing-12-00080" class="html-xx" data-content="27.">Bohac, J. Staphylinid Beetles as Bioindicators. <span class="html-italic">Agric. Ecosyst. Environ.</span> <b>1999</b>, <span class="html-italic">74</span>, 357â€“372. [<a href="https://scholar.google.com/scholar_lookup?title=Staphylinid+Beetles+as+Bioindicators&amp;author=Bohac,+J.&amp;publication_year=1999&amp;journal=Agric.+Ecosyst.+Environ.&amp;volume=74&amp;pages=357%E2%80%93372&amp;doi=10.1016/S0167-8809(99)00043-2" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1016/S0167-8809(99)00043-2" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B28-remotesensing-12-00080" class="html-xx" data-content="28.">Pohl,
 G.R.; Langor, D.W.; Spence, J.R. Rove Beetles and Ground Beetles 
(Coleoptera: Staphylinidae, Carabidae) as Indicators of Harvest and 
Regeneration Practices in Western Canadian Foothills Forests. <span class="html-italic">Biol. Conserv.</span> <b>2007</b>, <span class="html-italic">137</span>, 294â€“307. [<a href="https://scholar.google.com/scholar_lookup?title=Rove+Beetles+and+Ground+Beetles+(Coleoptera:+Staphylinidae,+Carabidae)+as+Indicators+of+Harvest+and+Regeneration+Practices+in+Western+Canadian+Foothills+Forests&amp;author=Pohl,+G.R.&amp;author=Langor,+D.W.&amp;author=Spence,+J.R.&amp;publication_year=2007&amp;journal=Biol.+Conserv.&amp;volume=137&amp;pages=294%E2%80%93307&amp;doi=10.1016/j.biocon.2007.02.011" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1016/j.biocon.2007.02.011" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B29-remotesensing-12-00080" class="html-xx" data-content="29.">Assing, V.; SchÃ¼lke, M. (Eds.) <span class="html-italic">Die KÃ¤fer Mitteleuropas, Bd. 4: Staphylinidae (Exklusive Aleocharinae, Pselaphinae Und Scydmaeninae)</span>, 2nd ed.; Springer Spektrum: Berlin/Heidelberg, Germany, 2012; ISBN 978-3-8274-1677-3. [<a href="https://scholar.google.com/scholar_lookup?title=Die+K%C3%A4fer+Mitteleuropas,+Bd.+4:+Staphylinidae+(Exklusive+Aleocharinae,+Pselaphinae+Und+Scydmaeninae)&amp;author=Assing,+V.&amp;author=Sch%C3%BClke,+M.&amp;publication_year=2012" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li><li id="B30-remotesensing-12-00080" class="html-xx" data-content="30.">Elith, J.; Leathwick, J.R. Species Distribution Models: Ecological Explanation and Prediction Across Space and Time. <span class="html-italic">Annu. Rev. Ecol. Evol. Syst.</span> <b>2009</b>, <span class="html-italic">40</span>, 677â€“697. [<a href="https://scholar.google.com/scholar_lookup?title=Species+Distribution+Models:+Ecological+Explanation+and+Prediction+Across+Space+and+Time&amp;author=Elith,+J.&amp;author=Leathwick,+J.R.&amp;publication_year=2009&amp;journal=Annu.+Rev.+Ecol.+Evol.+Syst.&amp;volume=40&amp;pages=677%E2%80%93697&amp;doi=10.1146/annurev.ecolsys.110308.120159" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1146/annurev.ecolsys.110308.120159" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B31-remotesensing-12-00080" class="html-xx" data-content="31.">AraÃºjo, M.B.; New, M. Ensemble Forecasting of Species Distributions. <span class="html-italic">Trends Ecol. Evol.</span> <b>2007</b>, <span class="html-italic">22</span>, 42â€“47. [<a href="https://scholar.google.com/scholar_lookup?title=Ensemble+Forecasting+of+Species+Distributions&amp;author=Ara%C3%BAjo,+M.B.&amp;author=New,+M.&amp;publication_year=2007&amp;journal=Trends+Ecol.+Evol.&amp;volume=22&amp;pages=42%E2%80%9347&amp;doi=10.1016/j.tree.2006.09.010" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1016/j.tree.2006.09.010" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B32-remotesensing-12-00080" class="html-xx" data-content="32.">McCullagh, P.; Nelder, J.A. <span class="html-italic">Generalized Linear Models</span>,
 2nd ed.; Monographs on Statistics and Applied Probability; Chapman and 
Hall/CRC: Boca Raton, FL, USA, 1989; ISBN 9780412317606. [<a href="https://scholar.google.com/scholar_lookup?title=Generalized+Linear+Models&amp;author=McCullagh,+P.&amp;author=Nelder,+J.A.&amp;publication_year=1989" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li><li id="B33-remotesensing-12-00080" class="html-xx" data-content="33.">Hastie, T.; Tibshirani, R. Generalized Additive Models. <span class="html-italic">Stat. Sci.</span> <b>1986</b>, 249â€“307. [<a href="https://scholar.google.com/scholar_lookup?title=Generalized+Additive+Models&amp;author=Hastie,+T.&amp;author=Tibshirani,+R.&amp;publication_year=1986&amp;journal=Stat.+Sci.&amp;pages=249%E2%80%93307&amp;doi=10.1214/ss/1177013604" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1214/ss/1177013604" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B34-remotesensing-12-00080" class="html-xx" data-content="34.">Breiman, L. Random Forests. <span class="html-italic">Mach. Learn.</span> <b>2001</b>, <span class="html-italic">45</span>, 5â€“32. [<a href="https://scholar.google.com/scholar_lookup?title=Random+Forests&amp;author=Breiman,+L.&amp;publication_year=2001&amp;journal=Mach.+Learn.&amp;volume=45&amp;pages=5%E2%80%9332&amp;doi=10.1023/A:1010933404324" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1023/A:1010933404324" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B35-remotesensing-12-00080" class="html-xx" data-content="35.">Elith, J.; Leathwick, J.R.; Hastie, T. A Working Guide to Boosted Regression Trees. <span class="html-italic">J. Anim. Ecol.</span> <b>2008</b>, <span class="html-italic">77</span>, 802â€“813. [<a href="https://scholar.google.com/scholar_lookup?title=A+Working+Guide+to+Boosted+Regression+Trees&amp;author=Elith,+J.&amp;author=Leathwick,+J.R.&amp;author=Hastie,+T.&amp;publication_year=2008&amp;journal=J.+Anim.+Ecol.&amp;volume=77&amp;pages=802%E2%80%93813&amp;doi=10.1111/j.1365-2656.2008.01390.x" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1111/j.1365-2656.2008.01390.x" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B36-remotesensing-12-00080" class="html-xx" data-content="36.">Phillips, S.J.; DudÃ­k, M. Modeling of Species Distributions with Maxent: New Extensions and a Comprehensive Evaluation. <span class="html-italic">Ecography</span> <b>2008</b>, <span class="html-italic">31</span>, 161â€“175. [<a href="https://scholar.google.com/scholar_lookup?title=Modeling+of+Species+Distributions+with+Maxent:+New+Extensions+and+a+Comprehensive+Evaluation&amp;author=Phillips,+S.J.&amp;author=Dud%C3%ADk,+M.&amp;publication_year=2008&amp;journal=Ecography&amp;volume=31&amp;pages=161%E2%80%93175&amp;doi=10.1111/j.0906-7590.2008.5203.x" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1111/j.0906-7590.2008.5203.x" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B37-remotesensing-12-00080" class="html-xx" data-content="37.">Marmion,
 M.; Parviainen, M.; Luoto, M.; Heikkinen, R.K.; Thuiller, W. Evaluation
 of Consensus Methods in Predictive Species Distribution Modelling. <span class="html-italic">Divers. Distrib.</span> <b>2009</b>, <span class="html-italic">15</span>, 59â€“69. [<a href="https://scholar.google.com/scholar_lookup?title=Evaluation+of+Consensus+Methods+in+Predictive+Species+Distribution+Modelling&amp;author=Marmion,+M.&amp;author=Parviainen,+M.&amp;author=Luoto,+M.&amp;author=Heikkinen,+R.K.&amp;author=Thuiller,+W.&amp;publication_year=2009&amp;journal=Divers.+Distrib.&amp;volume=15&amp;pages=59%E2%80%9369&amp;doi=10.1111/j.1472-4642.2008.00491.x" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1111/j.1472-4642.2008.00491.x" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B38-remotesensing-12-00080" class="html-xx" data-content="38.">Gran Paradiso National Parkâ€”Italy. Available online: <a href="https://data.lter-europe.net/deims/site/lter_eu_it_109" target="_blank" rel="noopener noreferrer">https://data.lter-europe.net/deims/site/lter_eu_it_109</a> (accessed on 20 October 2017).</li><li id="B39-remotesensing-12-00080" class="html-xx" data-content="39.">Viterbi,
 R.; Cerrato, C.; Bassano, B.; Bionda, R.; Hardenberg, A.; Provenzale, 
A.; Bogliani, G. Patterns of Biodiversity in the Northwestern Italian 
Alps: A Multi-Taxa Approach. <span class="html-italic">Community Ecol.</span> <b>2013</b>, <span class="html-italic">14</span>, 18â€“30. [<a href="https://scholar.google.com/scholar_lookup?title=Patterns+of+Biodiversity+in+the+Northwestern+Italian+Alps:+A+Multi-Taxa+Approach&amp;author=Viterbi,+R.&amp;author=Cerrato,+C.&amp;author=Bassano,+B.&amp;author=Bionda,+R.&amp;author=Hardenberg,+A.&amp;author=Provenzale,+A.&amp;author=Bogliani,+G.&amp;publication_year=2013&amp;journal=Community+Ecol.&amp;volume=14&amp;pages=18%E2%80%9330&amp;doi=10.1556/ComEc.14.2013.1.3" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1556/ComEc.14.2013.1.3" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B40-remotesensing-12-00080" class="html-xx" data-content="40.">Thomasset, F.; Ottino, M. Parco Nazionale Gran Paradiso. Piano del Parco. Relazione Illustrativa. Available online: <a href="http://www.pngp.it/documenti/Piano%20del%20parco/PNGP_Relazione.illustrativa.pdf" target="_blank" rel="noopener noreferrer">http://www.pngp.it/documenti/Piano del parco/PNGP_Relazione.illustrativa.pdf</a> (accessed on 2 March 2018).</li><li id="B41-remotesensing-12-00080" class="html-xx" data-content="41.">Cagnacci, F.; Lovari, S.; Meriggi, A. Carrion Dependence and Food Habits of the Red Fox in an Alpine Area. <span class="html-italic">Ital. J. Zool.</span> <b>2003</b>, <span class="html-italic">70</span>, 31â€“38. [<a href="https://scholar.google.com/scholar_lookup?title=Carrion+Dependence+and+Food+Habits+of+the+Red+Fox+in+an+Alpine+Area&amp;author=Cagnacci,+F.&amp;author=Lovari,+S.&amp;author=Meriggi,+A.&amp;publication_year=2003&amp;journal=Ital.+J.+Zool.&amp;volume=70&amp;pages=31%E2%80%9338&amp;doi=10.1080/11250000309356493" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1080/11250000309356493" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B42-remotesensing-12-00080" class="html-xx" data-content="42.">Poussin,
 C.; Guigoz, Y.; Palazzi, E.; Terzago, S.; Chatenoux, B.; Giuliani, G. 
Snow Cover Evolution in the Gran Paradiso National Park, Italian Alps, 
Using the Earth Observation Data Cube. <span class="html-italic">Data</span> <b>2019</b>, <span class="html-italic">4</span>, 138. [<a href="https://scholar.google.com/scholar_lookup?title=Snow+Cover+Evolution+in+the+Gran+Paradiso+National+Park,+Italian+Alps,+Using+the+Earth+Observation+Data+Cube&amp;author=Poussin,+C.&amp;author=Guigoz,+Y.&amp;author=Palazzi,+E.&amp;author=Terzago,+S.&amp;author=Chatenoux,+B.&amp;author=Giuliani,+G.&amp;publication_year=2019&amp;journal=Data&amp;volume=4&amp;pages=138&amp;doi=10.3390/data4040138" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.3390/data4040138" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B43-remotesensing-12-00080" class="html-xx" data-content="43.">Marcelino,
 J.A.P.; Giordano, R.; Borges, P.A.V.; Garcia, P.V.; Soto-Adames, F.N.; 
Soares, A.O. Distribution and Genetic Variability of Staphylinidae 
across a Gradient of Anthropogenically Influenced Insular Landscapes. <span class="html-italic">Bull. Insectol.</span> <b>2016</b>, <span class="html-italic">69</span>, 117â€“126. [<a href="https://scholar.google.com/scholar_lookup?title=Distribution+and+Genetic+Variability+of+Staphylinidae+across+a+Gradient+of+Anthropogenically+Influenced+Insular+Landscapes&amp;author=Marcelino,+J.A.P.&amp;author=Giordano,+R.&amp;author=Borges,+P.A.V.&amp;author=Garcia,+P.V.&amp;author=Soto-Adames,+F.N.&amp;author=Soares,+A.O.&amp;publication_year=2016&amp;journal=Bull.+Insectol.&amp;volume=69&amp;pages=117%E2%80%93126" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li><li id="B44-remotesensing-12-00080" class="html-xx" data-content="44.">Work,
 T.T.; Klimaszewski, J.; Thiffault, E.; Bourdon, C.; ParÃ©, D.; Bousquet,
 Y.; Venier, L.; Titus, B. Initial Responses of Rove and Ground Beetles 
(Coleoptera, Staphylinidae, Carabidae) to Removal of Logging Residues 
Following Clearcut Harvesting in the Boreal Forest of Quebec, Canada. <span class="html-italic">Zookeys</span> <b>2013</b>, <span class="html-italic">258</span>, 31â€“52. [<a href="https://scholar.google.com/scholar_lookup?title=Initial+Responses+of+Rove+and+Ground+Beetles+(Coleoptera,+Staphylinidae,+Carabidae)+to+Removal+of+Logging+Residues+Following+Clearcut+Harvesting+in+the+Boreal+Forest+of+Quebec,+Canada&amp;author=Work,+T.T.&amp;author=Klimaszewski,+J.&amp;author=Thiffault,+E.&amp;author=Bourdon,+C.&amp;author=Par%C3%A9,+D.&amp;author=Bousquet,+Y.&amp;author=Venier,+L.&amp;author=Titus,+B.&amp;publication_year=2013&amp;journal=Zookeys&amp;volume=258&amp;pages=31%E2%80%9352&amp;doi=10.3897/zookeys.258.4174" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.3897/zookeys.258.4174" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B45-remotesensing-12-00080" class="html-xx" data-content="45.">Standen, V. The Adequacy of Collecting Techniques for Estimating Species Richness of Grassland Invertebrates. <span class="html-italic">J. Appl. Ecol.</span> <b>2000</b>, <span class="html-italic">37</span>, 884â€“893. [<a href="https://scholar.google.com/scholar_lookup?title=The+Adequacy+of+Collecting+Techniques+for+Estimating+Species+Richness+of+Grassland+Invertebrates&amp;author=Standen,+V.&amp;publication_year=2000&amp;journal=J.+Appl.+Ecol.&amp;volume=37&amp;pages=884%E2%80%93893&amp;doi=10.1046/j.1365-2664.2000.00532.x" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1046/j.1365-2664.2000.00532.x" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B46-remotesensing-12-00080" class="html-xx" data-content="46.">Ribera,
 I.; DolÃ©dec, S.; Downie, I.S.; Foster, G.N. Effect of Land Disturbance 
and Stress on Species Traits of Ground Beetle Assemblages. <span class="html-italic">Ecology</span> <b>2001</b>, <span class="html-italic">82</span>, 1112â€“1129. [<a href="https://scholar.google.com/scholar_lookup?title=Effect+of+Land+Disturbance+and+Stress+on+Species+Traits+of+Ground+Beetle+Assemblages&amp;author=Ribera,+I.&amp;author=Dol%C3%A9dec,+S.&amp;author=Downie,+I.S.&amp;author=Foster,+G.N.&amp;publication_year=2001&amp;journal=Ecology&amp;volume=82&amp;pages=1112%E2%80%931129&amp;doi=10.1890/0012-9658(2001)082[1112:EOLDAS]2.0.CO;2" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1890/0012-9658(2001)082[1112:EOLDAS]2.0.CO;2" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B47-remotesensing-12-00080" class="html-xx" data-content="47.">Tagliapietra, A.; Zanetti, A. Staphylinid Beetles in Natura 2000 Sites of Friuli Venezia Giulia. <span class="html-italic">Gortania Botanica Zool.</span> <b>2012</b>, <span class="html-italic">33</span>, 97â€“124. [<a href="https://scholar.google.com/scholar_lookup?title=Staphylinid+Beetles+in+Natura+2000+Sites+of+Friuli+Venezia+Giulia&amp;author=Tagliapietra,+A.&amp;author=Zanetti,+A.&amp;publication_year=2012&amp;journal=Gortania+Botanica+Zool.&amp;volume=33&amp;pages=97%E2%80%93124" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li><li id="B48-remotesensing-12-00080" class="html-xx" data-content="48.">Zanetti,
 A.; Sette, A.; Poggi, R.; Tagliapietra, A. Biodiversity of 
Staphylinidae (Coleoptera) in the Province of Verona (Veneto, Northern 
Italy). <span class="html-italic">Mem. Della Soc. Entomol. Ital.</span> <b>2016</b>, <span class="html-italic">93</span>, 3â€“237. [<a href="https://scholar.google.com/scholar_lookup?title=Biodiversity+of+Staphylinidae+(Coleoptera)+in+the+Province+of+Verona+(Veneto,+Northern+Italy)&amp;author=Zanetti,+A.&amp;author=Sette,+A.&amp;author=Poggi,+R.&amp;author=Tagliapietra,+A.&amp;publication_year=2016&amp;journal=Mem.+Della+Soc.+Entomol.+Ital.&amp;volume=93&amp;pages=3%E2%80%93237&amp;doi=10.4081/MemorieSEI.2016.3" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.4081/MemorieSEI.2016.3" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B49-remotesensing-12-00080" class="html-xx" data-content="49.">Wisz,
 M.S.; Hijmans, R.J.; Li, J.; Peterson, A.T.; Graham, C.H.; Guisan, A. 
Effects of Sample Size on the Performance of Species Distribution 
Models. <span class="html-italic">Divers. Distrib.</span> <b>2008</b>, <span class="html-italic">14</span>, 763â€“773. [<a href="https://scholar.google.com/scholar_lookup?title=Effects+of+Sample+Size+on+the+Performance+of+Species+Distribution+Models&amp;author=Wisz,+M.S.&amp;author=Hijmans,+R.J.&amp;author=Li,+J.&amp;author=Peterson,+A.T.&amp;author=Graham,+C.H.&amp;author=Guisan,+A.&amp;publication_year=2008&amp;journal=Divers.+Distrib.&amp;volume=14&amp;pages=763%E2%80%93773&amp;doi=10.1111/j.1472-4642.2008.00482.x" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1111/j.1472-4642.2008.00482.x" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B50-remotesensing-12-00080" class="html-xx" data-content="50.">Van
 Proosdij, A.S.J.; Sosef, M.S.M.; Wieringa, J.J.; Raes, N. Minimum 
Required Number of Specimen Records to Develop Accurate Species 
Distribution Models. <span class="html-italic">Ecography</span> <b>2016</b>, <span class="html-italic">39</span>, 542â€“552. [<a href="https://scholar.google.com/scholar_lookup?title=Minimum+Required+Number+of+Specimen+Records+to+Develop+Accurate+Species+Distribution+Models&amp;author=Van+Proosdij,+A.S.J.&amp;author=Sosef,+M.S.M.&amp;author=Wieringa,+J.J.&amp;author=Raes,+N.&amp;publication_year=2016&amp;journal=Ecography&amp;volume=39&amp;pages=542%E2%80%93552&amp;doi=10.1111/ecog.01509" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1111/ecog.01509" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B51-remotesensing-12-00080" class="html-xx" data-content="51.">Stockwell, D.R.B.; Peterson, A.T. Effects of Sample Size on Accuracy of Species Distribution Models. <span class="html-italic">Ecol. Model.</span> <b>2002</b>, <span class="html-italic">148</span>, 1â€“13. [<a href="https://scholar.google.com/scholar_lookup?title=Effects+of+Sample+Size+on+Accuracy+of+Species+Distribution+Models&amp;author=Stockwell,+D.R.B.&amp;author=Peterson,+A.T.&amp;publication_year=2002&amp;journal=Ecol.+Model.&amp;volume=148&amp;pages=1%E2%80%9313&amp;doi=10.1016/S0304-3800(01)00388-X" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1016/S0304-3800(01)00388-X" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B52-remotesensing-12-00080" class="html-xx" data-content="52.">Hao,
 T.; Elith, J.; Guillera-Arroita, G.; Lahoz-Monfort, J.J. A Review of 
Evidence about Use and Performance of Species Distribution Modelling 
Ensembles like BIOMOD. <span class="html-italic">Divers. Distrib.</span> <b>2019</b>, <span class="html-italic">25</span>, 839â€“852. [<a href="https://scholar.google.com/scholar_lookup?title=A+Review+of+Evidence+about+Use+and+Performance+of+Species+Distribution+Modelling+Ensembles+like+BIOMOD&amp;author=Hao,+T.&amp;author=Elith,+J.&amp;author=Guillera-Arroita,+G.&amp;author=Lahoz-Monfort,+J.J.&amp;publication_year=2019&amp;journal=Divers.+Distrib.&amp;volume=25&amp;pages=839%E2%80%93852&amp;doi=10.1111/ddi.12892" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1111/ddi.12892" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B53-remotesensing-12-00080" class="html-xx" data-content="53.">Elith,
 J.; Graham, C.H.; Anderson, R.P.; DudÃ­k, M.; Ferrier, S.; Guisan, A.; 
Hijmans, R.J.; Huettmann, F.; Leathwick, J.R.; Lehmann, A.; et al. Novel
 Methods Improve Prediction of Speciesâ€™ Distributions from Occurrence 
Data. <span class="html-italic">Ecography</span> <b>2006</b>, <span class="html-italic">29</span>, 129â€“151. [<a href="https://scholar.google.com/scholar_lookup?title=Novel+Methods+Improve+Prediction+of+Species%E2%80%99+Distributions+from+Occurrence+Data&amp;author=Elith,+J.&amp;author=Graham,+C.H.&amp;author=Anderson,+R.P.&amp;author=Dud%C3%ADk,+M.&amp;author=Ferrier,+S.&amp;author=Guisan,+A.&amp;author=Hijmans,+R.J.&amp;author=Huettmann,+F.&amp;author=Leathwick,+J.R.&amp;author=Lehmann,+A.&amp;publication_year=2006&amp;journal=Ecography&amp;volume=29&amp;pages=129%E2%80%93151&amp;doi=10.1111/j.2006.0906-7590.04596.x" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1111/j.2006.0906-7590.04596.x" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B54-remotesensing-12-00080" class="html-xx" data-content="54.">Fletcher, R.; Fortin, M.J. <span class="html-italic">Spatial Ecology and Conservation Modeling: Applications with R</span>; Springer International Publishing: Cham, Switzerland, 2018. [<a href="https://scholar.google.com/scholar_lookup?title=Spatial+Ecology+and+Conservation+Modeling:+Applications+with+R&amp;author=Fletcher,+R.&amp;author=Fortin,+M.J.&amp;publication_year=2018" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1007/978-3-030-01989-1" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B55-remotesensing-12-00080" class="html-xx" data-content="55.">Lobo, J.M.; JimÃ©nez-valverde, A.; Real, R. AUC: A Misleading Measure of the Performance of Predictive Distribution Models. <span class="html-italic">Glob. Ecol. Biogeogr.</span> <b>2008</b>, <span class="html-italic">17</span>, 145â€“151. [<a href="https://scholar.google.com/scholar_lookup?title=AUC:+A+Misleading+Measure+of+the+Performance+of+Predictive+Distribution+Models&amp;author=Lobo,+J.M.&amp;author=Jim%C3%A9nez-valverde,+A.&amp;author=Real,+R.&amp;publication_year=2008&amp;journal=Glob.+Ecol.+Biogeogr.&amp;volume=17&amp;pages=145%E2%80%93151&amp;doi=10.1111/j.1466-8238.2007.00358.x" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1111/j.1466-8238.2007.00358.x" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B56-remotesensing-12-00080" class="html-xx" data-content="56.">Earth
 Resources Observation and Science (EROS) Center Science Processing 
Architecture (ESPA) on Demand Interface. On Demand Interface. Available 
online: <a href="https://landsat.usgs.gov/sites/default/files/documents/espa_odi_userguide.pdf" target="_blank" rel="noopener noreferrer">https://landsat.usgs.gov/sites/default/files/documents/espa_odi_userguide.pdf</a> (accessed on 5 April 2018).</li><li id="B57-remotesensing-12-00080" class="html-xx" data-content="57.">RÃ¶dder,
 D.; Nekum, S.; Cord, A.F.; Engler, J.O. Coupling Satellite Data with 
Species Distribution and Connectivity Models as a Tool for Environmental
 Management and Planning in Matrix-Sensitive Species. <span class="html-italic">Environ. Manag.</span> <b>2016</b>, <span class="html-italic">58</span>, 130â€“143. [<a href="https://scholar.google.com/scholar_lookup?title=Coupling+Satellite+Data+with+Species+Distribution+and+Connectivity+Models+as+a+Tool+for+Environmental+Management+and+Planning+in+Matrix-Sensitive+Species&amp;author=R%C3%B6dder,+D.&amp;author=Nekum,+S.&amp;author=Cord,+A.F.&amp;author=Engler,+J.O.&amp;publication_year=2016&amp;journal=Environ.+Manag.&amp;volume=58&amp;pages=130%E2%80%93143&amp;doi=10.1007/s00267-016-0698-y" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1007/s00267-016-0698-y" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B58-remotesensing-12-00080" class="html-xx" data-content="58.">Regos,
 A.; Tapia, L.; Gil-Carrera, A.; DomÃ­nguez, J. Monitoring Protected 
Areas from Space: A Multi-Temporal Assessment Using Raptors as 
Biodiversity Surrogates. <span class="html-italic">PLoS ONE</span> <b>2017</b>, <span class="html-italic">12</span>, e0181769. [<a href="https://scholar.google.com/scholar_lookup?title=Monitoring+Protected+Areas+from+Space:+A+Multi-Temporal+Assessment+Using+Raptors+as+Biodiversity+Surrogates&amp;author=Regos,+A.&amp;author=Tapia,+L.&amp;author=Gil-Carrera,+A.&amp;author=Dom%C3%ADnguez,+J.&amp;publication_year=2017&amp;journal=PLoS+ONE&amp;volume=12&amp;pages=e0181769&amp;doi=10.1371/journal.pone.0181769" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1371/journal.pone.0181769" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B59-remotesensing-12-00080" class="html-xx" data-content="59.">Crist, E.P.; Cicone, R.C. Application of the Tasseled Cap Concept to Simulated Thematic Mapper Data. <span class="html-italic">Photogramm. Eng. Remote Sens.</span> <b>1984</b>, <span class="html-italic">50</span>, 343â€“352. [<a href="https://scholar.google.com/scholar_lookup?title=Application+of+the+Tasseled+Cap+Concept+to+Simulated+Thematic+Mapper+Data&amp;author=Crist,+E.P.&amp;author=Cicone,+R.C.&amp;publication_year=1984&amp;journal=Photogramm.+Eng.+Remote+Sens.&amp;volume=50&amp;pages=343%E2%80%93352" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li><li id="B60-remotesensing-12-00080" class="html-xx" data-content="60.">Baig,
 M.H.A.; Zhang, L.; Shuai, T.; Tong, Q. Derivation of a Tasselled Cap 
Transformation Based on Landsat 8 At-Satellite Reflectance. <span class="html-italic">Remote Sens. Lett.</span> <b>2014</b>, <span class="html-italic">5</span>, 423â€“431. [<a href="https://scholar.google.com/scholar_lookup?title=Derivation+of+a+Tasselled+Cap+Transformation+Based+on+Landsat+8+At-Satellite+Reflectance&amp;author=Baig,+M.H.A.&amp;author=Zhang,+L.&amp;author=Shuai,+T.&amp;author=Tong,+Q.&amp;publication_year=2014&amp;journal=Remote+Sens.+Lett.&amp;volume=5&amp;pages=423%E2%80%93431&amp;doi=10.1080/2150704X.2014.915434" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1080/2150704X.2014.915434" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B61-remotesensing-12-00080" class="html-xx" data-content="61.">Myneni, R.B.; Hall, F.G.; Sellers, P.J.; Marshak, A.L. Interpretation of Spectral Vegetation Indexes. <span class="html-italic">IEEE Trans. Geosci. Remote Sens.</span> <b>1995</b>, <span class="html-italic">33</span>, 481â€“486. [<a href="https://scholar.google.com/scholar_lookup?title=Interpretation+of+Spectral+Vegetation+Indexes&amp;author=Myneni,+R.B.&amp;author=Hall,+F.G.&amp;author=Sellers,+P.J.&amp;author=Marshak,+A.L.&amp;publication_year=1995&amp;journal=IEEE+Trans.+Geosci.+Remote+Sens.&amp;volume=33&amp;pages=481%E2%80%93486&amp;doi=10.1109/TGRS.1995.8746029" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1109/TGRS.1995.8746029" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B62-remotesensing-12-00080" class="html-xx" data-content="62.">Parastatidis, D.; Mitraka, Z.; Chrysoulakis, N.; Abrams, M. Online Global Land Surface Temperature Estimation from Landsat. <span class="html-italic">Remote Sens.</span> <b>2017</b>, <span class="html-italic">9</span>, 1208. [<a href="https://scholar.google.com/scholar_lookup?title=Online+Global+Land+Surface+Temperature+Estimation+from+Landsat&amp;author=Parastatidis,+D.&amp;author=Mitraka,+Z.&amp;author=Chrysoulakis,+N.&amp;author=Abrams,+M.&amp;publication_year=2017&amp;journal=Remote+Sens.&amp;volume=9&amp;pages=1208&amp;doi=10.3390/rs9121208" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.3390/rs9121208" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B63-remotesensing-12-00080" class="html-xx" data-content="63.">ASTER Global Digital Elevation Map Announcement. Available online: <a href="https://asterweb.jpl.nasa.gov/gdem.asp" target="_blank" rel="noopener noreferrer">https://asterweb.jpl.nasa.gov/gdem.asp</a> (accessed on 10 May 2018).</li><li id="B64-remotesensing-12-00080" class="html-xx" data-content="64.">Hof, A.R.; Jansson, R.; Nilsson, C. The Usefulness of Elevation as a Predictor Variable in Species Distribution Modelling. <span class="html-italic">Ecol. Model.</span> <b>2012</b>, <span class="html-italic">246</span>, 86â€“90. [<a href="https://scholar.google.com/scholar_lookup?title=The+Usefulness+of+Elevation+as+a+Predictor+Variable+in+Species+Distribution+Modelling&amp;author=Hof,+A.R.&amp;author=Jansson,+R.&amp;author=Nilsson,+C.&amp;publication_year=2012&amp;journal=Ecol.+Model.&amp;volume=246&amp;pages=86%E2%80%9390&amp;doi=10.1016/j.ecolmodel.2012.07.028" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1016/j.ecolmodel.2012.07.028" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B65-remotesensing-12-00080" class="html-xx" data-content="65.">Riley, S.J.; DeGloria, S.D.; Elliot, R. A Terrain Ruggedness Index That Qauntifies Topographic Heterogeneity. <span class="html-italic">Int. J. Sci.</span> <b>1999</b>, <span class="html-italic">5</span>, 23â€“27. [<a href="https://scholar.google.com/scholar_lookup?title=A+Terrain+Ruggedness+Index+That+Qauntifies+Topographic+Heterogeneity&amp;author=Riley,+S.J.&amp;author=DeGloria,+S.D.&amp;author=Elliot,+R.&amp;publication_year=1999&amp;journal=Int.+J.+Sci.&amp;volume=5&amp;pages=23%E2%80%9327" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li><li id="B66-remotesensing-12-00080" class="html-xx" data-content="66.">Dormann,
 C.F.; Purschke, O.; MÃ¡rquez, J.R.G.; Lautenbach, S.; SchrÃ¶der, B. 
Components of Uncertainty in Species Distribution Analysis: A Case Study
 of the Great Grey Shrike. <span class="html-italic">Ecology</span> <b>2008</b>, <span class="html-italic">89</span>, 3371â€“3386. [<a href="https://scholar.google.com/scholar_lookup?title=Components+of+Uncertainty+in+Species+Distribution+Analysis:+A+Case+Study+of+the+Great+Grey+Shrike&amp;author=Dormann,+C.F.&amp;author=Purschke,+O.&amp;author=M%C3%A1rquez,+J.R.G.&amp;author=Lautenbach,+S.&amp;author=Schr%C3%B6der,+B.&amp;publication_year=2008&amp;journal=Ecology&amp;volume=89&amp;pages=3371%E2%80%933386&amp;doi=10.1890/07-1772.1" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1890/07-1772.1" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B67-remotesensing-12-00080" class="html-xx" data-content="67.">Wehn,
 S.; Johansen, L. The Distribution of the Endemic Plant Primula 
Scandinavica, at Local and National Scales, in Changing Mountainous 
Environments. <span class="html-italic">Biodiversity</span> <b>2015</b>, <span class="html-italic">16</span>, 278â€“288. [<a href="https://scholar.google.com/scholar_lookup?title=The+Distribution+of+the+Endemic+Plant+Primula+Scandinavica,+at+Local+and+National+Scales,+in+Changing+Mountainous+Environments&amp;author=Wehn,+S.&amp;author=Johansen,+L.&amp;publication_year=2015&amp;journal=Biodiversity&amp;volume=16&amp;pages=278%E2%80%93288&amp;doi=10.1080/14888386.2015.1116408" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1080/14888386.2015.1116408" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B68-remotesensing-12-00080" class="html-xx" data-content="68.">DirnbÃ¶ck, T.; Dullinger, S.; Grabherr, G. A Regional Impact Assessment of Climate and Land-Use Change on Alpine Vegetation. <span class="html-italic">J. Biogeogr.</span> <b>2003</b>, 401â€“417. [<a href="https://scholar.google.com/scholar_lookup?title=A+Regional+Impact+Assessment+of+Climate+and+Land-Use+Change+on+Alpine+Vegetation&amp;author=Dirnb%C3%B6ck,+T.&amp;author=Dullinger,+S.&amp;author=Grabherr,+G.&amp;publication_year=2003&amp;journal=J.+Biogeogr.&amp;pages=401%E2%80%93417&amp;doi=10.1046/j.1365-2699.2003.00839.x" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1046/j.1365-2699.2003.00839.x" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B69-remotesensing-12-00080" class="html-xx" data-content="69.">Pepin, N.; Kidd, D. Spatial Temperature Variation in the Eastern Pyrenees. <span class="html-italic">Weather</span> <b>2006</b>, <span class="html-italic">61</span>, 300â€“310. [<a href="https://scholar.google.com/scholar_lookup?title=Spatial+Temperature+Variation+in+the+Eastern+Pyrenees&amp;author=Pepin,+N.&amp;author=Kidd,+D.&amp;publication_year=2006&amp;journal=Weather&amp;volume=61&amp;pages=300%E2%80%93310&amp;doi=10.1256/wea.106.06" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1256/wea.106.06" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B70-remotesensing-12-00080" class="html-xx" data-content="70.">Kramer-Schadt,
 S.; Niedballa, J.; Pilgrim, J.D.; SchrÃ¶der, B.; Lindenborn, J.; 
Reinfelder, V.; Stillfried, M.; Heckmann, I.; Scharf, A.K.; Augeri, 
D.M.; et al. The Importance of Correcting for Sampling Bias in MaxEnt 
Species Distribution Models. <span class="html-italic">Divers. Distrib.</span> <b>2013</b>, <span class="html-italic">19</span>, 1366â€“1379. [<a href="https://scholar.google.com/scholar_lookup?title=The+Importance+of+Correcting+for+Sampling+Bias+in+MaxEnt+Species+Distribution+Models&amp;author=Kramer-Schadt,+S.&amp;author=Niedballa,+J.&amp;author=Pilgrim,+J.D.&amp;author=Schr%C3%B6der,+B.&amp;author=Lindenborn,+J.&amp;author=Reinfelder,+V.&amp;author=Stillfried,+M.&amp;author=Heckmann,+I.&amp;author=Scharf,+A.K.&amp;author=Augeri,+D.M.&amp;publication_year=2013&amp;journal=Divers.+Distrib.&amp;volume=19&amp;pages=1366%E2%80%931379&amp;doi=10.1111/ddi.12096" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1111/ddi.12096" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B71-remotesensing-12-00080" class="html-xx" data-content="71.">Cohen, W.B.; Goward, S.N. Landsatâ€™s Role in Ecological Applications of Remote Sensing. <span class="html-italic">Bioscience</span> <b>2004</b>, <span class="html-italic">54</span>, 535â€“545. [<a href="https://scholar.google.com/scholar_lookup?title=Landsat%E2%80%99s+Role+in+Ecological+Applications+of+Remote+Sensing&amp;author=Cohen,+W.B.&amp;author=Goward,+S.N.&amp;publication_year=2004&amp;journal=Bioscience&amp;volume=54&amp;pages=535%E2%80%93545&amp;doi=10.1641/0006-3568(2004)054[0535:LRIEAO]2.0.CO;2" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1641/0006-3568(2004)054[0535:LRIEAO]2.0.CO;2" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B72-remotesensing-12-00080" class="html-xx" data-content="72.">Carroll, C.; Noss, R.F.; Paquet, P.C. Carnivores as Focal Species for Conservation Planning in the Rocky Mountain Region. <span class="html-italic">Ecol. Appl.</span> <b>2001</b>, <span class="html-italic">11</span>, 961â€“980. [<a href="https://scholar.google.com/scholar_lookup?title=Carnivores+as+Focal+Species+for+Conservation+Planning+in+the+Rocky+Mountain+Region&amp;author=Carroll,+C.&amp;author=Noss,+R.F.&amp;author=Paquet,+P.C.&amp;publication_year=2001&amp;journal=Ecol.+Appl.&amp;volume=11&amp;pages=961%E2%80%93980&amp;doi=10.1890/1051-0761(2001)011[0961:CAFSFC]2.0.CO;2" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1890/1051-0761(2001)011[0961:CAFSFC]2.0.CO;2" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B73-remotesensing-12-00080" class="html-xx" data-content="73.">Bartel,
 R.A.; Sexton, J.O. Monitoring Habitat Dynamics for Rare and Endangered 
Species Using Satellite Images and Niche-Based Models. <span class="html-italic">Ecography</span> <b>2009</b>, <span class="html-italic">32</span>, 888â€“896. [<a href="https://scholar.google.com/scholar_lookup?title=Monitoring+Habitat+Dynamics+for+Rare+and+Endangered+Species+Using+Satellite+Images+and+Niche-Based+Models&amp;author=Bartel,+R.A.&amp;author=Sexton,+J.O.&amp;publication_year=2009&amp;journal=Ecography&amp;volume=32&amp;pages=888%E2%80%93896&amp;doi=10.1111/j.1600-0587.2009.05797.x" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1111/j.1600-0587.2009.05797.x" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B74-remotesensing-12-00080" class="html-xx" data-content="74.">Cohen,
 W.B.; Spies, T.A.; Fiorella, M. Estimating the Age and Structure of 
Forests in a Multi-Ownership Landscape of Western Oregon, U.S.A. <span class="html-italic">Int. J. Remote Sens.</span> <b>1995</b>, <span class="html-italic">16</span>, 721â€“746. [<a href="https://scholar.google.com/scholar_lookup?title=Estimating+the+Age+and+Structure+of+Forests+in+a+Multi-Ownership+Landscape+of+Western+Oregon,+U.S.A&amp;author=Cohen,+W.B.&amp;author=Spies,+T.A.&amp;author=Fiorella,+M.&amp;publication_year=1995&amp;journal=Int.+J.+Remote+Sens.&amp;volume=16&amp;pages=721%E2%80%93746&amp;doi=10.1080/01431169508954436" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1080/01431169508954436" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B75-remotesensing-12-00080" class="html-xx" data-content="75.">Aguirre-GutiÃ©rrez,
 J.; Carvalheiro, L.G.; Polce, C.; van Loon, E.E.; Raes, N.; Reemer, M.;
 Biesmeijer, J.C. Fit-for-Purpose: Species Distribution Model 
Performance Depends on Evaluation Criteriaâ€”Dutch Hoverflies as a Case 
Study. <span class="html-italic">PLoS ONE</span> <b>2013</b>, <span class="html-italic">8</span>, e63708. [<a href="https://scholar.google.com/scholar_lookup?title=Fit-for-Purpose:+Species+Distribution+Model+Performance+Depends+on+Evaluation+Criteria%E2%80%94Dutch+Hoverflies+as+a+Case+Study&amp;author=Aguirre-Guti%C3%A9rrez,+J.&amp;author=Carvalheiro,+L.G.&amp;author=Polce,+C.&amp;author=van+Loon,+E.E.&amp;author=Raes,+N.&amp;author=Reemer,+M.&amp;author=Biesmeijer,+J.C.&amp;publication_year=2013&amp;journal=PLoS+ONE&amp;volume=8&amp;pages=e63708&amp;doi=10.1371/journal.pone.0063708" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1371/journal.pone.0063708" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B76-remotesensing-12-00080" class="html-xx" data-content="76.">Thuiller,
 W.; Georges, D.; Engler, R. Biomod2: Ensemble Platform for Species 
Distribution Modeling. R Package Version 3.1-48. 2014. Available online:
 <a href="https://cran.r-project.org/package=biomod2" target="_blank" rel="noopener noreferrer">http://cran.r-project.org/package=biomod2</a> (accessed on 20 May 2019).</li><li id="B77-remotesensing-12-00080" class="html-xx" data-content="77.">R Core Team. <span class="html-italic">R: A Language and Environment for Statistical Computing</span>; R Foundation for Statistical Computing: Vienna, Austria, 2014; Available online: <a href="https://www.r-project.org/" target="_blank" rel="noopener noreferrer">http://www.r-project.org/</a> (accessed on 6 May 2019).</li><li id="B78-remotesensing-12-00080" class="html-xx" data-content="78.">Hijmans, R.J. Cross-Validation of Species Distribution Models: Removing Spatial Sorting Bias and Calibration with a Null Model. <span class="html-italic">Ecology</span> <b>2012</b>, <span class="html-italic">93</span>, 679â€“688. [<a href="https://scholar.google.com/scholar_lookup?title=Cross-Validation+of+Species+Distribution+Models:+Removing+Spatial+Sorting+Bias+and+Calibration+with+a+Null+Model&amp;author=Hijmans,+R.J.&amp;publication_year=2012&amp;journal=Ecology&amp;volume=93&amp;pages=679%E2%80%93688&amp;doi=10.1890/11-0826.1" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1890/11-0826.1" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B79-remotesensing-12-00080" class="html-xx" data-content="79.">Anderson, K.P.; Burnham, D.A. <span class="html-italic">Model Selection and Multi-Model Inference: A Practical Information-Theoretic Approach</span>, 2nd ed.; Springer: Berlin/Heidelberg, Germany, 2002. [<a href="https://scholar.google.com/scholar_lookup?title=Model+Selection+and+Multi-Model+Inference:+A+Practical+Information-Theoretic+Approach&amp;author=Anderson,+K.P.&amp;author=Burnham,+D.A.&amp;publication_year=2002" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1016/j.ecolmodel.2003.11.004" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B80-remotesensing-12-00080" class="html-xx" data-content="80.">Wood, S.N. MGCV: Mixed GAM Computation Vehicle with Automatic Smoothness Estimation; R Package Version 1.8. Available online: <a href="https://cran.r-project.org/web/packages/mgcv/mgcv.pdf" target="_blank" rel="noopener noreferrer">https://cran.r-project.org/web/packages/mgcv/mgcv.pdf</a> (accessed on 20 October 2019).</li><li id="B81-remotesensing-12-00080" class="html-xx" data-content="81.">Merow,
 C.; Smith, M.J.; Silander, J.A. A Practical Guide to MaxEnt for 
Modeling Speciesâ€™ Distributions: What It Does, and Why Inputs and 
Settings Matter. <span class="html-italic">Ecography</span> <b>2013</b>, <span class="html-italic">36</span>, 1058â€“1069. [<a href="https://scholar.google.com/scholar_lookup?title=A+Practical+Guide+to+MaxEnt+for+Modeling+Species%E2%80%99+Distributions:+What+It+Does,+and+Why+Inputs+and+Settings+Matter&amp;author=Merow,+C.&amp;author=Smith,+M.J.&amp;author=Silander,+J.A.&amp;publication_year=2013&amp;journal=Ecography&amp;volume=36&amp;pages=1058%E2%80%931069&amp;doi=10.1111/j.1600-0587.2013.07872.x" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1111/j.1600-0587.2013.07872.x" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B82-remotesensing-12-00080" class="html-xx" data-content="82.">Phillips, S.J.; Anderson, R.P.; Schapire, R.E. Maximum Entropy Modeling of Species Geographic Distributions. <span class="html-italic">Ecol. Model.</span> <b>2006</b>, <span class="html-italic">190</span>, 231â€“259. [<a href="https://scholar.google.com/scholar_lookup?title=Maximum+Entropy+Modeling+of+Species+Geographic+Distributions&amp;author=Phillips,+S.J.&amp;author=Anderson,+R.P.&amp;author=Schapire,+R.E.&amp;publication_year=2006&amp;journal=Ecol.+Model.&amp;volume=190&amp;pages=231%E2%80%93259&amp;doi=10.1016/j.ecolmodel.2005.03.026" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1016/j.ecolmodel.2005.03.026" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B83-remotesensing-12-00080" class="html-xx" data-content="83.">Greenwell,
 B.; Boehmke, B.; Cunningham, J. GBM: Generalized Boosted Regression 
Models; R Package Version 2.1.5. Available online: <a href="https://cran.r-project.org/web/packages/gbm/gbm.pdf" target="_blank" rel="noopener noreferrer">https://cran.r-project.org/web/packages/gbm/gbm.pdf</a> (accessed on 20 October 2019).</li><li id="B84-remotesensing-12-00080" class="html-xx" data-content="84.">Elith,
 J.; Ferrier, S.; Huettmann, F.; Leathwick, J. The Evaluation Strip: A 
New and Robust Method for Plotting Predicted Responses from Species 
Distribution Models. <span class="html-italic">Ecol. Model.</span> <b>2005</b>, <span class="html-italic">186</span>, 280â€“289. [<a href="https://scholar.google.com/scholar_lookup?title=The+Evaluation+Strip:+A+New+and+Robust+Method+for+Plotting+Predicted+Responses+from+Species+Distribution+Models&amp;author=Elith,+J.&amp;author=Ferrier,+S.&amp;author=Huettmann,+F.&amp;author=Leathwick,+J.&amp;publication_year=2005&amp;journal=Ecol.+Model.&amp;volume=186&amp;pages=280%E2%80%93289&amp;doi=10.1016/j.ecolmodel.2004.12.007" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1016/j.ecolmodel.2004.12.007" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B85-remotesensing-12-00080" class="html-xx" data-content="85.">Peterson,
 A.T.; PapeÅŸ, M.; SoberÃ³n, J. Rethinking Receiver Operating 
Characteristic Analysis Applications in Ecological Niche Modeling. <span class="html-italic">Ecol. Model.</span> <b>2008</b>, <span class="html-italic">213</span>, 63â€“72. [<a href="https://scholar.google.com/scholar_lookup?title=Rethinking+Receiver+Operating+Characteristic+Analysis+Applications+in+Ecological+Niche+Modeling&amp;author=Peterson,+A.T.&amp;author=Pape%C5%9F,+M.&amp;author=Sober%C3%B3n,+J.&amp;publication_year=2008&amp;journal=Ecol.+Model.&amp;volume=213&amp;pages=63%E2%80%9372&amp;doi=10.1016/j.ecolmodel.2007.11.008" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1016/j.ecolmodel.2007.11.008" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B86-remotesensing-12-00080" class="html-xx" data-content="86.">Swets, J.A. Measuring the Accuracy of Diagnostic Systems. <span class="html-italic">Science</span> <b>1988</b>, <span class="html-italic">240</span>, 1285â€“1293. [<a href="https://scholar.google.com/scholar_lookup?title=Measuring+the+Accuracy+of+Diagnostic+Systems&amp;author=Swets,+J.A.&amp;publication_year=1988&amp;journal=Science&amp;volume=240&amp;pages=1285%E2%80%931293&amp;doi=10.1126/science.3287615" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1126/science.3287615" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B87-remotesensing-12-00080" class="html-xx" data-content="87.">Allouche,
 O.; Tsoar, A.; Kadmon, R. Assessing the Accuracy of Species 
Distribution Models: Prevalence, Kappa and the True Skill Statistic 
(TSS). <span class="html-italic">J. Appl. Ecol.</span> <b>2006</b>, <span class="html-italic">43</span>, 1223â€“1232. [<a href="https://scholar.google.com/scholar_lookup?title=Assessing+the+Accuracy+of+Species+Distribution+Models:+Prevalence,+Kappa+and+the+True+Skill+Statistic+(TSS)&amp;author=Allouche,+O.&amp;author=Tsoar,+A.&amp;author=Kadmon,+R.&amp;publication_year=2006&amp;journal=J.+Appl.+Ecol.&amp;volume=43&amp;pages=1223%E2%80%931232&amp;doi=10.1111/j.1365-2664.2006.01214.x" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1111/j.1365-2664.2006.01214.x" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B88-remotesensing-12-00080" class="html-xx" data-content="88.">GonzÃ¡lez-Ferreras,
 A.M.; BarquÃ­n, J.; PeÃ±as, F.J. Integration of Habitat Models to Predict
 Fish Distributions in Several Watersheds of Northern Spain. <span class="html-italic">J. Appl. Ichthyol.</span> <b>2016</b>, <span class="html-italic">32</span>, 204â€“216. [<a href="https://scholar.google.com/scholar_lookup?title=Integration+of+Habitat+Models+to+Predict+Fish+Distributions+in+Several+Watersheds+of+Northern+Spain&amp;author=Gonz%C3%A1lez-Ferreras,+A.M.&amp;author=Barqu%C3%ADn,+J.&amp;author=Pe%C3%B1as,+F.J.&amp;publication_year=2016&amp;journal=J.+Appl.+Ichthyol.&amp;volume=32&amp;pages=204%E2%80%93216&amp;doi=10.1111/jai.13024" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1111/jai.13024" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B89-remotesensing-12-00080" class="html-xx" data-content="89.">Elith,
 J.; Burgman, M.A.; Regan, H.M. Mapping Epistemic Uncertainties and 
Vague Concepts in Predictions of Species Distribution. <span class="html-italic">Ecol. Model.</span> <b>2002</b>, <span class="html-italic">157</span>, 313â€“329. [<a href="https://scholar.google.com/scholar_lookup?title=Mapping+Epistemic+Uncertainties+and+Vague+Concepts+in+Predictions+of+Species+Distribution&amp;author=Elith,+J.&amp;author=Burgman,+M.A.&amp;author=Regan,+H.M.&amp;publication_year=2002&amp;journal=Ecol.+Model.&amp;volume=157&amp;pages=313%E2%80%93329&amp;doi=10.1016/S0304-3800(02)00202-8" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1016/S0304-3800(02)00202-8" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B90-remotesensing-12-00080" class="html-xx" data-content="90.">Bonn,
 A.; SchrÃ¶der, B. Habitat Models and Their Transfer for Single and Multi
 Species Groups: A Case Study of Carabids in an Alluvial Forest. <span class="html-italic">Ecography</span> <b>2008</b>, <span class="html-italic">24</span>, 483â€“496. [<a href="https://scholar.google.com/scholar_lookup?title=Habitat+Models+and+Their+Transfer+for+Single+and+Multi+Species+Groups:+A+Case+Study+of+Carabids+in+an+Alluvial+Forest&amp;author=Bonn,+A.&amp;author=Schr%C3%B6der,+B.&amp;publication_year=2008&amp;journal=Ecography&amp;volume=24&amp;pages=483%E2%80%93496&amp;doi=10.1111/j.1600-0587.2001.tb00483.x" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1111/j.1600-0587.2001.tb00483.x" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B91-remotesensing-12-00080" class="html-xx" data-content="91.">De
 Groot, M.; RebeuÅ¡ek, F.; Grobelnik, V.; GovediÄ, M.; Å alamun, A.; 
Verovnik, R. Distribution Modelling as an Approach to the Conservation 
of a Threatened Alpine Endemic Butterfly (Lepidoptera: Satyridae). <span class="html-italic">Eur. J. Entomol.</span> <b>2009</b>, <span class="html-italic">106</span>, 77â€“84. [<a href="https://scholar.google.com/scholar_lookup?title=Distribution+Modelling+as+an+Approach+to+the+Conservation+of+a+Threatened+Alpine+Endemic+Butterfly+(Lepidoptera:+Satyridae)&amp;author=De+Groot,+M.&amp;author=Rebeu%C5%A1ek,+F.&amp;author=Grobelnik,+V.&amp;author=Govedi%C4%8D,+M.&amp;author=%C5%A0alamun,+A.&amp;author=Verovnik,+R.&amp;publication_year=2009&amp;journal=Eur.+J.+Entomol.&amp;volume=106&amp;pages=77%E2%80%9384&amp;doi=10.14411/eje.2009.012" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.14411/eje.2009.012" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B92-remotesensing-12-00080" class="html-xx" data-content="92.">Porfirio,
 L.L.; Harris, R.M.B.; Lefroy, E.C.; Hugh, S.; Gould, S.F.; Lee, G.; 
Bindoff, N.L.; Mackey, B. Improving the Use of Species Distribution 
Models in Conservation Planning and Management under Climate Change. <span class="html-italic">PLoS ONE</span> <b>2014</b>, <span class="html-italic">9</span>, e113749. [<a href="https://scholar.google.com/scholar_lookup?title=Improving+the+Use+of+Species+Distribution+Models+in+Conservation+Planning+and+Management+under+Climate+Change&amp;author=Porfirio,+L.L.&amp;author=Harris,+R.M.B.&amp;author=Lefroy,+E.C.&amp;author=Hugh,+S.&amp;author=Gould,+S.F.&amp;author=Lee,+G.&amp;author=Bindoff,+N.L.&amp;author=Mackey,+B.&amp;publication_year=2014&amp;journal=PLoS+ONE&amp;volume=9&amp;pages=e113749&amp;doi=10.1371/journal.pone.0113749&amp;pmid=25420020" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1371/journal.pone.0113749" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>] [<a href="https://www.ncbi.nlm.nih.gov/pubmed/25420020" class="cross-ref" data-typ="pmid" target="_blank" rel="noopener noreferrer">PubMed</a>]</li><li id="B93-remotesensing-12-00080" class="html-xx" data-content="93.">Luoto, M.; Kuussaari, M.; Toivonen, T. Modelling Butterfly Distribution Based on Remote Sensing Data. <span class="html-italic">J. Biogeogr.</span> <b>2002</b>, <span class="html-italic">29</span>, 1027â€“1037. [<a href="https://scholar.google.com/scholar_lookup?title=Modelling+Butterfly+Distribution+Based+on+Remote+Sensing+Data&amp;author=Luoto,+M.&amp;author=Kuussaari,+M.&amp;author=Toivonen,+T.&amp;publication_year=2002&amp;journal=J.+Biogeogr.&amp;volume=29&amp;pages=1027%E2%80%931037&amp;doi=10.1046/j.1365-2699.2002.00728.x" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1046/j.1365-2699.2002.00728.x" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B94-remotesensing-12-00080" class="html-xx" data-content="94.">Eyre,
 M.D.; Rushton, S.P.; Luff, M.L.; Telfer, M.G. Predicting the 
Distributions of Ground Beetle Species (Coleoptera, Carabidae) in 
Britain Using Land Cover Variables. <span class="html-italic">J. Environ. Manag.</span> <b>2004</b>, <span class="html-italic">72</span>, 163â€“174. [<a href="https://scholar.google.com/scholar_lookup?title=Predicting+the+Distributions+of+Ground+Beetle+Species+(Coleoptera,+Carabidae)+in+Britain+Using+Land+Cover+Variables&amp;author=Eyre,+M.D.&amp;author=Rushton,+S.P.&amp;author=Luff,+M.L.&amp;author=Telfer,+M.G.&amp;publication_year=2004&amp;journal=J.+Environ.+Manag.&amp;volume=72&amp;pages=163%E2%80%93174&amp;doi=10.1016/j.jenvman.2004.04.007" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1016/j.jenvman.2004.04.007" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B95-remotesensing-12-00080" class="html-xx" data-content="95.">Heikkinen,
 R.K.; Luoto, M.; Kuussaari, M.; Toivonen, T. Modelling the Spatial 
Distribution of a Threatened Butterfly: Impacts of Scale and Statistical
 Technique. <span class="html-italic">Landsc. Urban Plan.</span> <b>2007</b>, <span class="html-italic">79</span>, 347â€“357. [<a href="https://scholar.google.com/scholar_lookup?title=Modelling+the+Spatial+Distribution+of+a+Threatened+Butterfly:+Impacts+of+Scale+and+Statistical+Technique&amp;author=Heikkinen,+R.K.&amp;author=Luoto,+M.&amp;author=Kuussaari,+M.&amp;author=Toivonen,+T.&amp;publication_year=2007&amp;journal=Landsc.+Urban+Plan.&amp;volume=79&amp;pages=347%E2%80%93357&amp;doi=10.1016/j.landurbplan.2006.04.002" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1016/j.landurbplan.2006.04.002" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B96-remotesensing-12-00080" class="html-xx" data-content="96.">Widenfalk,
 L.A.; AhrnÃ©, K.; Berggren, Ã…. Using Citizen-Reported Data to Predict 
Distributions of Two Non-Native Insect Species in Sweden. <span class="html-italic">Ecosphere</span> <b>2014</b>, <span class="html-italic">5</span>, 1â€“16. [<a href="https://scholar.google.com/scholar_lookup?title=Using+Citizen-Reported+Data+to+Predict+Distributions+of+Two+Non-Native+Insect+Species+in+Sweden&amp;author=Widenfalk,+L.A.&amp;author=Ahrn%C3%A9,+K.&amp;author=Berggren,+%C3%85.&amp;publication_year=2014&amp;journal=Ecosphere&amp;volume=5&amp;pages=1%E2%80%9316&amp;doi=10.1890/ES14-00212.1" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1890/ES14-00212.1" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B97-remotesensing-12-00080" class="html-xx" data-content="97.">Romo,
 H.; GarcÃ­a-Barros, E.; MÃ¡rquez, A.L.; Moreno, J.C.; Real, R. Effects of
 Climate Change on the Distribution of Ecologically Interacting Species:
 Butterflies and Their Main Food Plants in Spain. <span class="html-italic">Ecography</span> <b>2014</b>, <span class="html-italic">37</span>, 1063â€“1072. [<a href="https://scholar.google.com/scholar_lookup?title=Effects+of+Climate+Change+on+the+Distribution+of+Ecologically+Interacting+Species:+Butterflies+and+Their+Main+Food+Plants+in+Spain&amp;author=Romo,+H.&amp;author=Garc%C3%ADa-Barros,+E.&amp;author=M%C3%A1rquez,+A.L.&amp;author=Moreno,+J.C.&amp;author=Real,+R.&amp;publication_year=2014&amp;journal=Ecography&amp;volume=37&amp;pages=1063%E2%80%931072&amp;doi=10.1111/ecog.00706" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1111/ecog.00706" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B98-remotesensing-12-00080" class="html-xx" data-content="98.">Grenouillet,
 G.; Buisson, L.; Casajus, N.; Lek, S. Ensemble Modelling of Species 
Distribution: The Effects of Geographical and Environmental Ranges. <span class="html-italic">Ecography</span> <b>2011</b>, <span class="html-italic">34</span>, 9â€“17. [<a href="https://scholar.google.com/scholar_lookup?title=Ensemble+Modelling+of+Species+Distribution:+The+Effects+of+Geographical+and+Environmental+Ranges&amp;author=Grenouillet,+G.&amp;author=Buisson,+L.&amp;author=Casajus,+N.&amp;author=Lek,+S.&amp;publication_year=2011&amp;journal=Ecography&amp;volume=34&amp;pages=9%E2%80%9317&amp;doi=10.1111/j.1600-0587.2010.06152.x" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1111/j.1600-0587.2010.06152.x" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B99-remotesensing-12-00080" class="html-xx" data-content="99.">Wisz,
 M.S.; Guisan, A. Do Pseudo-Absence Selection Strategies Influence 
Species Distribution Models and Their Predictions? An 
Information-Theoretic Approach Based on Simulated Data. <span class="html-italic">BMC Ecol.</span> <b>2009</b>, <span class="html-italic">9</span>, 8. [<a href="https://scholar.google.com/scholar_lookup?title=Do+Pseudo-Absence+Selection+Strategies+Influence+Species+Distribution+Models+and+Their+Predictions?+An+Information-Theoretic+Approach+Based+on+Simulated+Data&amp;author=Wisz,+M.S.&amp;author=Guisan,+A.&amp;publication_year=2009&amp;journal=BMC+Ecol.&amp;volume=9&amp;pages=8&amp;doi=10.1186/1472-6785-9-8&amp;pmid=19393082" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1186/1472-6785-9-8" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>] [<a href="https://www.ncbi.nlm.nih.gov/pubmed/19393082" class="cross-ref" data-typ="pmid" target="_blank" rel="noopener noreferrer">PubMed</a>]</li><li id="B100-remotesensing-12-00080" class="html-xxx" data-content="100.">Barbet-Massin,
 M.; Jiguet, F.; Albert, C.H.; Thuiller, W. Selecting Pseudo-Absences 
for Species Distribution Models: How, Where and How Many? <span class="html-italic">Methods Ecol. Evol.</span> <b>2012</b>, <span class="html-italic">3</span>, 327â€“338. [<a href="https://scholar.google.com/scholar_lookup?title=Selecting+Pseudo-Absences+for+Species+Distribution+Models:+How,+Where+and+How+Many?&amp;author=Barbet-Massin,+M.&amp;author=Jiguet,+F.&amp;author=Albert,+C.H.&amp;author=Thuiller,+W.&amp;publication_year=2012&amp;journal=Methods+Ecol.+Evol.&amp;volume=3&amp;pages=327%E2%80%93338&amp;doi=10.1111/j.2041-210X.2011.00172.x" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1111/j.2041-210X.2011.00172.x" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B101-remotesensing-12-00080" class="html-xxx" data-content="101.">Krueger, T.; Page, T.; Hubacek, K.; Smith, L.; Hiscock, K. The Role of Expert Opinion in Environmental Modelling. <span class="html-italic">Environ. Model. Softw.</span> <b>2012</b>, <span class="html-italic">36</span>, 4â€“18. [<a href="https://scholar.google.com/scholar_lookup?title=The+Role+of+Expert+Opinion+in+Environmental+Modelling&amp;author=Krueger,+T.&amp;author=Page,+T.&amp;author=Hubacek,+K.&amp;author=Smith,+L.&amp;author=Hiscock,+K.&amp;publication_year=2012&amp;journal=Environ.+Model.+Softw.&amp;volume=36&amp;pages=4%E2%80%9318&amp;doi=10.1016/j.envsoft.2012.01.011" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1016/j.envsoft.2012.01.011" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B102-remotesensing-12-00080" class="html-xxx" data-content="102.">BÃ©lisle, A.C.; Asselin, H.; Leblanc, P.; Gauthier, S. Local Knowledge in Ecological Modeling. <span class="html-italic">Ecol. Soc.</span> <b>2018</b>, <span class="html-italic">23</span>, 14. [<a href="https://scholar.google.com/scholar_lookup?title=Local+Knowledge+in+Ecological+Modeling&amp;author=B%C3%A9lisle,+A.C.&amp;author=Asselin,+H.&amp;author=Leblanc,+P.&amp;author=Gauthier,+S.&amp;publication_year=2018&amp;journal=Ecol.+Soc.&amp;volume=23&amp;pages=14&amp;doi=10.5751/ES-09949-230214" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.5751/ES-09949-230214" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B103-remotesensing-12-00080" class="html-xxx" data-content="103.">Anderson, C.B. Biodiversity Monitoring, Earth Observations and the Ecology of Scale. <span class="html-italic">Ecol. Lett.</span> <b>2018</b>, <span class="html-italic">21</span>, 1572â€“1585. [<a href="https://scholar.google.com/scholar_lookup?title=Biodiversity+Monitoring,+Earth+Observations+and+the+Ecology+of+Scale&amp;author=Anderson,+C.B.&amp;publication_year=2018&amp;journal=Ecol.+Lett.&amp;volume=21&amp;pages=1572%E2%80%931585&amp;doi=10.1111/ele.13106" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1111/ele.13106" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B104-remotesensing-12-00080" class="html-xxx" data-content="104.">Guisan,
 A.; Graham, C.H.; Elith, J.; Huettmann, F.; Dudik, M.; Ferrier, S.; 
Hijmans, R.; Lehmann, A.; Li, J.; Lohmann, L.G.; et al. Sensitivity of 
Predictive Species Distribution Models to Change in Grain Size. <span class="html-italic">Divers. Distrib.</span> <b>2007</b>, <span class="html-italic">13</span>, 332â€“340. [<a href="https://scholar.google.com/scholar_lookup?title=Sensitivity+of+Predictive+Species+Distribution+Models+to+Change+in+Grain+Size&amp;author=Guisan,+A.&amp;author=Graham,+C.H.&amp;author=Elith,+J.&amp;author=Huettmann,+F.&amp;author=Dudik,+M.&amp;author=Ferrier,+S.&amp;author=Hijmans,+R.&amp;author=Lehmann,+A.&amp;author=Li,+J.&amp;author=Lohmann,+L.G.&amp;publication_year=2007&amp;journal=Divers.+Distrib.&amp;volume=13&amp;pages=332%E2%80%93340&amp;doi=10.1111/j.1472-4642.2007.00342.x" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1111/j.1472-4642.2007.00342.x" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B105-remotesensing-12-00080" class="html-xxx" data-content="105.">Thiele, H.-U. <span class="html-italic">Carabid Beetles in Their Environments</span>; Springer: Berlin/Heidelberg, Germany, 1977; ISBN 978-3-642-81156-2. [<a href="https://scholar.google.com/scholar_lookup?title=Carabid+Beetles+in+Their+Environments&amp;author=Thiele,+H.-U.&amp;publication_year=1977" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li><li id="B106-remotesensing-12-00080" class="html-xxx" data-content="106.">Saska,
 P.; van der Werf, W.; Hemerik, L.; Luff, M.L.; Hatten, T.D.; Honek, A. 
Temperature Effects on Pitfall Catches of Epigeal Arthropods: A Model 
and Method for Bias Correction. <span class="html-italic">J. Appl. Ecol.</span> <b>2013</b>, <span class="html-italic">50</span>, 181â€“189. [<a href="https://scholar.google.com/scholar_lookup?title=Temperature+Effects+on+Pitfall+Catches+of+Epigeal+Arthropods:+A+Model+and+Method+for+Bias+Correction&amp;author=Saska,+P.&amp;author=van+der+Werf,+W.&amp;author=Hemerik,+L.&amp;author=Luff,+M.L.&amp;author=Hatten,+T.D.&amp;author=Honek,+A.&amp;publication_year=2013&amp;journal=J.+Appl.+Ecol.&amp;volume=50&amp;pages=181%E2%80%93189&amp;doi=10.1111/1365-2664.12023" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1111/1365-2664.12023" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B107-remotesensing-12-00080" class="html-xxx" data-content="107.">Southwood, T.R.E.; Henderson, P.A. <span class="html-italic">Ecological Methods</span>, 3rd ed.; Blackwell Science: Hoboken, NJ, USA, 2000; ISBN 978-0-632-05477-0. [<a href="https://scholar.google.com/scholar_lookup?title=Ecological+Methods&amp;author=Southwood,+T.R.E.&amp;author=Henderson,+P.A.&amp;publication_year=2000" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li><li id="B108-remotesensing-12-00080" class="html-xxx" data-content="108.">Buse,
 A.; Good, J.E.G. The Effects of Conifer Forest Design and Management on
 Abundance and Diversity of Rove Beetles (Coleoptera: Staphylinidae): 
Implications for Conservation. <span class="html-italic">Biol. Conserv.</span> <b>1993</b>, <span class="html-italic">64</span>, 67â€“76. [<a href="https://scholar.google.com/scholar_lookup?title=The+Effects+of+Conifer+Forest+Design+and+Management+on+Abundance+and+Diversity+of+Rove+Beetles+(Coleoptera:+Staphylinidae):+Implications+for+Conservation&amp;author=Buse,+A.&amp;author=Good,+J.E.G.&amp;publication_year=1993&amp;journal=Biol.+Conserv.&amp;volume=64&amp;pages=67%E2%80%9376&amp;doi=10.1016/0006-3207(93)90384-D" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1016/0006-3207(93)90384-D" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B109-remotesensing-12-00080" class="html-xxx" data-content="109.">Hoffmann,
 H.; Michalik, P.; GÃ¶rn, S.; Fischer, K. Effects of Fen Management and 
Habitat Parameters on Staphylinid Beetle (Coleoptera: Staphylinidae) 
Assemblages in North-Eastern Germany. <span class="html-italic">Insect Conserv.</span> <b>2016</b>, <span class="html-italic">20</span>, 129â€“139. [<a href="https://scholar.google.com/scholar_lookup?title=Effects+of+Fen+Management+and+Habitat+Parameters+on+Staphylinid+Beetle+(Coleoptera:+Staphylinidae)+Assemblages+in+North-Eastern+Germany&amp;author=Hoffmann,+H.&amp;author=Michalik,+P.&amp;author=G%C3%B6rn,+S.&amp;author=Fischer,+K.&amp;publication_year=2016&amp;journal=Insect+Conserv.&amp;volume=20&amp;pages=129%E2%80%93139&amp;doi=10.1007/s10841-016-9847-0" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1007/s10841-016-9847-0" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B110-remotesensing-12-00080" class="html-xxx" data-content="110.">Fontana,
 F.; Rixen, C.; Jonas, T.; Aberegg, G.; Wunderle, S. Alpine Grassland 
Phenology as Seen in AVHRR, VEGETATION, and MODIS NDVI Time Seriesâ€”A 
Comparison with in Situ Measurements. <span class="html-italic">Sensors</span> <b>2008</b>, <span class="html-italic">8</span>, 2833â€“2853. [<a href="https://scholar.google.com/scholar_lookup?title=Alpine+Grassland+Phenology+as+Seen+in+AVHRR,+VEGETATION,+and+MODIS+NDVI+Time+Series%E2%80%94A+Comparison+with+in+Situ+Measurements&amp;author=Fontana,+F.&amp;author=Rixen,+C.&amp;author=Jonas,+T.&amp;author=Aberegg,+G.&amp;author=Wunderle,+S.&amp;publication_year=2008&amp;journal=Sensors&amp;volume=8&amp;pages=2833%E2%80%932853&amp;doi=10.3390/s8042833&amp;pmid=27879852" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.3390/s8042833" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>] [<a href="https://www.ncbi.nlm.nih.gov/pubmed/27879852" class="cross-ref" data-typ="pmid" target="_blank" rel="noopener noreferrer">PubMed</a>]</li><li id="B111-remotesensing-12-00080" class="html-xxx" data-content="111.">Bino,
 G.; Levin, N.; Darawshi, S.; Van Der Hal, N.; Reich-Solomon, A.; Kark, 
S. Accurate Prediction of Bird Species Richness Patterns in an Urban 
Environment Using Landsat-Derived NDVI and Spectral Unmixing. <span class="html-italic">Int. J. Remote Sens.</span> <b>2008</b>, <span class="html-italic">29</span>, 3675â€“3700. [<a href="https://scholar.google.com/scholar_lookup?title=Accurate+Prediction+of+Bird+Species+Richness+Patterns+in+an+Urban+Environment+Using+Landsat-Derived+NDVI+and+Spectral+Unmixing&amp;author=Bino,+G.&amp;author=Levin,+N.&amp;author=Darawshi,+S.&amp;author=Van+Der+Hal,+N.&amp;author=Reich-Solomon,+A.&amp;author=Kark,+S.&amp;publication_year=2008&amp;journal=Int.+J.+Remote+Sens.&amp;volume=29&amp;pages=3675%E2%80%933700&amp;doi=10.1080/01431160701772534" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1080/01431160701772534" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B112-remotesensing-12-00080" class="html-xxx" data-content="112.">Sheeren,
 D.; Bonthoux, S.; Balent, G. Modeling Bird Communities Using 
Unclassified Remote Sensing Imagery: Effects of the Spatial Resolution 
and Data Period. <span class="html-italic">Ecol. Indic.</span> <b>2014</b>, <span class="html-italic">43</span>, 69â€“82. [<a href="https://scholar.google.com/scholar_lookup?title=Modeling+Bird+Communities+Using+Unclassified+Remote+Sensing+Imagery:+Effects+of+the+Spatial+Resolution+and+Data+Period&amp;author=Sheeren,+D.&amp;author=Bonthoux,+S.&amp;author=Balent,+G.&amp;publication_year=2014&amp;journal=Ecol.+Indic.&amp;volume=43&amp;pages=69%E2%80%9382&amp;doi=10.1016/j.ecolind.2014.02.023" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1016/j.ecolind.2014.02.023" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B113-remotesensing-12-00080" class="html-xxx" data-content="113.">Pickens, B.A.; King, S.L. Linking Multi-Temporal Satellite Imagery to Coastal Wetland Dynamics and Bird Distribution. <span class="html-italic">Ecol. Model.</span> <b>2014</b>, <span class="html-italic">285</span>, 1â€“12. [<a href="https://scholar.google.com/scholar_lookup?title=Linking+Multi-Temporal+Satellite+Imagery+to+Coastal+Wetland+Dynamics+and+Bird+Distribution&amp;author=Pickens,+B.A.&amp;author=King,+S.L.&amp;publication_year=2014&amp;journal=Ecol.+Model.&amp;volume=285&amp;pages=1%E2%80%9312&amp;doi=10.1016/j.ecolmodel.2014.04.013" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1016/j.ecolmodel.2014.04.013" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B114-remotesensing-12-00080" class="html-xxx" data-content="114.">Cord,
 A.; Klein, D.; Dech, S. The Impact of Inter-Annual Variability in 
Remote Sensing Time Series on Modelling Tree Species Distributions. In 
Proceedings of the 6th International Workshop on the Analysis of 
Multi-Temporal Remote Sensing Images (Multi-Temp), Trento, Italy, 12â€“14 
July 2011. [<a href="https://scholar.google.com/scholar_lookup?title=The+Impact+of+Inter-Annual+Variability+in+Remote+Sensing+Time+Series+on+Modelling+Tree+Species+Distributions&amp;conference=Proceedings+of+the+6th+International+Workshop+on+the+Analysis+of+Multi-Temporal+Remote+Sensing+Images+(Multi-Temp)&amp;author=Cord,+A.&amp;author=Klein,+D.&amp;author=Dech,+S.&amp;publication_year=2011&amp;doi=10.1109/Multi-Temp.2011.6005078" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1109/Multi-Temp.2011.6005078" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B115-remotesensing-12-00080" class="html-xxx" data-content="115.">Young,
 N.E.; Anderson, R.S.; Chignell, S.M.; Vorster, A.G.; Lawrence, R.; 
Evangelista, P.H. A Survival Guide to Landsat Preprocessing. <span class="html-italic">Ecology</span> <b>2017</b>, <span class="html-italic">98</span>, 920â€“932. [<a href="https://scholar.google.com/scholar_lookup?title=A+Survival+Guide+to+Landsat+Preprocessing&amp;author=Young,+N.E.&amp;author=Anderson,+R.S.&amp;author=Chignell,+S.M.&amp;author=Vorster,+A.G.&amp;author=Lawrence,+R.&amp;author=Evangelista,+P.H.&amp;publication_year=2017&amp;journal=Ecology&amp;volume=98&amp;pages=920%E2%80%93932&amp;doi=10.1002/ecy.1730" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1002/ecy.1730" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B116-remotesensing-12-00080" class="html-xxx" data-content="116.">Zanetti,
 A.; Tagliapietra, A. Studi Sulle Taxocenosi Di Staphylinidae in Boschi 
Di Latifoglie Italiani (Coleoptera, Staphylinidae). <span class="html-italic">Stud. Trentini Sci. Nat. Acta Biol.</span> <b>2004</b>, <span class="html-italic">81</span>, 207â€“231. [<a href="https://scholar.google.com/scholar_lookup?title=Studi+Sulle+Taxocenosi+Di+Staphylinidae+in+Boschi+Di+Latifoglie+Italiani+(Coleoptera,+Staphylinidae)&amp;author=Zanetti,+A.&amp;author=Tagliapietra,+A.&amp;publication_year=2004&amp;journal=Stud.+Trentini+Sci.+Nat.+Acta+Biol.&amp;volume=81&amp;pages=207%E2%80%93231" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li><li id="B117-remotesensing-12-00080" class="html-xxx" data-content="117.">Balog,
 A.; MarkÃ³, V.; Imre, A. Farming System and Habitat Structure Effects on
 Rove Beetles (Coleoptera: Staphylinidae) Assembly in Central European 
Apple and Pear Orchards. <span class="html-italic">Biologia</span> <b>2009</b>, <span class="html-italic">64</span>, 343â€“349. [<a href="https://scholar.google.com/scholar_lookup?title=Farming+System+and+Habitat+Structure+Effects+on+Rove+Beetles+(Coleoptera:+Staphylinidae)+Assembly+in+Central+European+Apple+and+Pear+Orchards&amp;author=Balog,+A.&amp;author=Mark%C3%B3,+V.&amp;author=Imre,+A.&amp;publication_year=2009&amp;journal=Biologia&amp;volume=64&amp;pages=343%E2%80%93349&amp;doi=10.2478/s11756-009-0045-3" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.2478/s11756-009-0045-3" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B118-remotesensing-12-00080" class="html-xxx" data-content="118.">Lupi,
 D.; Colombo, M.; Facchini, S. The Ground Beetles (Coleoptera: 
Carabidae) of Three Horticultural Farms in Lombardy, Northern Italy. <span class="html-italic">Boll. Zool. Agr. Bachic.</span> <b>2006</b>, <span class="html-italic">39</span>, 193â€“209. [<a href="https://scholar.google.com/scholar_lookup?title=The+Ground+Beetles+(Coleoptera:+Carabidae)+of+Three+Horticultural+Farms+in+Lombardy,+Northern+Italy&amp;author=Lupi,+D.&amp;author=Colombo,+M.&amp;author=Facchini,+S.&amp;publication_year=2006&amp;journal=Boll.+Zool.+Agr.+Bachic.&amp;volume=39&amp;pages=193%E2%80%93209" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li><li id="B119-remotesensing-12-00080" class="html-xxx" data-content="119.">Dauber,
 J.; Purtauf, T.; Allspach, A.; Frisch, J.; VoigtlÃ¤nder, K.; Wolters, V.
 Local vs. Landscape Controls on Diversity: A Test Using 
Surface-Dwelling Soil Macroinvertebrates of Differing Mobility. <span class="html-italic">Glob. Ecol. Biogeogr.</span> <b>2005</b>, <span class="html-italic">14</span>, 213â€“221. [<a href="https://scholar.google.com/scholar_lookup?title=Local+vs.+Landscape+Controls+on+Diversity:+A+Test+Using+Surface-Dwelling+Soil+Macroinvertebrates+of+Differing+Mobility&amp;author=Dauber,+J.&amp;author=Purtauf,+T.&amp;author=Allspach,+A.&amp;author=Frisch,+J.&amp;author=Voigtl%C3%A4nder,+K.&amp;author=Wolters,+V.&amp;publication_year=2005&amp;journal=Glob.+Ecol.+Biogeogr.&amp;volume=14&amp;pages=213%E2%80%93221&amp;doi=10.1111/j.1466-822X.2005.00150.x" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1111/j.1466-822X.2005.00150.x" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B120-remotesensing-12-00080" class="html-xxx" data-content="120.">Magura, T.; Nagy, D.; TÃ³thmÃ©rÃ©sz, B. Rove Beetles Respond Heterogeneously to Urbanization. <span class="html-italic">J. Insect Conserv.</span> <b>2013</b>, <span class="html-italic">17</span>, 715â€“724. [<a href="https://scholar.google.com/scholar_lookup?title=Rove+Beetles+Respond+Heterogeneously+to+Urbanization&amp;author=Magura,+T.&amp;author=Nagy,+D.&amp;author=T%C3%B3thm%C3%A9r%C3%A9sz,+B.&amp;publication_year=2013&amp;journal=J.+Insect+Conserv.&amp;volume=17&amp;pages=715%E2%80%93724&amp;doi=10.1007/s10841-013-9555-y" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1007/s10841-013-9555-y" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B121-remotesensing-12-00080" class="html-xxx" data-content="121.">Raxworthy,
 C.J.; Martinez-Meyer, E.; Horning, N.; Nussbaum, R.A.; Schneider, G.E.;
 Ortega-Huerta, M.A.; Peterson, A.T. Predicting Distributions of Known 
and Unknown Reptile Species in Madagascar. <span class="html-italic">Nature</span> <b>2003</b>, <span class="html-italic">426</span>, 837â€“841. [<a href="https://scholar.google.com/scholar_lookup?title=Predicting+Distributions+of+Known+and+Unknown+Reptile+Species+in+Madagascar&amp;author=Raxworthy,+C.J.&amp;author=Martinez-Meyer,+E.&amp;author=Horning,+N.&amp;author=Nussbaum,+R.A.&amp;author=Schneider,+G.E.&amp;author=Ortega-Huerta,+M.A.&amp;author=Peterson,+A.T.&amp;publication_year=2003&amp;journal=Nature&amp;volume=426&amp;pages=837%E2%80%93841&amp;doi=10.1038/nature02205&amp;pmid=14685238" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1038/nature02205" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>] [<a href="https://www.ncbi.nlm.nih.gov/pubmed/14685238" class="cross-ref" data-typ="pmid" target="_blank" rel="noopener noreferrer">PubMed</a>]</li><li id="B122-remotesensing-12-00080" class="html-xxx" data-content="122.">Remelgado,
 R.; Leutner, B.; Safi, K.; Sonnenschein, R.; Kuebert, C.; Wegmann, M. 
Linking animal movement and remote sensingâ€”Mapping resource suitability 
from a remote sensing perspective. <span class="html-italic">Remote Sens. Ecol. Conserv.</span> <b>2017</b>, <span class="html-italic">4</span>, 211â€“224. [<a href="https://scholar.google.com/scholar_lookup?title=Linking+animal+movement+and+remote+sensing%E2%80%94Mapping+resource+suitability+from+a+remote+sensing+perspective&amp;author=Remelgado,+R.&amp;author=Leutner,+B.&amp;author=Safi,+K.&amp;author=Sonnenschein,+R.&amp;author=Kuebert,+C.&amp;author=Wegmann,+M.&amp;publication_year=2017&amp;journal=Remote+Sens.+Ecol.+Conserv.&amp;volume=4&amp;pages=211%E2%80%93224&amp;doi=10.1002/rse2.70" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1002/rse2.70" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B123-remotesensing-12-00080" class="html-xxx" data-content="123.">Alexakis,
 D.D.; Mexis, F.D.K.; Vozinaki, A.E.K.; Daliakopoulos, I.N.; Tsanis, 
I.K. Soil Moisture Content Estimation Based on Sentinel-1 and Auxiliary 
Earth Observation Products. A Hydrological Approach. <span class="html-italic">Sensors</span> <b>2017</b>, <span class="html-italic">17</span>, 1455. [<a href="https://scholar.google.com/scholar_lookup?title=Soil+Moisture+Content+Estimation+Based+on+Sentinel-1+and+Auxiliary+Earth+Observation+Products.+A+Hydrological+Approach&amp;author=Alexakis,+D.D.&amp;author=Mexis,+F.D.K.&amp;author=Vozinaki,+A.E.K.&amp;author=Daliakopoulos,+I.N.&amp;author=Tsanis,+I.K.&amp;publication_year=2017&amp;journal=Sensors&amp;volume=17&amp;pages=1455&amp;doi=10.3390/s17061455" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.3390/s17061455" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B124-remotesensing-12-00080" class="html-xxx" data-content="124.">Frampton,
 W.J.; Dash, J.; Watmough, G.; Milton, E.J. Evaluating the Capabilities 
of Sentinel-2 for Quantitative Estimation of Biophysical Variables in 
Vegetation. <span class="html-italic">ISPRS J. Photogramm. Remote Sens.</span> <b>2013</b>, <span class="html-italic">82</span>, 83â€“92. [<a href="https://scholar.google.com/scholar_lookup?title=Evaluating+the+Capabilities+of+Sentinel-2+for+Quantitative+Estimation+of+Biophysical+Variables+in+Vegetation&amp;author=Frampton,+W.J.&amp;author=Dash,+J.&amp;author=Watmough,+G.&amp;author=Milton,+E.J.&amp;publication_year=2013&amp;journal=ISPRS+J.+Photogramm.+Remote+Sens.&amp;volume=82&amp;pages=83%E2%80%9392&amp;doi=10.1016/j.isprsjprs.2013.04.007" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1016/j.isprsjprs.2013.04.007" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B125-remotesensing-12-00080" class="html-xxx" data-content="125.">Belgiu,
 M.; Csillik, O. Sentinel-2 Cropland Mapping Using Pixel-Based and 
Object-Based Time-Weighted Dynamic Time Warping Analysis. <span class="html-italic">Remote Sens. Environ.</span> <b>2018</b>, <span class="html-italic">204</span>, 509â€“523. [<a href="https://scholar.google.com/scholar_lookup?title=Sentinel-2+Cropland+Mapping+Using+Pixel-Based+and+Object-Based+Time-Weighted+Dynamic+Time+Warping+Analysis&amp;author=Belgiu,+M.&amp;author=Csillik,+O.&amp;publication_year=2018&amp;journal=Remote+Sens.+Environ.&amp;volume=204&amp;pages=509%E2%80%93523&amp;doi=10.1016/j.rse.2017.10.005" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.1016/j.rse.2017.10.005" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B126-remotesensing-12-00080" class="html-xxx" data-content="126.">Manfreda,
 S.; McCabe, M.F.; Miller, P.E.; Lucas, R.; Madrigal, V.P.; Mallinis, 
G.; Dor, E.B.; Helman, D.; Estes, L.; Ciraolo, G.; et al. On the Use of 
Unmanned Aerial Systems for Environmental Monitoring. <span class="html-italic">Remote Sens.</span> <b>2018</b>, <span class="html-italic">10</span>, 641. [<a href="https://scholar.google.com/scholar_lookup?title=On+the+Use+of+Unmanned+Aerial+Systems+for+Environmental+Monitoring&amp;author=Manfreda,+S.&amp;author=McCabe,+M.F.&amp;author=Miller,+P.E.&amp;author=Lucas,+R.&amp;author=Madrigal,+V.P.&amp;author=Mallinis,+G.&amp;author=Dor,+E.B.&amp;author=Helman,+D.&amp;author=Estes,+L.&amp;author=Ciraolo,+G.&amp;publication_year=2018&amp;journal=Remote+Sens.&amp;volume=10&amp;pages=641&amp;doi=10.3390/rs10040641" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.3390/rs10040641" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li><li id="B127-remotesensing-12-00080" class="html-xxx" data-content="127.">Cruzan,
 M.B.; Weinstein, B.G.; Grasty, M.R.; Kohrn, B.F.; Hendrickson, E.C.; 
Arredondo, T.M.; Thompson, P.G. Small Unmanned Aerial Vehicles 
(Micro-Uavs, Drones) in Plant Ecology. <span class="html-italic">Appl. Plant Sci.</span> <b>2016</b>, <span class="html-italic">4</span>, 160041. [<a href="https://scholar.google.com/scholar_lookup?title=Small+Unmanned+Aerial+Vehicles+(Micro-Uavs,+Drones)+in+Plant+Ecology&amp;author=Cruzan,+M.B.&amp;author=Weinstein,+B.G.&amp;author=Grasty,+M.R.&amp;author=Kohrn,+B.F.&amp;author=Hendrickson,+E.C.&amp;author=Arredondo,+T.M.&amp;author=Thompson,+P.G.&amp;publication_year=2016&amp;journal=Appl.+Plant+Sci.&amp;volume=4&amp;pages=160041&amp;doi=10.3732/apps.1600041" class="google-scholar" target="_blank" rel="noopener noreferrer">Google Scholar</a>] [<a href="https://dx.doi.org/10.3732/apps.1600041" class="cross-ref" target="_blank" rel="noopener noreferrer">CrossRef</a>]</li></ol></section><section id="FiguresandTables" type="display-objects"><div class="html-fig-wrap" id="remotesensing-12-00080-f001">
<div class="html-fig_img">
<div class="html-figpopup html-figpopup-link" href="#fig_body_display_remotesensing-12-00080-f001">
<img alt="Remotesensing 12 00080 g001 550" data-large="/remotesensing/remotesensing-12-00080/article_deploy/html/images/remotesensing-12-00080-g001.png" data-original="/remotesensing/remotesensing-12-00080/article_deploy/html/images/remotesensing-12-00080-g001.png" onerror="/remotesensing/remotesensing-12-00080/article_deploy/html/images/remotesensing-12-00080-g001.png" src="remotesensing-12-00080-g001-550.jpg">
<a class="html-expand html-figpopup" href="#fig_body_display_remotesensing-12-00080-f001"></a>
</div>
</div>
<div class="html-fig_description">
<b>Figure 1.</b>
Study area and sampling design for long-term biodiversity monitoring efforts. (<b>a</b>)
 Location of Gran Paradiso National Park. A buffer of 1 km was added to 
the official boundaries of the park to avoid edge effects when modelling
 species distributions. (<b>b</b>) Locations of transects within the park boundaries. (<b>c</b>) Schematic representation of the distribution of plots along a transect (following contour lines). (<b>d</b>) Locations of pitfall traps within each plot.

</div>
</div>
<div class="html-fig_show  mfp-hide" id="fig_body_display_remotesensing-12-00080-f001">
<div class="html-caption"> <b>Figure 1.</b>
Study area and sampling design for long-term biodiversity monitoring efforts. (<b>a</b>)
 Location of Gran Paradiso National Park. A buffer of 1 km was added to 
the official boundaries of the park to avoid edge effects when modelling
 species distributions. (<b>b</b>) Locations of transects within the park boundaries. (<b>c</b>) Schematic representation of the distribution of plots along a transect (following contour lines). (<b>d</b>) Locations of pitfall traps within each plot.</div>
<div class="html-img"><img alt="Remotesensing 12 00080 g001" data-large="/remotesensing/remotesensing-12-00080/article_deploy/html/images/remotesensing-12-00080-g001.png" data-original="/remotesensing/remotesensing-12-00080/article_deploy/html/images/remotesensing-12-00080-g001.png" onerror="/remotesensing/remotesensing-12-00080/article_deploy/html/images/remotesensing-12-00080-g001.png" src="remotesensing-12-00080-g001.png"></div>
</div><div class="html-fig-wrap" id="remotesensing-12-00080-f002">
<div class="html-fig_img">
<div class="html-figpopup html-figpopup-link" href="#fig_body_display_remotesensing-12-00080-f002">
<img alt="Remotesensing 12 00080 g002 550" data-large="/remotesensing/remotesensing-12-00080/article_deploy/html/images/remotesensing-12-00080-g002.png" data-original="/remotesensing/remotesensing-12-00080/article_deploy/html/images/remotesensing-12-00080-g002.png" onerror="/remotesensing/remotesensing-12-00080/article_deploy/html/images/remotesensing-12-00080-g002.png" src="remotesensing-12-00080-g002-550.jpg">
<a class="html-expand html-figpopup" href="#fig_body_display_remotesensing-12-00080-f002"></a>
</div>
</div>
<div class="html-fig_description">
<b>Figure 2.</b>
Comparison of modelled presence probabilities (ensemble models) at 
presence (traps in which individuals of each target species were 
captured) and absence locations (traps without captured individuals, not
 used for model training) for the time periods 2006â€“2007 and 2012â€“2013.

</div>
</div>
<div class="html-fig_show  mfp-hide" id="fig_body_display_remotesensing-12-00080-f002">
<div class="html-caption"> <b>Figure 2.</b>
Comparison of modelled presence probabilities (ensemble models) at 
presence (traps in which individuals of each target species were 
captured) and absence locations (traps without captured individuals, not
 used for model training) for the time periods 2006â€“2007 and 2012â€“2013.</div>
<div class="html-img"><img alt="Remotesensing 12 00080 g002" data-large="/remotesensing/remotesensing-12-00080/article_deploy/html/images/remotesensing-12-00080-g002.png" data-original="/remotesensing/remotesensing-12-00080/article_deploy/html/images/remotesensing-12-00080-g002.png" onerror="/remotesensing/remotesensing-12-00080/article_deploy/html/images/remotesensing-12-00080-g002.png" src="remotesensing-12-00080-g002.png"></div>
</div><div class="html-fig-wrap" id="remotesensing-12-00080-f003">
<div class="html-fig_img">
<div class="html-figpopup html-figpopup-link" href="#fig_body_display_remotesensing-12-00080-f003">
<img alt="Remotesensing 12 00080 g003 550" data-large="/remotesensing/remotesensing-12-00080/article_deploy/html/images/remotesensing-12-00080-g003.png" data-original="/remotesensing/remotesensing-12-00080/article_deploy/html/images/remotesensing-12-00080-g003.png" onerror="/remotesensing/remotesensing-12-00080/article_deploy/html/images/remotesensing-12-00080-g003.png" src="remotesensing-12-00080-g003-550.jpg">
<a class="html-expand html-figpopup" href="#fig_body_display_remotesensing-12-00080-f003"></a>
</div>
</div>
<div class="html-fig_description">
<b>Figure 3.</b>
Species distribution likelihood for each species in 2006â€“2007 and 
2012â€“2013 based on the ensemble models. Locations of the plots used as 
presence points for model training are shown to emphasize the 
variability of input data. Coordinates shown are based ETRS (European 
Terrestrial Reference System) 1989 LAEA (Lambert Azimuthal Equal-Area) 
projection.

</div>
</div>
<div class="html-fig_show  mfp-hide" id="fig_body_display_remotesensing-12-00080-f003">
<div class="html-caption"> <b>Figure 3.</b>
Species distribution likelihood for each species in 2006â€“2007 and 
2012â€“2013 based on the ensemble models. Locations of the plots used as 
presence points for model training are shown to emphasize the 
variability of input data. Coordinates shown are based ETRS (European 
Terrestrial Reference System) 1989 LAEA (Lambert Azimuthal Equal-Area) 
projection.</div>
<div class="html-img"><img alt="Remotesensing 12 00080 g003" data-large="/remotesensing/remotesensing-12-00080/article_deploy/html/images/remotesensing-12-00080-g003.png" data-original="/remotesensing/remotesensing-12-00080/article_deploy/html/images/remotesensing-12-00080-g003.png" onerror="/remotesensing/remotesensing-12-00080/article_deploy/html/images/remotesensing-12-00080-g003.png" src="remotesensing-12-00080-g003.png"></div>
</div><div class="html-table-wrap" id="remotesensing-12-00080-t001">
<div class="html-table_wrap_td">
<div class="html-tablepopup html-tablepopup-link" href="#table_body_display_remotesensing-12-00080-t001">
<img alt="Table" src="table.png">
<a class="html-expand html-tablepopup" href="#table_body_display_remotesensing-12-00080-t001"></a>
</div>
</div>
<div class="html-table_wrap_discription">
<b>Table 1.</b>
Number of traps, plots, and transects in which individuals of each 
species were captured, located in areas free of cloud and snow cover in 
the analyzed SRS data. Number of plots (used as presence data) used for 
modelling and the correspondent time periods are highlighted in bold.
</div>
</div>
<div class="html-table_show  mfp-hide " id="table_body_display_remotesensing-12-00080-t001">
<div class="html-caption"><b>Table 1.</b>
Number of traps, plots, and transects in which individuals of each 
species were captured, located in areas free of cloud and snow cover in 
the analyzed SRS data. Number of plots (used as presence data) used for 
modelling and the correspondent time periods are highlighted in bold.</div>
<table>
<tbody><tr><td style="border-top:solid thin;border-bottom:solid thin" class="html-align-center" valign="middle" align="center"> </td><td colspan="6" style="border-top:solid thin;border-bottom:solid thin" class="html-align-center" valign="middle" align="center"><span class="html-italic">Dinothenarus fossor</span></td><td colspan="6" style="border-top:solid thin;border-bottom:solid thin" class="html-align-center" valign="middle" align="center"><span class="html-italic">Platydracus stercorarius</span></td><td colspan="6" style="border-top:solid thin;border-bottom:solid thin" class="html-align-center" valign="middle" align="center"><span class="html-italic">Ocypus ophthalmicus</span></td></tr><tr><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">Year(s)</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">2006</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">2007</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">2012</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">2013</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center"><b>2006/2007</b></td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center"><b>2012/2013</b></td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">2006</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">2007</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">2012</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">2013</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center"><b>2006/2007</b></td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center"><b>2012/2013</b></td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">2006</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">2007</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">2012</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">2013</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center"><b>2006/2007</b></td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center"><b>2012/2013</b></td></tr><tr><td class="html-align-center" valign="middle" align="center">Traps</td><td class="html-align-center" valign="middle" align="center">42</td><td class="html-align-center" valign="middle" align="center">36</td><td class="html-align-center" valign="middle" align="center">55</td><td class="html-align-center" valign="middle" align="center">49</td><td class="html-align-center" valign="middle" align="center">52</td><td class="html-align-center" valign="middle" align="center">66</td><td class="html-align-center" valign="middle" align="center">14</td><td class="html-align-center" valign="middle" align="center">12</td><td class="html-align-center" valign="middle" align="center">25</td><td class="html-align-center" valign="middle" align="center">35</td><td class="html-align-center" valign="middle" align="center">22</td><td class="html-align-center" valign="middle" align="center">45</td><td class="html-align-center" valign="middle" align="center">19</td><td class="html-align-center" valign="middle" align="center">15</td><td class="html-align-center" valign="middle" align="center">16</td><td class="html-align-center" valign="middle" align="center">16</td><td class="html-align-center" valign="middle" align="center">27</td><td class="html-align-center" valign="middle" align="center">26</td></tr><tr><td class="html-align-center" valign="middle" align="center">Plots</td><td class="html-align-center" valign="middle" align="center">15</td><td class="html-align-center" valign="middle" align="center">15</td><td class="html-align-center" valign="middle" align="center">18</td><td class="html-align-center" valign="middle" align="center">17</td><td class="html-align-center" valign="middle" align="center"><b>17</b></td><td class="html-align-center" valign="middle" align="center"><b>19</b></td><td class="html-align-center" valign="middle" align="center">9</td><td class="html-align-center" valign="middle" align="center">10</td><td class="html-align-center" valign="middle" align="center">14</td><td class="html-align-center" valign="middle" align="center">16</td><td class="html-align-center" valign="middle" align="center"><b>12</b></td><td class="html-align-center" valign="middle" align="center"><b>18</b></td><td class="html-align-center" valign="middle" align="center">11</td><td class="html-align-center" valign="middle" align="center">10</td><td class="html-align-center" valign="middle" align="center">9</td><td class="html-align-center" valign="middle" align="center">10</td><td class="html-align-center" valign="middle" align="center"><b>14</b></td><td class="html-align-center" valign="middle" align="center"><b>12</b></td></tr><tr><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">Tran-sects</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">4</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">5</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">4</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">4</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">5</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">4</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">4</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">5</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">5</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">5</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">5</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">5</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">5</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">5</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">4</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">5</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">5</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">5</td></tr></tbody>
</table>
</div><div class="html-table-wrap" id="remotesensing-12-00080-t002">
<div class="html-table_wrap_td">
<div class="html-tablepopup html-tablepopup-link" href="#table_body_display_remotesensing-12-00080-t002">
<img alt="Table" src="table.png">
<a class="html-expand html-tablepopup" href="#table_body_display_remotesensing-12-00080-t002"></a>
</div>
</div>
<div class="html-table_wrap_discription">
<b>Table 2.</b>
Characteristics of noncorrelated SRS-based variables used in species 
distribution modelling. Abbreviations: TCT, tasseled cap transformation.
 References indicate studies highlighting the ecological relevance of 
the considered variables.
</div>
</div>
<div class="html-table_show  mfp-hide " id="table_body_display_remotesensing-12-00080-t002">
<div class="html-caption"><b>Table 2.</b>
Characteristics of noncorrelated SRS-based variables used in species 
distribution modelling. Abbreviations: TCT, tasseled cap transformation.
 References indicate studies highlighting the ecological relevance of 
the considered variables.</div>
<table>
<thead><tr><th style="border-top:solid thin;border-bottom:solid thin" class="html-align-center" valign="middle" align="center">Variable</th><th style="border-top:solid thin;border-bottom:solid thin" class="html-align-center" valign="middle" align="center">Ecological Relevance</th><th style="border-top:solid thin;border-bottom:solid thin" class="html-align-center" valign="middle" align="center">References</th><th style="border-top:solid thin;border-bottom:solid thin" class="html-align-center" valign="middle" align="center">Source</th></tr></thead><tbody><tr><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">Altitude</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">Directly correlated with temperature, of particular importance in mountain ecosystems</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">[<a href="#B9-remotesensing-12-00080" class="html-bibr">9</a>,<a href="#B18-remotesensing-12-00080" class="html-bibr">18</a>,<a href="#B67-remotesensing-12-00080" class="html-bibr">67</a>]</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">ASTER GDEM</td></tr><tr><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">Eastness/Northness</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">Gradients
 represent differences in the aspect affecting irradiation and 
precipitation (e.g. an orientation towards east favours heating up in 
the morning while west-facing ensure higher temperatures in the 
afternoon)</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">[<a href="#B68-remotesensing-12-00080" class="html-bibr">68</a>,<a href="#B69-remotesensing-12-00080" class="html-bibr">69</a>]</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">ASTER GDEM</td></tr><tr><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">Slope</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">Terrain steepness reflects gradients in humidity, thickness of soil layer and transport dynamics of matter</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">[<a href="#B18-remotesensing-12-00080" class="html-bibr">18</a>]</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">ASTER GDEM</td></tr><tr><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">Terrain Ruggedness Index</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">Represents
 elevational differences among neighbouring grid cells (high values 
relate to large topographic heterogeneity and impenetrable terrain)</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">[<a href="#B65-remotesensing-12-00080" class="html-bibr">65</a>,<a href="#B70-remotesensing-12-00080" class="html-bibr">70</a>]</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">ASTER GDEM</td></tr><tr><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">Greenness (TCT)</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">Responds to photosynthetic capacity, vegetation cover and primary productivity (sensitive to topography)</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">[<a href="#B71-remotesensing-12-00080" class="html-bibr">71</a>,<a href="#B72-remotesensing-12-00080" class="html-bibr">72</a>]</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">Landsat imagery</td></tr><tr><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">Wetness (TCT)</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">Responds to soil/canopy moisture and standing water, being often highest in young forest stands (not sensitive to topography)</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">[<a href="#B73-remotesensing-12-00080" class="html-bibr">73</a>,<a href="#B74-remotesensing-12-00080" class="html-bibr">74</a>]</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">Landsat imagery</td></tr></tbody>
</table>
</div><div class="html-table-wrap" id="remotesensing-12-00080-t003">
<div class="html-table_wrap_td">
<div class="html-tablepopup html-tablepopup-link" href="#table_body_display_remotesensing-12-00080-t003">
<img alt="Table" src="table.png">
<a class="html-expand html-tablepopup" href="#table_body_display_remotesensing-12-00080-t003"></a>
</div>
</div>
<div class="html-table_wrap_discription">
<b>Table 3.</b>
Mean area under curve (AUC) and true skills statistics (TSS) values, 
with standard deviation (SD), calculated on the 50 model repetitions for
 the different time periods, species, and model algorithms. AUC values 
&gt;0.7 and TSS values &gt;0.4 are highlighted in green. Abbreviations: 
GLM, generalized linear model; GAM, generalized additive model; RF, 
random forest; GBM, generalized boosting model.
</div>
</div>
<div class="html-table_show  mfp-hide " id="table_body_display_remotesensing-12-00080-t003">
<div class="html-caption"><b>Table 3.</b>
Mean area under curve (AUC) and true skills statistics (TSS) values, 
with standard deviation (SD), calculated on the 50 model repetitions for
 the different time periods, species, and model algorithms. AUC values 
&gt;0.7 and TSS values &gt;0.4 are highlighted in green. Abbreviations: 
GLM, generalized linear model; GAM, generalized additive model; RF, 
random forest; GBM, generalized boosting model.</div>
<table>
<tbody><tr><td style="border-top:solid thin" class="html-align-center" valign="middle" align="center"> </td><td colspan="8" style="border-top:solid thin" class="html-align-center" valign="middle" align="center"><b><span class="html-italic">Dinothenarus fossor</span></b></td></tr><tr><td style="border-top:solid thin;border-bottom:solid thin" class="html-align-center" valign="middle" align="center"><b>Years</b></td><td colspan="4" style="border-top:solid thin;border-bottom:solid thin" class="html-align-center" valign="middle" align="center"><b>2006/2007</b></td><td colspan="4" style="border-top:solid thin;border-bottom:solid thin" class="html-align-center" valign="middle" align="center"><b>2012/2013</b></td></tr><tr><td class="html-align-center" valign="middle" align="center"> </td><td class="html-align-center" valign="middle" align="center">mean AUC</td><td class="html-align-center" valign="middle" align="center">Â±SD</td><td class="html-align-center" valign="middle" align="center">mean TSS</td><td class="html-align-center" valign="middle" align="center">Â±SD</td><td class="html-align-center" valign="middle" align="center">mean AUC</td><td class="html-align-center" valign="middle" align="center">Â±SD</td><td class="html-align-center" valign="middle" align="center">mean TSS</td><td class="html-align-center" valign="middle" align="center">Â±SD</td></tr><tr><td class="html-align-center" valign="middle" align="center">GLM</td><td class="html-align-center" valign="middle" align="center">0.549</td><td class="html-align-center" valign="middle" align="center">0.11</td><td class="html-align-center" valign="middle" align="center">0.164</td><td class="html-align-center" valign="middle" align="center">0.17</td><td class="html-align-center" valign="middle" align="center">0.67</td><td class="html-align-center" valign="middle" align="center">0.104</td><td class="html-align-center" valign="middle" align="center">0.354</td><td class="html-align-center" valign="middle" align="center">0.19</td></tr><tr><td class="html-align-center" valign="middle" align="center">GAM</td><td style="background:#C6EFCE" class="html-align-center" valign="middle" align="center"><span style="color:#006100">0.717</span></td><td class="html-align-center" valign="middle" align="center">0.118</td><td style="background:#C6EFCE" class="html-align-center" valign="middle" align="center"><span style="color:#006100">0.455</span></td><td class="html-align-center" valign="middle" align="center">0.185</td><td style="background:#C6EFCE" class="html-align-center" valign="middle" align="center"><span style="color:#006100">0.782</span></td><td class="html-align-center" valign="middle" align="center">0.064</td><td style="background:#C6EFCE" class="html-align-center" valign="middle" align="center"><span style="color:#006100">0.557</span></td><td class="html-align-center" valign="middle" align="center">0.128</td></tr><tr><td class="html-align-center" valign="middle" align="center">MAXENT</td><td style="background:#C6EFCE" class="html-align-center" valign="middle" align="center"><span style="color:#006100">0.725</span></td><td class="html-align-center" valign="middle" align="center">0.083</td><td style="background:#C6EFCE" class="html-align-center" valign="middle" align="center"><span style="color:#006100">0.454</span></td><td class="html-align-center" valign="middle" align="center">0.126</td><td style="background:#C6EFCE" class="html-align-center" valign="middle" align="center"><span style="color:#006100">0.801</span></td><td class="html-align-center" valign="middle" align="center">0.102</td><td style="background:#C6EFCE" class="html-align-center" valign="middle" align="center"><span style="color:#006100">0.574</span></td><td class="html-align-center" valign="middle" align="center">0.191</td></tr><tr><td class="html-align-center" valign="middle" align="center">RF</td><td style="background:#C6EFCE" class="html-align-center" valign="middle" align="center"><span style="color:#006100">0.74</span></td><td class="html-align-center" valign="middle" align="center">0.094</td><td style="background:#C6EFCE" class="html-align-center" valign="middle" align="center"><span style="color:#006100">0.5</span></td><td class="html-align-center" valign="middle" align="center">0.149</td><td style="background:#C6EFCE" class="html-align-center" valign="middle" align="center"><span style="color:#006100">0.813</span></td><td class="html-align-center" valign="middle" align="center">0.085</td><td style="background:#C6EFCE" class="html-align-center" valign="middle" align="center"><span style="color:#006100">0.618</span></td><td class="html-align-center" valign="middle" align="center">0.12</td></tr><tr><td class="html-align-center" valign="middle" align="center">GBM</td><td style="background:#C6EFCE" class="html-align-center" valign="middle" align="center"><span style="color:#006100">0.83</span></td><td class="html-align-center" valign="middle" align="center">0.047</td><td style="background:#C6EFCE" class="html-align-center" valign="middle" align="center"><span style="color:#006100">0.588</span></td><td class="html-align-center" valign="middle" align="center">0.088</td><td style="background:#C6EFCE" class="html-align-center" valign="middle" align="center"><span style="color:#006100">0.835</span></td><td class="html-align-center" valign="middle" align="center">0.064</td><td style="background:#C6EFCE" class="html-align-center" valign="middle" align="center"><span style="color:#006100">0.609</span></td><td class="html-align-center" valign="middle" align="center">0.137</td></tr><tr><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">Ensemble</td><td style="border-bottom:solid thin;background:#C6EFCE" class="html-align-center" valign="middle" align="center"><span style="color:#006100">0.845</span></td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">0.044</td><td style="border-bottom:solid thin;background:#C6EFCE" class="html-align-center" valign="middle" align="center"><span style="color:#006100">0.663</span></td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">0.076</td><td style="border-bottom:solid thin;background:#C6EFCE" class="html-align-center" valign="middle" align="center"><span style="color:#006100">0.869</span></td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">0.053</td><td style="border-bottom:solid thin;background:#C6EFCE" class="html-align-center" valign="middle" align="center"><span style="color:#006100">0.666</span></td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">0.111</td></tr><tr><td class="html-align-center" valign="middle" align="center"> </td><td colspan="8" class="html-align-center" valign="middle" align="center"><b><span class="html-italic">Platydracus stercorarius</span></b></td></tr><tr><td style="border-top:solid thin;border-bottom:solid thin" class="html-align-center" valign="middle" align="center"><b>Years</b></td><td colspan="4" style="border-top:solid thin;border-bottom:solid thin" class="html-align-center" valign="middle" align="center"><b>2006/2007</b></td><td colspan="4" style="border-top:solid thin;border-bottom:solid thin" class="html-align-center" valign="middle" align="center"><b>2012/2013</b></td></tr><tr><td class="html-align-center" valign="middle" align="center"> </td><td class="html-align-center" valign="middle" align="center">mean AUC</td><td class="html-align-center" valign="middle" align="center">Â±SD</td><td class="html-align-center" valign="middle" align="center">mean TSS</td><td class="html-align-center" valign="middle" align="center">Â±SD</td><td class="html-align-center" valign="middle" align="center">mean AUC</td><td class="html-align-center" valign="middle" align="center">Â±SD</td><td class="html-align-center" valign="middle" align="center">mean TSS</td><td class="html-align-center" valign="middle" align="center">Â±SD</td></tr><tr><td class="html-align-center" valign="middle" align="center">GLM</td><td class="html-align-center" valign="middle" align="center">0.532</td><td class="html-align-center" valign="middle" align="center">0.073</td><td class="html-align-center" valign="middle" align="center">0.098</td><td class="html-align-center" valign="middle" align="center">0.117</td><td class="html-align-center" valign="middle" align="center">0.529</td><td class="html-align-center" valign="middle" align="center">0.102</td><td class="html-align-center" valign="middle" align="center">0.168</td><td class="html-align-center" valign="middle" align="center">0.169</td></tr><tr><td class="html-align-center" valign="middle" align="center">GAM</td><td class="html-align-center" valign="middle" align="center">0.626</td><td class="html-align-center" valign="middle" align="center">0.124</td><td class="html-align-center" valign="middle" align="center">0.312</td><td class="html-align-center" valign="middle" align="center">0.226</td><td class="html-align-center" valign="middle" align="center">0.625</td><td class="html-align-center" valign="middle" align="center">0.098</td><td class="html-align-center" valign="middle" align="center">0.337</td><td class="html-align-center" valign="middle" align="center">0.117</td></tr><tr><td class="html-align-center" valign="middle" align="center">MAXENT</td><td style="background:#C6EFCE" class="html-align-center" valign="middle" align="center"><span style="color:#006100">0.78</span></td><td class="html-align-center" valign="middle" align="center">0.068</td><td style="background:#C6EFCE" class="html-align-center" valign="middle" align="center"><span style="color:#006100">0.605</span></td><td class="html-align-center" valign="middle" align="center">0.126</td><td class="html-align-center" valign="middle" align="center">0.655</td><td class="html-align-center" valign="middle" align="center">0.055</td><td class="html-align-center" valign="middle" align="center">0.375</td><td class="html-align-center" valign="middle" align="center">0.086</td></tr><tr><td class="html-align-center" valign="middle" align="center">RF</td><td class="html-align-center" valign="middle" align="center">0.638</td><td class="html-align-center" valign="middle" align="center">0.115</td><td class="html-align-center" valign="middle" align="center">0.368</td><td class="html-align-center" valign="middle" align="center">0.161</td><td style="background:#C6EFCE" class="html-align-center" valign="middle" align="center"><span style="color:#006100">0.754</span></td><td class="html-align-center" valign="middle" align="center">0.077</td><td style="background:#C6EFCE" class="html-align-center" valign="middle" align="center"><span style="color:#006100">0.505</span></td><td class="html-align-center" valign="middle" align="center">0.121</td></tr><tr><td class="html-align-center" valign="middle" align="center">GBM</td><td class="html-align-center" valign="middle" align="center">0.664</td><td class="html-align-center" valign="middle" align="center">0.133</td><td class="html-align-center" valign="middle" align="center">0.391</td><td class="html-align-center" valign="middle" align="center">0.221</td><td style="background:#C6EFCE" class="html-align-center" valign="middle" align="center"><span style="color:#006100">0.77</span></td><td class="html-align-center" valign="middle" align="center">0.059</td><td style="background:#C6EFCE" class="html-align-center" valign="middle" align="center"><span style="color:#006100">0.542</span></td><td class="html-align-center" valign="middle" align="center">0.085</td></tr><tr><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">Ensemble</td><td style="border-bottom:solid thin;background:#C6EFCE" class="html-align-center" valign="middle" align="center"><span style="color:#006100">0.798</span></td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">0.058</td><td style="border-bottom:solid thin;background:#C6EFCE" class="html-align-center" valign="middle" align="center"><span style="color:#006100">0.618</span></td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">0.1</td><td style="border-bottom:solid thin;background:#C6EFCE" class="html-align-center" valign="middle" align="center"><span style="color:#006100">0.788</span></td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">0.051</td><td style="border-bottom:solid thin;background:#C6EFCE" class="html-align-center" valign="middle" align="center"><span style="color:#006100">0.551</span></td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">0.095</td></tr><tr><td class="html-align-center" valign="middle" align="center"> </td><td colspan="8" class="html-align-center" valign="middle" align="center"><b><span class="html-italic">Ocypus ophthalmicus</span></b></td></tr><tr><td style="border-top:solid thin;border-bottom:solid thin" class="html-align-center" valign="middle" align="center"><b>Years</b></td><td colspan="4" style="border-top:solid thin;border-bottom:solid thin" class="html-align-center" valign="middle" align="center"><b>2006/2007</b></td><td colspan="4" style="border-top:solid thin;border-bottom:solid thin" class="html-align-center" valign="middle" align="center"><b>2012/2013</b></td></tr><tr><td class="html-align-center" valign="middle" align="center"> </td><td class="html-align-center" valign="middle" align="center">mean AUC</td><td class="html-align-center" valign="middle" align="center">Â±SD</td><td class="html-align-center" valign="middle" align="center">mean TSS</td><td class="html-align-center" valign="middle" align="center">Â±SD</td><td class="html-align-center" valign="middle" align="center">mean AUC</td><td class="html-align-center" valign="middle" align="center">Â±SD</td><td class="html-align-center" valign="middle" align="center">mean TSS</td><td class="html-align-center" valign="middle" align="center">Â±SD</td></tr><tr><td class="html-align-center" valign="middle" align="center">GLM</td><td class="html-align-center" valign="middle" align="center">0.685</td><td class="html-align-center" valign="middle" align="center">0.137</td><td class="html-align-center" valign="middle" align="center">0.391</td><td class="html-align-center" valign="middle" align="center">0.267</td><td class="html-align-center" valign="middle" align="center">0.573</td><td class="html-align-center" valign="middle" align="center">0.149</td><td class="html-align-center" valign="middle" align="center">0.186</td><td class="html-align-center" valign="middle" align="center">0.268</td></tr><tr><td class="html-align-center" valign="middle" align="center">GAM</td><td style="background:#C6EFCE" class="html-align-center" valign="middle" align="center"><span style="color:#006100">0.8</span></td><td class="html-align-center" valign="middle" align="center">0.142</td><td style="background:#C6EFCE" class="html-align-center" valign="middle" align="center"><span style="color:#006100">0.615</span></td><td class="html-align-center" valign="middle" align="center">0.26</td><td class="html-align-center" valign="middle" align="center">0.587</td><td class="html-align-center" valign="middle" align="center">0.125</td><td class="html-align-center" valign="middle" align="center">0.22</td><td class="html-align-center" valign="middle" align="center">0.214</td></tr><tr><td class="html-align-center" valign="middle" align="center">MAXENT</td><td style="background:#C6EFCE" class="html-align-center" valign="middle" align="center"><span style="color:#006100">0.809</span></td><td class="html-align-center" valign="middle" align="center">0.148</td><td style="background:#C6EFCE" class="html-align-center" valign="middle" align="center"><span style="color:#006100">0.707</span></td><td class="html-align-center" valign="middle" align="center">0.217</td><td style="background:#C6EFCE" class="html-align-center" valign="middle" align="center"><span style="color:#006100">0.709</span></td><td class="html-align-center" valign="middle" align="center">0.112</td><td style="background:#C6EFCE" class="html-align-center" valign="middle" align="center"><span style="color:#006100">0.474</span></td><td class="html-align-center" valign="middle" align="center">0.141</td></tr><tr><td class="html-align-center" valign="middle" align="center">RF</td><td style="background:#C6EFCE" class="html-align-center" valign="middle" align="center"><span style="color:#006100">0.881</span></td><td class="html-align-center" valign="middle" align="center">0.079</td><td style="background:#C6EFCE" class="html-align-center" valign="middle" align="center"><span style="color:#006100">0.754</span></td><td class="html-align-center" valign="middle" align="center">0.15</td><td style="background:#C6EFCE" class="html-align-center" valign="middle" align="center"><span style="color:#006100">0.767</span></td><td class="html-align-center" valign="middle" align="center">0.102</td><td style="background:#C6EFCE" class="html-align-center" valign="middle" align="center"><span style="color:#006100">0.572</span></td><td class="html-align-center" valign="middle" align="center">0.142</td></tr><tr><td class="html-align-center" valign="middle" align="center">GBM</td><td style="background:#C6EFCE" class="html-align-center" valign="middle" align="center"><span style="color:#006100">0.843</span></td><td class="html-align-center" valign="middle" align="center">0.087</td><td style="background:#C6EFCE" class="html-align-center" valign="middle" align="center"><span style="color:#006100">0.7</span></td><td class="html-align-center" valign="middle" align="center">0.158</td><td style="background:#C6EFCE" class="html-align-center" valign="middle" align="center"><span style="color:#006100">0.713</span></td><td class="html-align-center" valign="middle" align="center">0.14</td><td style="background:#C6EFCE" class="html-align-center" valign="middle" align="center"><span style="color:#006100">0.525</span></td><td class="html-align-center" valign="middle" align="center">0.191</td></tr><tr><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">Ensemble</td><td style="border-bottom:solid thin;background:#C6EFCE" class="html-align-center" valign="middle" align="center"><span style="color:#006100">0.893</span></td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">0.086</td><td style="border-bottom:solid thin;background:#C6EFCE" class="html-align-center" valign="middle" align="center"><span style="color:#006100">0.806</span></td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">0.145</td><td style="border-bottom:solid thin;background:#C6EFCE" class="html-align-center" valign="middle" align="center"><span style="color:#006100">0.843</span></td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">0.07</td><td style="border-bottom:solid thin;background:#C6EFCE" class="html-align-center" valign="middle" align="center"><span style="color:#006100">0.672</span></td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">0.127</td></tr></tbody>
</table>
</div><div class="html-table-wrap" id="remotesensing-12-00080-t004">
<div class="html-table_wrap_td">
<div class="html-tablepopup html-tablepopup-link" href="#table_body_display_remotesensing-12-00080-t004">
<img alt="Table" src="table.png">
<a class="html-expand html-tablepopup" href="#table_body_display_remotesensing-12-00080-t004"></a>
</div>
</div>
<div class="html-table_wrap_discription">
<b>Table 4.</b>
Variable importance according to correlation analyses between model 
predictions based on permuted and original datasets for all study and 
model algorithms trained by data collected in 2006â€“2007 and 2012â€“2013. 
Values shown are the mean values across the 50 model repetitions. Cell 
coloring in dark grey highlights the most important variable and in 
light grey the second most important one. Abbreviations: GLM, 
generalized linear model; GAM, generalized additive model; RF, random 
forest; GBM, generalized boosting model.
</div>
</div>
<div class="html-table_show  mfp-hide " id="table_body_display_remotesensing-12-00080-t004">
<div class="html-caption"><b>Table 4.</b>
Variable importance according to correlation analyses between model 
predictions based on permuted and original datasets for all study and 
model algorithms trained by data collected in 2006â€“2007 and 2012â€“2013. 
Values shown are the mean values across the 50 model repetitions. Cell 
coloring in dark grey highlights the most important variable and in 
light grey the second most important one. Abbreviations: GLM, 
generalized linear model; GAM, generalized additive model; RF, random 
forest; GBM, generalized boosting model.</div>
<table>
<tbody><tr><td style="border-top:solid thin;border-bottom:solid thin" class="html-align-center" valign="middle" align="center"> </td><td colspan="12" style="border-top:solid thin;border-bottom:solid thin" class="html-align-center" valign="middle" align="center"><b><span class="html-italic">Dinothenarus fossor</span></b></td></tr><tr><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center"><b>Years</b></td><td colspan="6" style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center"><b>2006/2007</b></td><td colspan="6" style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center"><b>2012/2013</b></td></tr><tr><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">Algorithm</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">GLM</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">GAM</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">MAXENT</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">RF</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">GBM</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">Ensemble</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">GLM</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">GAM</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">MAXENT</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">RF</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">GBM</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">Ensemble</td></tr><tr><td class="html-align-center" valign="middle" align="center">Altitude</td><td class="html-align-center" valign="middle" align="center">0.169</td><td class="html-align-center" valign="middle" align="center">0.127</td><td class="html-align-center" valign="middle" align="center">0.175</td><td class="html-align-center" valign="middle" align="center">0.071</td><td class="html-align-center" valign="middle" align="center">0.11</td><td class="html-align-center" valign="middle" align="center">0.117</td><td class="html-align-center" valign="middle" align="center">0.115</td><td class="html-align-center" valign="middle" align="center">0.073</td><td class="html-align-center" valign="middle" align="center">0.105</td><td class="html-align-center" valign="middle" align="center">0.05</td><td class="html-align-center" valign="middle" align="center">0.01</td><td class="html-align-center" valign="middle" align="center">0.048</td></tr><tr><td class="html-align-center" valign="middle" align="center">Northness</td><td style="background:#D9D9D9" class="html-align-center" valign="middle" align="center">0.177</td><td style="background:#D9D9D9" class="html-align-center" valign="middle" align="center">0.182</td><td class="html-align-center" valign="middle" align="center">0.084</td><td class="html-align-center" valign="middle" align="center">0.175</td><td class="html-align-center" valign="middle" align="center">0.002</td><td class="html-align-center" valign="middle" align="center">0.062</td><td style="background:#D9D9D9" class="html-align-center" valign="middle" align="center">0.16</td><td style="background:#D9D9D9" class="html-align-center" valign="middle" align="center">0.234</td><td class="html-align-center" valign="middle" align="center">0.091</td><td class="html-align-center" valign="middle" align="center">0.174</td><td class="html-align-center" valign="middle" align="center">0.022</td><td class="html-align-center" valign="middle" align="center">0.123</td></tr><tr><td class="html-align-center" valign="middle" align="center">Eastness</td><td class="html-align-center" valign="middle" align="center">0.137</td><td style="background:#A6A6A6" class="html-align-center" valign="middle" align="center">0.211</td><td class="html-align-center" valign="middle" align="center">0.02</td><td class="html-align-center" valign="middle" align="center">0.171</td><td class="html-align-center" valign="middle" align="center">0.003</td><td class="html-align-center" valign="middle" align="center">0.079</td><td class="html-align-center" valign="middle" align="center">0.152</td><td class="html-align-center" valign="middle" align="center">0.227</td><td class="html-align-center" valign="middle" align="center">0.035</td><td class="html-align-center" valign="middle" align="center">0.133</td><td class="html-align-center" valign="middle" align="center">0.022</td><td class="html-align-center" valign="middle" align="center">0.11</td></tr><tr><td class="html-align-center" valign="middle" align="center">Slope</td><td class="html-align-center" valign="middle" align="center">0.083</td><td class="html-align-center" valign="middle" align="center">0.05</td><td class="html-align-center" valign="middle" align="center">0.046</td><td class="html-align-center" valign="middle" align="center">0.095</td><td class="html-align-center" valign="middle" align="center">0.001</td><td class="html-align-center" valign="middle" align="center">0.021</td><td class="html-align-center" valign="middle" align="center">0.12</td><td class="html-align-center" valign="middle" align="center">0.053</td><td class="html-align-center" valign="middle" align="center">0.074</td><td class="html-align-center" valign="middle" align="center">0.075</td><td class="html-align-center" valign="middle" align="center">0.022</td><td class="html-align-center" valign="middle" align="center">0.04</td></tr><tr><td class="html-align-center" valign="middle" align="center">Ruggedness</td><td class="html-align-center" valign="middle" align="center">0.122</td><td class="html-align-center" valign="middle" align="center">0.166</td><td style="background:#D9D9D9" class="html-align-center" valign="middle" align="center">0.227</td><td style="background:#D9D9D9" class="html-align-center" valign="middle" align="center">0.179</td><td style="background:#D9D9D9" class="html-align-center" valign="middle" align="center">0.243</td><td style="background:#D9D9D9" class="html-align-center" valign="middle" align="center">0.238</td><td class="html-align-center" valign="middle" align="center">0.138</td><td class="html-align-center" valign="middle" align="center">0.089</td><td style="background:#D9D9D9" class="html-align-center" valign="middle" align="center">0.23</td><td style="background:#D9D9D9" class="html-align-center" valign="middle" align="center">0.182</td><td style="background:#D9D9D9" class="html-align-center" valign="middle" align="center">0.124</td><td style="background:#D9D9D9" class="html-align-center" valign="middle" align="center">0.153</td></tr><tr><td class="html-align-center" valign="middle" align="center">Greenness</td><td style="background:#A6A6A6" class="html-align-center" valign="middle" align="center">0.204</td><td style="background:#D9D9D9" class="html-align-center" valign="middle" align="center">0.182</td><td style="background:#A6A6A6" class="html-align-center" valign="middle" align="center">0.388</td><td style="background:#A6A6A6" class="html-align-center" valign="middle" align="center">0.24</td><td style="background:#A6A6A6" class="html-align-center" valign="middle" align="center">0.642</td><td style="background:#A6A6A6" class="html-align-center" valign="middle" align="center">0.458</td><td style="background:#A6A6A6" class="html-align-center" valign="middle" align="center">0.228</td><td style="background:#A6A6A6" class="html-align-center" valign="middle" align="center">0.237</td><td style="background:#A6A6A6" class="html-align-center" valign="middle" align="center">0.379</td><td style="background:#A6A6A6" class="html-align-center" valign="middle" align="center">0.292</td><td style="background:#A6A6A6" class="html-align-center" valign="middle" align="center">0.792</td><td style="background:#A6A6A6" class="html-align-center" valign="middle" align="center">0.488</td></tr><tr><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">Wetness</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">0.11</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">0.081</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">0.06</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">0.069</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">0.000</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">0.024</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">0.087</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">0.086</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">0.085</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">0.094</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">0.008</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">0.038</td></tr><tr><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center"> </td><td colspan="12" style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center"><b><span class="html-italic">Platydracus stercorarius</span></b></td></tr><tr><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center"><b>Years</b></td><td colspan="6" style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center"><b>2006/2007</b></td><td colspan="6" style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center"><b>2012/2013</b></td></tr><tr><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">Algorithm</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">GLM</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">GAM</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">MAXENT</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">RF</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">GBM</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">Ensemble</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">GLM</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">GAM</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">MAXENT</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">RF</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">GBM</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">Ensemble</td></tr><tr><td class="html-align-center" valign="middle" align="center">Altitude</td><td class="html-align-center" valign="middle" align="center">0.107</td><td class="html-align-center" valign="middle" align="center">0.077</td><td class="html-align-center" valign="middle" align="center">0.029</td><td class="html-align-center" valign="middle" align="center">0.063</td><td class="html-align-center" valign="middle" align="center">0.002</td><td class="html-align-center" valign="middle" align="center">0.029</td><td class="html-align-center" valign="middle" align="center">0.136</td><td class="html-align-center" valign="middle" align="center">0.084</td><td class="html-align-center" valign="middle" align="center">0.144</td><td class="html-align-center" valign="middle" align="center">0.076</td><td class="html-align-center" valign="middle" align="center">0.038</td><td class="html-align-center" valign="middle" align="center">0.041</td></tr><tr><td class="html-align-center" valign="middle" align="center">Northness</td><td class="html-align-center" valign="middle" align="center">0.149</td><td class="html-align-center" valign="middle" align="center">0.145</td><td class="html-align-center" valign="middle" align="center">0.087</td><td style="background:#D9D9D9" class="html-align-center" valign="middle" align="center">0.167</td><td class="html-align-center" valign="middle" align="center">0.019</td><td class="html-align-center" valign="middle" align="center">0.083</td><td class="html-align-center" valign="middle" align="center">0.141</td><td class="html-align-center" valign="middle" align="center">0.124</td><td class="html-align-center" valign="middle" align="center">0.063</td><td class="html-align-center" valign="middle" align="center">0.136</td><td class="html-align-center" valign="middle" align="center">0.025</td><td class="html-align-center" valign="middle" align="center">0.07</td></tr><tr><td class="html-align-center" valign="middle" align="center">Eastness</td><td class="html-align-center" valign="middle" align="center">0.133</td><td class="html-align-center" valign="middle" align="center">0.096</td><td class="html-align-center" valign="middle" align="center">0.029</td><td class="html-align-center" valign="middle" align="center">0.155</td><td class="html-align-center" valign="middle" align="center">0.003</td><td class="html-align-center" valign="middle" align="center">0.033</td><td style="background:#D9D9D9" class="html-align-center" valign="middle" align="center">0.149</td><td class="html-align-center" valign="middle" align="center">0.128</td><td class="html-align-center" valign="middle" align="center">0.051</td><td class="html-align-center" valign="middle" align="center">0.139</td><td class="html-align-center" valign="middle" align="center">0.02</td><td class="html-align-center" valign="middle" align="center">0.064</td></tr><tr><td class="html-align-center" valign="middle" align="center">Slope</td><td class="html-align-center" valign="middle" align="center">0.123</td><td class="html-align-center" valign="middle" align="center">0.131</td><td class="html-align-center" valign="middle" align="center">0.005</td><td class="html-align-center" valign="middle" align="center">0.052</td><td class="html-align-center" valign="middle" align="center">0.001</td><td class="html-align-center" valign="middle" align="center">0.019</td><td class="html-align-center" valign="middle" align="center">0.142</td><td class="html-align-center" valign="middle" align="center">0.118</td><td class="html-align-center" valign="middle" align="center">0.156</td><td class="html-align-center" valign="middle" align="center">0.071</td><td class="html-align-center" valign="middle" align="center">0.086</td><td class="html-align-center" valign="middle" align="center">0.076</td></tr><tr><td class="html-align-center" valign="middle" align="center">Ruggedness</td><td class="html-align-center" valign="middle" align="center">0.145</td><td style="background:#D9D9D9" class="html-align-center" valign="middle" align="center">0.165</td><td class="html-align-center" valign="middle" align="center">0.148</td><td class="html-align-center" valign="middle" align="center">0.107</td><td style="background:#D9D9D9" class="html-align-center" valign="middle" align="center">0.174</td><td class="html-align-center" valign="middle" align="center">0.147</td><td class="html-align-center" valign="middle" align="center">0.12</td><td style="background:#D9D9D9" class="html-align-center" valign="middle" align="center">0.152</td><td style="background:#A6A6A6" class="html-align-center" valign="middle" align="center">0.278</td><td style="background:#D9D9D9" class="html-align-center" valign="middle" align="center">0.238</td><td style="background:#D9D9D9" class="html-align-center" valign="middle" align="center">0.299</td><td style="background:#D9D9D9" class="html-align-center" valign="middle" align="center">0.289</td></tr><tr><td class="html-align-center" valign="middle" align="center">Greenness</td><td style="background:#A6A6A6" class="html-align-center" valign="middle" align="center">0.189</td><td style="background:#A6A6A6" class="html-align-center" valign="middle" align="center">0.268</td><td style="background:#A6A6A6" class="html-align-center" valign="middle" align="center">0.516</td><td style="background:#A6A6A6" class="html-align-center" valign="middle" align="center">0.294</td><td style="background:#A6A6A6" class="html-align-center" valign="middle" align="center">0.649</td><td style="background:#A6A6A6" class="html-align-center" valign="middle" align="center">0.532</td><td style="background:#A6A6A6" class="html-align-center" valign="middle" align="center">0.178</td><td style="background:#A6A6A6" class="html-align-center" valign="middle" align="center">0.254</td><td style="background:#D9D9D9" class="html-align-center" valign="middle" align="center">0.227</td><td style="background:#A6A6A6" class="html-align-center" valign="middle" align="center">0.247</td><td style="background:#A6A6A6" class="html-align-center" valign="middle" align="center">0.494</td><td style="background:#A6A6A6" class="html-align-center" valign="middle" align="center">0.403</td></tr><tr><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">Wetness</td><td style="border-bottom:solid thin;background:#D9D9D9" class="html-align-center" valign="middle" align="center">0.152</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">0.119</td><td style="border-bottom:solid thin;background:#D9D9D9" class="html-align-center" valign="middle" align="center">0.186</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">0.162</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">0.152</td><td style="border-bottom:solid thin;background:#D9D9D9" class="html-align-center" valign="middle" align="center">0.157</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">0.134</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">0.14</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">0.082</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">0.092</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">0.038</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">0.057</td></tr><tr><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center"> </td><td colspan="12" style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center"><b><span class="html-italic">Ocypus ophthalmicus</span></b></td></tr><tr><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center"><b>Years</b></td><td colspan="6" style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center"><b>2006/2007</b></td><td colspan="6" style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center"><b>2012/2013</b></td></tr><tr><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">Algorithm</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">GLM</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">GAM</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">MAXENT</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">RF</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">GBM</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">Ensemble</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">GLM</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">GAM</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">MAXENT</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">RF</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">GBM</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">Ensemble</td></tr><tr><td class="html-align-center" valign="middle" align="center">Altitude</td><td class="html-align-center" valign="middle" align="center">0.161</td><td class="html-align-center" valign="middle" align="center">0.101</td><td class="html-align-center" valign="middle" align="center">0.074</td><td class="html-align-center" valign="middle" align="center">0.044</td><td class="html-align-center" valign="middle" align="center">0.002</td><td class="html-align-center" valign="middle" align="center">0.052</td><td class="html-align-center" valign="middle" align="center">0.155</td><td class="html-align-center" valign="middle" align="center">0.127</td><td style="background:#D9D9D9" class="html-align-center" valign="middle" align="center">0.222</td><td class="html-align-center" valign="middle" align="center">0.133</td><td class="html-align-center" valign="middle" align="center">0.187</td><td class="html-align-center" valign="middle" align="center">0.096</td></tr><tr><td class="html-align-center" valign="middle" align="center">Northness</td><td class="html-align-center" valign="middle" align="center">0.135</td><td style="background:#D9D9D9" class="html-align-center" valign="middle" align="center">0.194</td><td class="html-align-center" valign="middle" align="center">0.067</td><td style="background:#D9D9D9" class="html-align-center" valign="middle" align="center">0.232</td><td class="html-align-center" valign="middle" align="center">0.054</td><td class="html-align-center" valign="middle" align="center">0.143</td><td class="html-align-center" valign="middle" align="center">0.097</td><td style="background:#A6A6A6" class="html-align-center" valign="middle" align="center">0.238</td><td class="html-align-center" valign="middle" align="center">0.059</td><td style="background:#D9D9D9" class="html-align-center" valign="middle" align="center">0.221</td><td class="html-align-center" valign="middle" align="center">0.108</td><td class="html-align-center" valign="middle" align="center">0.166</td></tr><tr><td class="html-align-center" valign="middle" align="center">Eastness</td><td class="html-align-center" valign="middle" align="center">0.133</td><td class="html-align-center" valign="middle" align="center">0.16</td><td class="html-align-center" valign="middle" align="center">0.043</td><td class="html-align-center" valign="middle" align="center">0.122</td><td class="html-align-center" valign="middle" align="center">0.005</td><td class="html-align-center" valign="middle" align="center">0.076</td><td class="html-align-center" valign="middle" align="center">0.163</td><td style="background:#D9D9D9" class="html-align-center" valign="middle" align="center">0.208</td><td class="html-align-center" valign="middle" align="center">0.025</td><td class="html-align-center" valign="middle" align="center">0.139</td><td class="html-align-center" valign="middle" align="center">0.002</td><td class="html-align-center" valign="middle" align="center">0.081</td></tr><tr><td class="html-align-center" valign="middle" align="center">Slope</td><td class="html-align-center" valign="middle" align="center">0.05</td><td class="html-align-center" valign="middle" align="center">0.076</td><td class="html-align-center" valign="middle" align="center">0.031</td><td class="html-align-center" valign="middle" align="center">0.06</td><td class="html-align-center" valign="middle" align="center">0.057</td><td class="html-align-center" valign="middle" align="center">0.037</td><td class="html-align-center" valign="middle" align="center">0.045</td><td class="html-align-center" valign="middle" align="center">0.054</td><td class="html-align-center" valign="middle" align="center">0.02</td><td class="html-align-center" valign="middle" align="center">0.042</td><td class="html-align-center" valign="middle" align="center">0.027</td><td class="html-align-center" valign="middle" align="center">0.029</td></tr><tr><td class="html-align-center" valign="middle" align="center">Ruggedness</td><td style="background:#D9D9D9" class="html-align-center" valign="middle" align="center">0.208</td><td class="html-align-center" valign="middle" align="center">0.166</td><td style="background:#D9D9D9" class="html-align-center" valign="middle" align="center">0.313</td><td style="background:#A6A6A6" class="html-align-center" valign="middle" align="center">0.266</td><td style="background:#A6A6A6" class="html-align-center" valign="middle" align="center">0.432</td><td style="background:#D9D9D9" class="html-align-center" valign="middle" align="center">0.319</td><td style="background:#A6A6A6" class="html-align-center" valign="middle" align="center">0.199</td><td class="html-align-center" valign="middle" align="center">0.16</td><td style="background:#A6A6A6" class="html-align-center" valign="middle" align="center">0.437</td><td style="background:#A6A6A6" class="html-align-center" valign="middle" align="center">0.225</td><td style="background:#A6A6A6" class="html-align-center" valign="middle" align="center">0.341</td><td style="background:#A6A6A6" class="html-align-center" valign="middle" align="center">0.324</td></tr><tr><td class="html-align-center" valign="middle" align="center">Greenness</td><td style="background:#A6A6A6" class="html-align-center" valign="middle" align="center">0.231</td><td style="background:#A6A6A6" class="html-align-center" valign="middle" align="center">0.2</td><td style="background:#A6A6A6" class="html-align-center" valign="middle" align="center">0.409</td><td class="html-align-center" valign="middle" align="center">0.18</td><td style="background:#D9D9D9" class="html-align-center" valign="middle" align="center">0.425</td><td style="background:#A6A6A6" class="html-align-center" valign="middle" align="center">0.32</td><td style="background:#D9D9D9" class="html-align-center" valign="middle" align="center">0.196</td><td class="html-align-center" valign="middle" align="center">0.144</td><td class="html-align-center" valign="middle" align="center">0.176</td><td class="html-align-center" valign="middle" align="center">0.176</td><td style="background:#D9D9D9" class="html-align-center" valign="middle" align="center">0.313</td><td style="background:#D9D9D9" class="html-align-center" valign="middle" align="center">0.256</td></tr><tr><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">Wetness</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">0.083</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">0.103</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">0.063</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">0.095</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">0.025</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">0.053</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">0.145</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">0.07</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">0.062</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">0.063</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">0.022</td><td style="border-bottom:solid thin" class="html-align-center" valign="middle" align="center">0.048</td></tr></tbody>
</table>
</div></section>
<section id="html-copyright"><br>Â© 2019 by the authors. Licensee MDPI, 
Basel, Switzerland. This article is an open access article distributed 
under the terms and conditions of the Creative Commons Attribution (CC 
BY) license (<a href="https://creativecommons.org/licenses/by/4.0/" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>).</section>
</div>
</div>
</article>
</div>
</div>
</div>
</div>
</div>
</section>
<div id="footer">
<div class="journal-info">
<span>
<em><a class="Var_JournalInfo" href="https://www.mdpi.com/journal/remotesensing">Remote Sens.</a></em>,
EISSN 2072-4292,
Published by MDPI AG
</span>
<span>
<a data-dropdown="drop-view-disclaimer-journal" aria-controls="drop-view-disclaimer-journal" aria-expanded="false" data-options="align:top; is_hover:true; hover_timeout:2000;">
Disclaimer
</a>
<div id="drop-view-disclaimer-journal" class="f-dropdown label__btn__dropdown label__btn__dropdown--wide text-left" data-dropdown-content="" aria-hidden="true" tabindex="-1">
The statements, opinions and data contained in the journal <em>Remote Sensing</em> are solely
those of the individual authors and contributors and not of the publisher and the editor(s).
</div>
</span>
<div class="large-right">
<span>
<a href="https://www.mdpi.com/rss/journal/remotesensing">RSS</a>
</span>
<span>
<a href="https://www.mdpi.com/journal/remotesensing/toc-alert">Content Alert</a>
</span>
</div>
</div>
<div class="row full-width footer-links" data-equalizer="footer" data-equalizer-mq="small">
<div class="large-2 large-push-4 medium-3 small-6 columns" data-equalizer-watch="footer" style="height: 199px;">
<h3>
Further Information
</h3>
<a href="https://www.mdpi.com/apc">
Article Processing Charges
</a>
<a href="https://www.mdpi.com/about/payment">
Pay an Invoice
</a>
<a href="https://www.mdpi.com/openaccess">
Open Access Policy
</a>
<a href="https://www.mdpi.com/about/contact">
Contact MDPI
</a>
<a href="https://www.mdpi.com/about/jobs">
Jobs at MDPI
</a>
</div>
<div class="large-2 large-push-4 medium-3 small-6 columns" data-equalizer-watch="footer" style="height: 199px;">
<h3>
Guidelines
</h3>
<a href="https://www.mdpi.com/authors">
For Authors
</a>
<a href="https://www.mdpi.com/reviewers">
For Reviewers
</a>
<a href="https://www.mdpi.com/editors">
For Editors
</a>
<a href="https://www.mdpi.com/librarians">
For Librarians
</a>
<a href="https://www.mdpi.com/publishing_services">
For Publishers
</a>
<a href="https://www.mdpi.com/societies">
For Societies
</a>
</div>
<div class="large-2 large-push-4 medium-3 small-6 columns">
<h3>
MDPI Initiatives
</h3>
<a href="https://www.mdpi.com/ioap">
Institutional Open Access Program (IOAP)
</a>
<a href="https://sciforum.net/" target="_blank" rel="noopener noreferrer">
Sciforum
</a>
<a href="https://www.preprints.org/" target="_blank" rel="noopener noreferrer">
Preprints
</a>
<a href="https://www.scilit.net/" target="_blank" rel="noopener noreferrer">
Scilit
</a>
<a href="https://sciprofiles.com/" target="_blank" rel="noopener noreferrer">
SciProfiles
</a>
<a href="https://www.mdpi.com/books" target="_blank" rel="noopener noreferrer">
MDPI Books
</a>
<a href="https://encyclopedia.pub/" target="_blank" rel="noopener noreferrer">
Encyclopedia
</a>
<a href="http://blog.mdpi.com/" onclick="if (!window.__cfRLUnblockHandlers) return false; ga('send', 'pageview', 'homeBlog');" target="_blank" rel="noopener noreferrer">
MDPI Blog
</a>
</div>
<div class="large-2 large-push-4 medium-3 small-6 right-border-large-without columns UA_FooterFollowMDPI">
<h3>
Follow MDPI
</h3>
<a href="https://www.linkedin.com/company/mdpi" target="_blank" rel="noopener noreferrer">
LinkedIn
</a>
<a href="https://www.facebook.com/MDPIOpenAccessPublishing" target="_blank" rel="noopener noreferrer">
Facebook
</a>
<a href="https://twitter.com/MDPIOpenAccess" target="_blank" rel="noopener noreferrer">
Twitter
</a>
</div>
<div id="footer-subscribe" class="large-4 large-pull-8 medium-12 small-12 left-border-large columns">
<div class="footer-subscribe__container">
<img class="show-for-large-up" src="mdpi-pub-logo-white-small.png" alt="MDPI" title="MDPI Open Access Journals" style="height: 50px; margin-bottom: 10px;">
<form id="newsletter" method="POST" action="/subscribe">
<p>
Subscribe to receive issue release notifications and newsletters from MDPI journals
</p>
<select multiple="multiple" id="newsletter-journal" class="foundation-select jqmsLoaded" name="journals[]" style="display: none;">
<option value="acoustics">Acoustics</option>
<option value="actuators">Actuators</option>
<option value="admsci">Administrative Sciences</option>
<option value="aerospace">Aerospace</option>
<option value="agriculture">Agriculture</option>
<option value="agriengineering">AgriEngineering</option>
<option value="agronomy">Agronomy</option>
<option value="ai">AI</option>
<option value="algorithms">Algorithms</option>
<option value="animals">Animals</option>
<option value="antibiotics">Antibiotics</option>
<option value="antibodies">Antibodies</option>
<option value="antioxidants">Antioxidants</option>
<option value="applmech">Applied Mechanics</option>
<option value="applnano">Applied Nano</option>
<option value="applsci">Applied Sciences</option>
<option value="asi">Applied System Innovation</option>
 <option value="arts">Arts</option>
<option value="atmosphere">Atmosphere</option>
<option value="atoms">Atoms</option>
<option value="axioms">Axioms</option>
<option value="batteries">Batteries</option>
<option value="behavsci">Behavioral Sciences</option>
<option value="beverages">Beverages</option>
<option value="BDCC">Big Data and Cognitive Computing</option>
<option value="bioengineering">Bioengineering</option>
<option value="biology">Biology</option>
<option value="biomedicines">Biomedicines</option>
<option value="biomimetics">Biomimetics</option>
<option value="biomolecules">Biomolecules</option>
<option value="biosensors">Biosensors</option>
<option value="bloods">Bloods</option>
<option value="brainsci">Brain Sciences</option>
<option value="buildings">Buildings</option>
<option value="carbon">C â€” Journal of Carbon Research</option>
<option value="cancers">Cancers</option>
<option value="catalysts">Catalysts</option>
<option value="cells">Cells</option>
<option value="ceramics">Ceramics</option>
<option value="challenges">Challenges</option>
<option value="ChemEngineering">ChemEngineering</option>
<option value="chemistry">Chemistry</option>
<option value="chemosensors">Chemosensors</option>
<option value="children">Children</option>
<option value="cleantechnol">Clean Technologies</option>
<option value="climate">Climate</option>
<option value="clockssleep">Clocks &amp; Sleep</option>
<option value="coatings">Coatings</option>
<option value="colloids">Colloids and Interfaces</option>
<option value="computation">Computation</option>
<option value="computers">Computers</option>
<option value="condensedmatter">Condensed Matter</option>
<option value="cmd">Corrosion and Materials Degradation</option>
<option value="cosmetics">Cosmetics</option>
<option value="cryptography">Cryptography</option>
<option value="crystals">Crystals</option>
<option value="dairy">Dairy</option>
<option value="data">Data</option>
<option value="dentistry">Dentistry Journal</option>
<option value="designs">Designs</option>
<option value="diagnostics">Diagnostics</option>
<option value="diseases">Diseases</option>
<option value="diversity">Diversity</option>
<option value="drones">Drones</option>
<option value="econometrics">Econometrics</option>
<option value="economies">Economies</option>
<option value="education">Education Sciences</option>
<option value="electrochem">Electrochem</option>
<option value="electronics">Electronics</option>
<option value="endocrines">Endocrines</option>
<option value="energies">Energies</option>
<option value="entropy">Entropy</option>
<option value="environments">Environments</option>
<option value="epigenomes">Epigenomes</option>
<option value="ejbc">European Journal of Burn Care</option>
<option value="ejihpe">European Journal of Investigation in Health, Psychology and Education</option>
<option value="fermentation">Fermentation</option>
<option value="fibers">Fibers</option>
<option value="fire">Fire</option>
<option value="fishes">Fishes</option>
<option value="fluids">Fluids</option>
<option value="foods">Foods</option>
<option value="forecasting">Forecasting</option>
<option value="forests">Forests</option>
<option value="fractalfract">Fractal and Fractional</option>
<option value="futureinternet">Future Internet</option>
<option value="galaxies">Galaxies</option>
<option value="games">Games</option>
<option value="gastrointestdisord">Gastrointestinal Disorders</option>
<option value="gels">Gels</option>
<option value="genealogy">Genealogy</option>
<option value="genes">Genes</option>
<option value="geohazards">GeoHazards</option>
<option value="geosciences">Geosciences</option>
<option value="geriatrics">Geriatrics</option>
<option value="healthcare">Healthcare</option>
<option value="hearts">Hearts</option>
<option value="heritage">Heritage</option>
<option value="high-throughput">High-Throughput</option>
<option value="horticulturae">Horticulturae</option>
<option value="humanities">Humanities</option>
<option value="hydrology">Hydrology</option>
<option value="informatics">Informatics</option>
<option value="information">Information</option>
<option value="infrastructures">Infrastructures</option>
<option value="inorganics">Inorganics</option>
<option value="insects">Insects</option>
<option value="instruments">Instruments</option>
<option value="ijd">International Journal of Diabetology</option>
<option value="ijerph">International Journal of Environmental Research and Public Health</option>
<option value="ijfs">International Journal of Financial Studies</option>
<option value="ijms">International Journal of Molecular Sciences</option>
<option value="IJNS">International Journal of Neonatal Screening</option>
<option value="ijtpp">International Journal of Turbomachinery, Propulsion and Power</option>
<option value="inventions">Inventions</option>
<option value="IoT">IoT</option>
<option value="ijgi">ISPRS International Journal of Geo-Information</option>
<option value="J">J â€” Multidisciplinary Scientific Journal</option>
<option value="jcdd">Journal of Cardiovascular Development and Disease</option>
<option value="jcm">Journal of Clinical Medicine</option>
<option value="jcs">Journal of Composites Science</option>
<option value="jdb">Journal of Developmental Biology</option>
<option value="jfb">Journal of Functional Biomaterials</option>
<option value="jfmk">Journal of Functional Morphology and Kinesiology</option>
<option value="jof">Journal of Fungi</option>
<option value="jimaging">Journal of Imaging</option>
<option value="jintelligence">Journal of Intelligence</option>
<option value="jlpea">Journal of Low Power Electronics and Applications</option>
<option value="jmmp">Journal of Manufacturing and Materials Processing</option>
<option value="jmse">Journal of Marine Science and Engineering</option>
<option value="JOItmC">Journal of Open Innovation: Technology, Market, and Complexity</option>
 <option value="jpm">Journal of Personalized Medicine</option>
<option value="jrfm">Journal of Risk and Financial Management</option>
<option value="jsan">Journal of Sensor and Actuator Networks</option>
<option value="land">Land</option>
<option value="languages">Languages</option>
<option value="laws">Laws</option>
<option value="life">Life</option>
<option value="logistics">Logistics</option>
<option value="lubricants">Lubricants</option>
<option value="make">Machine Learning and Knowledge Extraction</option>
<option value="machines">Machines</option>
<option value="magnetochemistry">Magnetochemistry</option>
<option value="marinedrugs">Marine Drugs</option>
<option value="materials">Materials</option>
<option value="mca">Mathematical and Computational Applications</option>
<option value="mathematics">Mathematics</option>
<option value="medsci">Medical Sciences</option>
<option value="medicina">Medicina</option>
<option value="medicines">Medicines</option>
<option value="membranes">Membranes</option>
<option value="metabolites">Metabolites</option>
<option value="metals">Metals</option>
<option value="mps">Methods and Protocols</option>
<option value="micromachines">Micromachines</option>
<option value="microorganisms">Microorganisms</option>
<option value="minerals">Minerals</option>
<option value="molbank">Molbank</option>
<option value="molecules">Molecules</option>
<option value="mti">Multimodal Technologies and Interaction</option>
<option value="nanomaterials">Nanomaterials</option>
<option value="nitrogen">Nitrogen</option>
<option value="ncrna">Non-Coding RNA</option>
<option value="nutrients">Nutrients</option>
<option value="oceans">Oceans</option>
<option value="optics">Optics</option>
<option value="particles">Particles</option>
<option value="pathogens">Pathogens</option>
<option value="pharmaceuticals">Pharmaceuticals</option>
<option value="pharmaceutics">Pharmaceutics</option>
<option value="pharmacy">Pharmacy</option>
<option value="philosophies">Philosophies</option>
<option value="photonics">Photonics</option>
<option value="physics">Physics</option>
<option value="plants">Plants</option>
<option value="plasma">Plasma</option>
<option value="polymers">Polymers</option>
<option value="proceedings">Proceedings</option>
<option value="processes">Processes</option>
<option value="prosthesis">Prosthesis</option>
<option value="proteomes">Proteomes</option>
<option value="psych">Psych</option>
<option value="publications">Publications</option>
<option value="qubs">Quantum Beam Science</option>
<option value="quantumrep">Quantum Reports</option>
<option value="quaternary">Quaternary</option>
<option value="reactions">Reactions</option>
<option value="recycling">Recycling</option>
<option value="religions">Religions</option>
<option value="remotesensing">Remote Sensing</option>
<option value="reports">Reports â€” Medical Cases, Images, and Videos</option>
<option value="reprodmed">Reproductive Medicine</option>
<option value="resources">Resources</option>
<option value="risks">Risks</option>
<option value="robotics">Robotics</option>
<option value="safety">Safety</option>
<option value="sci">Sci</option>
<option value="scipharm">Scientia Pharmaceutica</option>
<option value="sensors">Sensors</option>
<option value="separations">Separations</option>
<option value="signals">Signals</option>
<option value="sinusitis">Sinusitis</option>
<option value="smartcities">Smart Cities</option>
<option value="socsci">Social Sciences</option>
<option value="societies">Societies</option>
<option value="soilsystems">Soil Systems</option>
<option value="sports">Sports</option>
<option value="stats">Stats</option>
<option value="surfaces">Surfaces</option>
<option value="surgeries">Surgeries</option>
<option value="sustainability">Sustainability</option>
<option value="symmetry">Symmetry</option>
<option value="systems">Systems</option>
<option value="technologies">Technologies</option>
<option value="telecom">Telecom</option>
<option value="toxics">Toxics</option>
<option value="toxins">Toxins</option>
<option value="tropicalmed">Tropical Medicine and Infectious Disease</option>
<option value="universe">Universe</option>
<option value="urbansci">Urban Science</option>
<option value="vaccines">Vaccines</option>
<option value="vehicles">Vehicles</option>
<option value="vetsci">Veterinary Sciences</option>
<option value="vibration">Vibration</option>
<option value="viruses">Viruses</option>
<option value="vision">Vision</option>
<option value="water">Water</option>
<option value="world">World</option>
<option value="wevj">World Electric Vehicle Journal</option>
</select><div class="ms-options-wrap" style="position: relative;"><button>Select options</button><div class="ms-options" style="min-height: 130px; max-height: 130px; overflow: auto; display: none;"><div class="ms-search"><input placeholder="Search" type="text"></div><ul style="column-count: 1; column-gap: 0px;"><li class=""><label for="ms-opt-1" style="padding-left: 23px;"><input value="acoustics" title="Acoustics" id="ms-opt-1" type="checkbox">Acoustics</label></li><li class=""><label for="ms-opt-2" style="padding-left: 23px;"><input value="actuators" title="Actuators" id="ms-opt-2" type="checkbox">Actuators</label></li><li class=""><label for="ms-opt-3" style="padding-left: 23px;"><input value="admsci" title="Administrative Sciences" id="ms-opt-3" type="checkbox">Administrative Sciences</label></li><li class=""><label for="ms-opt-4" style="padding-left: 23px;"><input value="aerospace" title="Aerospace" id="ms-opt-4" type="checkbox">Aerospace</label></li><li class=""><label for="ms-opt-5" style="padding-left: 23px;"><input value="agriculture" title="Agriculture" id="ms-opt-5" type="checkbox">Agriculture</label></li><li class=""><label for="ms-opt-6" style="padding-left: 23px;"><input value="agriengineering" title="AgriEngineering" id="ms-opt-6" type="checkbox">AgriEngineering</label></li><li class=""><label for="ms-opt-7" style="padding-left: 23px;"><input value="agronomy" title="Agronomy" id="ms-opt-7" type="checkbox">Agronomy</label></li><li class=""><label for="ms-opt-8" style="padding-left: 23px;"><input value="ai" title="AI" id="ms-opt-8" type="checkbox">AI</label></li><li class=""><label for="ms-opt-9" style="padding-left: 23px;"><input value="algorithms" title="Algorithms" id="ms-opt-9" type="checkbox">Algorithms</label></li><li class=""><label for="ms-opt-10" style="padding-left: 23px;"><input value="animals" title="Animals" id="ms-opt-10" type="checkbox">Animals</label></li><li class=""><label for="ms-opt-11" style="padding-left: 23px;"><input value="antibiotics" title="Antibiotics" id="ms-opt-11" type="checkbox">Antibiotics</label></li><li class=""><label for="ms-opt-12" style="padding-left: 23px;"><input value="antibodies" title="Antibodies" id="ms-opt-12" type="checkbox">Antibodies</label></li><li class=""><label for="ms-opt-13" style="padding-left: 23px;"><input value="antioxidants" title="Antioxidants" id="ms-opt-13" type="checkbox">Antioxidants</label></li><li class=""><label for="ms-opt-14" style="padding-left: 23px;"><input value="applmech" title="Applied Mechanics" id="ms-opt-14" type="checkbox">Applied Mechanics</label></li><li class=""><label for="ms-opt-15" style="padding-left: 23px;"><input value="applnano" title="Applied Nano" id="ms-opt-15" type="checkbox">Applied Nano</label></li><li class=""><label for="ms-opt-16" style="padding-left: 23px;"><input value="applsci" title="Applied Sciences" id="ms-opt-16" type="checkbox">Applied Sciences</label></li><li class=""><label for="ms-opt-17" style="padding-left: 23px;"><input value="asi" title="Applied System Innovation" id="ms-opt-17" type="checkbox">Applied System Innovation</label></li><li class=""><label for="ms-opt-18" style="padding-left: 23px;"><input value="arts" title="Arts" id="ms-opt-18" type="checkbox">Arts</label></li><li class=""><label for="ms-opt-19" style="padding-left: 23px;"><input value="atmosphere" title="Atmosphere" id="ms-opt-19" type="checkbox">Atmosphere</label></li><li class=""><label for="ms-opt-20" style="padding-left: 23px;"><input value="atoms" title="Atoms" id="ms-opt-20" type="checkbox">Atoms</label></li><li class=""><label for="ms-opt-21" style="padding-left: 23px;"><input value="axioms" title="Axioms" id="ms-opt-21" type="checkbox">Axioms</label></li><li class=""><label for="ms-opt-22" style="padding-left: 23px;"><input value="batteries" title="Batteries" id="ms-opt-22" type="checkbox">Batteries</label></li><li class=""><label for="ms-opt-23" style="padding-left: 23px;"><input value="behavsci" title="Behavioral Sciences" id="ms-opt-23" type="checkbox">Behavioral Sciences</label></li><li class=""><label for="ms-opt-24" style="padding-left: 23px;"><input value="beverages" title="Beverages" id="ms-opt-24" type="checkbox">Beverages</label></li><li class=""><label for="ms-opt-25" style="padding-left: 23px;"><input value="BDCC" title="Big Data and Cognitive Computing" id="ms-opt-25" type="checkbox">Big Data and Cognitive Computing</label></li><li class=""><label for="ms-opt-26" style="padding-left: 23px;"><input value="bioengineering" title="Bioengineering" id="ms-opt-26" type="checkbox">Bioengineering</label></li><li class=""><label for="ms-opt-27" style="padding-left: 23px;"><input value="biology" title="Biology" id="ms-opt-27" type="checkbox">Biology</label></li><li class=""><label for="ms-opt-28" style="padding-left: 23px;"><input value="biomedicines" title="Biomedicines" id="ms-opt-28" type="checkbox">Biomedicines</label></li><li class=""><label for="ms-opt-29" style="padding-left: 23px;"><input value="biomimetics" title="Biomimetics" id="ms-opt-29" type="checkbox">Biomimetics</label></li><li class=""><label for="ms-opt-30" style="padding-left: 23px;"><input value="biomolecules" title="Biomolecules" id="ms-opt-30" type="checkbox">Biomolecules</label></li><li class=""><label for="ms-opt-31" style="padding-left: 23px;"><input value="biosensors" title="Biosensors" id="ms-opt-31" type="checkbox">Biosensors</label></li><li class=""><label for="ms-opt-32" style="padding-left: 23px;"><input value="bloods" title="Bloods" id="ms-opt-32" type="checkbox">Bloods</label></li><li class=""><label for="ms-opt-33" style="padding-left: 23px;"><input value="brainsci" title="Brain Sciences" id="ms-opt-33" type="checkbox">Brain Sciences</label></li><li class=""><label for="ms-opt-34" style="padding-left: 23px;"><input value="buildings" title="Buildings" id="ms-opt-34" type="checkbox">Buildings</label></li><li class=""><label for="ms-opt-35" style="padding-left: 23px;"><input value="carbon" title="C â€” Journal of Carbon Research" id="ms-opt-35" type="checkbox">C â€” Journal of Carbon Research</label></li><li class=""><label for="ms-opt-36" style="padding-left: 23px;"><input value="cancers" title="Cancers" id="ms-opt-36" type="checkbox">Cancers</label></li><li class=""><label for="ms-opt-37" style="padding-left: 23px;"><input value="catalysts" title="Catalysts" id="ms-opt-37" type="checkbox">Catalysts</label></li><li class=""><label for="ms-opt-38" style="padding-left: 23px;"><input value="cells" title="Cells" id="ms-opt-38" type="checkbox">Cells</label></li><li class=""><label for="ms-opt-39" style="padding-left: 23px;"><input value="ceramics" title="Ceramics" id="ms-opt-39" type="checkbox">Ceramics</label></li><li class=""><label for="ms-opt-40" style="padding-left: 23px;"><input value="challenges" title="Challenges" id="ms-opt-40" type="checkbox">Challenges</label></li><li class=""><label for="ms-opt-41" style="padding-left: 23px;"><input value="ChemEngineering" title="ChemEngineering" id="ms-opt-41" type="checkbox">ChemEngineering</label></li><li class=""><label for="ms-opt-42" style="padding-left: 23px;"><input value="chemistry" title="Chemistry" id="ms-opt-42" type="checkbox">Chemistry</label></li><li class=""><label for="ms-opt-43" style="padding-left: 23px;"><input value="chemosensors" title="Chemosensors" id="ms-opt-43" type="checkbox">Chemosensors</label></li><li class=""><label for="ms-opt-44" style="padding-left: 23px;"><input value="children" title="Children" id="ms-opt-44" type="checkbox">Children</label></li><li class=""><label for="ms-opt-45" style="padding-left: 23px;"><input value="cleantechnol" title="Clean Technologies" id="ms-opt-45" type="checkbox">Clean Technologies</label></li><li class=""><label for="ms-opt-46" style="padding-left: 23px;"><input value="climate" title="Climate" id="ms-opt-46" type="checkbox">Climate</label></li><li class=""><label for="ms-opt-47" style="padding-left: 23px;"><input value="clockssleep" title="Clocks &amp; Sleep" id="ms-opt-47" type="checkbox">Clocks &amp; Sleep</label></li><li class=""><label for="ms-opt-48" style="padding-left: 23px;"><input value="coatings" title="Coatings" id="ms-opt-48" type="checkbox">Coatings</label></li><li class=""><label for="ms-opt-49" style="padding-left: 23px;"><input value="colloids" title="Colloids and Interfaces" id="ms-opt-49" type="checkbox">Colloids and Interfaces</label></li><li class=""><label for="ms-opt-50" style="padding-left: 23px;"><input value="computation" title="Computation" id="ms-opt-50" type="checkbox">Computation</label></li><li class=""><label for="ms-opt-51" style="padding-left: 23px;"><input value="computers" title="Computers" id="ms-opt-51" type="checkbox">Computers</label></li><li class=""><label for="ms-opt-52" style="padding-left: 23px;"><input value="condensedmatter" title="Condensed Matter" id="ms-opt-52" type="checkbox">Condensed Matter</label></li><li class=""><label for="ms-opt-53" style="padding-left: 23px;"><input value="cmd" title="Corrosion and Materials Degradation" id="ms-opt-53" type="checkbox">Corrosion and Materials Degradation</label></li><li class=""><label for="ms-opt-54" style="padding-left: 23px;"><input value="cosmetics" title="Cosmetics" id="ms-opt-54" type="checkbox">Cosmetics</label></li><li class=""><label for="ms-opt-55" style="padding-left: 23px;"><input value="cryptography" title="Cryptography" id="ms-opt-55" type="checkbox">Cryptography</label></li><li class=""><label for="ms-opt-56" style="padding-left: 23px;"><input value="crystals" title="Crystals" id="ms-opt-56" type="checkbox">Crystals</label></li><li class=""><label for="ms-opt-57" style="padding-left: 23px;"><input value="dairy" title="Dairy" id="ms-opt-57" type="checkbox">Dairy</label></li><li class=""><label for="ms-opt-58" style="padding-left: 23px;"><input value="data" title="Data" id="ms-opt-58" type="checkbox">Data</label></li><li class=""><label for="ms-opt-59" style="padding-left: 23px;"><input value="dentistry" title="Dentistry Journal" id="ms-opt-59" type="checkbox">Dentistry Journal</label></li><li class=""><label for="ms-opt-60" style="padding-left: 23px;"><input value="designs" title="Designs" id="ms-opt-60" type="checkbox">Designs</label></li><li class=""><label for="ms-opt-61" style="padding-left: 23px;"><input value="diagnostics" title="Diagnostics" id="ms-opt-61" type="checkbox">Diagnostics</label></li><li class=""><label for="ms-opt-62" style="padding-left: 23px;"><input value="diseases" title="Diseases" id="ms-opt-62" type="checkbox">Diseases</label></li><li class=""><label for="ms-opt-63" style="padding-left: 23px;"><input value="diversity" title="Diversity" id="ms-opt-63" type="checkbox">Diversity</label></li><li class=""><label for="ms-opt-64" style="padding-left: 23px;"><input value="drones" title="Drones" id="ms-opt-64" type="checkbox">Drones</label></li><li class=""><label for="ms-opt-65" style="padding-left: 23px;"><input value="econometrics" title="Econometrics" id="ms-opt-65" type="checkbox">Econometrics</label></li><li class=""><label for="ms-opt-66" style="padding-left: 23px;"><input value="economies" title="Economies" id="ms-opt-66" type="checkbox">Economies</label></li><li class=""><label for="ms-opt-67" style="padding-left: 23px;"><input value="education" title="Education Sciences" id="ms-opt-67" type="checkbox">Education Sciences</label></li><li class=""><label for="ms-opt-68" style="padding-left: 23px;"><input value="electrochem" title="Electrochem" id="ms-opt-68" type="checkbox">Electrochem</label></li><li class=""><label for="ms-opt-69" style="padding-left: 23px;"><input value="electronics" title="Electronics" id="ms-opt-69" type="checkbox">Electronics</label></li><li class=""><label for="ms-opt-70" style="padding-left: 23px;"><input value="endocrines" title="Endocrines" id="ms-opt-70" type="checkbox">Endocrines</label></li><li class=""><label for="ms-opt-71" style="padding-left: 23px;"><input value="energies" title="Energies" id="ms-opt-71" type="checkbox">Energies</label></li><li class=""><label for="ms-opt-72" style="padding-left: 23px;"><input value="entropy" title="Entropy" id="ms-opt-72" type="checkbox">Entropy</label></li><li class=""><label for="ms-opt-73" style="padding-left: 23px;"><input value="environments" title="Environments" id="ms-opt-73" type="checkbox">Environments</label></li><li class=""><label for="ms-opt-74" style="padding-left: 23px;"><input value="epigenomes" title="Epigenomes" id="ms-opt-74" type="checkbox">Epigenomes</label></li><li class=""><label for="ms-opt-75" style="padding-left: 23px;"><input value="ejbc" title="European Journal of Burn Care" id="ms-opt-75" type="checkbox">European Journal of Burn Care</label></li><li class=""><label for="ms-opt-76" style="padding-left: 23px;"><input value="ejihpe" title="European Journal of Investigation in Health, Psychology and Education" id="ms-opt-76" type="checkbox">European Journal of Investigation in Health, Psychology and Education</label></li><li class=""><label for="ms-opt-77" style="padding-left: 23px;"><input value="fermentation" title="Fermentation" id="ms-opt-77" type="checkbox">Fermentation</label></li><li class=""><label for="ms-opt-78" style="padding-left: 23px;"><input value="fibers" title="Fibers" id="ms-opt-78" type="checkbox">Fibers</label></li><li class=""><label for="ms-opt-79" style="padding-left: 23px;"><input value="fire" title="Fire" id="ms-opt-79" type="checkbox">Fire</label></li><li class=""><label for="ms-opt-80" style="padding-left: 23px;"><input value="fishes" title="Fishes" id="ms-opt-80" type="checkbox">Fishes</label></li><li class=""><label for="ms-opt-81" style="padding-left: 23px;"><input value="fluids" title="Fluids" id="ms-opt-81" type="checkbox">Fluids</label></li><li class=""><label for="ms-opt-82" style="padding-left: 23px;"><input value="foods" title="Foods" id="ms-opt-82" type="checkbox">Foods</label></li><li class=""><label for="ms-opt-83" style="padding-left: 23px;"><input value="forecasting" title="Forecasting" id="ms-opt-83" type="checkbox">Forecasting</label></li><li class=""><label for="ms-opt-84" style="padding-left: 23px;"><input value="forests" title="Forests" id="ms-opt-84" type="checkbox">Forests</label></li><li class=""><label for="ms-opt-85" style="padding-left: 23px;"><input value="fractalfract" title="Fractal and Fractional" id="ms-opt-85" type="checkbox">Fractal and Fractional</label></li><li class=""><label for="ms-opt-86" style="padding-left: 23px;"><input value="futureinternet" title="Future Internet" id="ms-opt-86" type="checkbox">Future Internet</label></li><li class=""><label for="ms-opt-87" style="padding-left: 23px;"><input value="galaxies" title="Galaxies" id="ms-opt-87" type="checkbox">Galaxies</label></li><li class=""><label for="ms-opt-88" style="padding-left: 23px;"><input value="games" title="Games" id="ms-opt-88" type="checkbox">Games</label></li><li class=""><label for="ms-opt-89" style="padding-left: 23px;"><input value="gastrointestdisord" title="Gastrointestinal Disorders" id="ms-opt-89" type="checkbox">Gastrointestinal Disorders</label></li><li class=""><label for="ms-opt-90" style="padding-left: 23px;"><input value="gels" title="Gels" id="ms-opt-90" type="checkbox">Gels</label></li><li class=""><label for="ms-opt-91" style="padding-left: 23px;"><input value="genealogy" title="Genealogy" id="ms-opt-91" type="checkbox">Genealogy</label></li><li class=""><label for="ms-opt-92" style="padding-left: 23px;"><input value="genes" title="Genes" id="ms-opt-92" type="checkbox">Genes</label></li><li class=""><label for="ms-opt-93" style="padding-left: 23px;"><input value="geohazards" title="GeoHazards" id="ms-opt-93" type="checkbox">GeoHazards</label></li><li class=""><label for="ms-opt-94" style="padding-left: 23px;"><input value="geosciences" title="Geosciences" id="ms-opt-94" type="checkbox">Geosciences</label></li><li class=""><label for="ms-opt-95" style="padding-left: 23px;"><input value="geriatrics" title="Geriatrics" id="ms-opt-95" type="checkbox">Geriatrics</label></li><li class=""><label for="ms-opt-96" style="padding-left: 23px;"><input value="healthcare" title="Healthcare" id="ms-opt-96" type="checkbox">Healthcare</label></li><li class=""><label for="ms-opt-97" style="padding-left: 23px;"><input value="hearts" title="Hearts" id="ms-opt-97" type="checkbox">Hearts</label></li><li class=""><label for="ms-opt-98" style="padding-left: 23px;"><input value="heritage" title="Heritage" id="ms-opt-98" type="checkbox">Heritage</label></li><li class=""><label for="ms-opt-99" style="padding-left: 23px;"><input value="high-throughput" title="High-Throughput" id="ms-opt-99" type="checkbox">High-Throughput</label></li><li class=""><label for="ms-opt-100" style="padding-left: 23px;"><input value="horticulturae" title="Horticulturae" id="ms-opt-100" type="checkbox">Horticulturae</label></li><li class=""><label for="ms-opt-101" style="padding-left: 23px;"><input value="humanities" title="Humanities" id="ms-opt-101" type="checkbox">Humanities</label></li><li class=""><label for="ms-opt-102" style="padding-left: 23px;"><input value="hydrology" title="Hydrology" id="ms-opt-102" type="checkbox">Hydrology</label></li><li class=""><label for="ms-opt-103" style="padding-left: 23px;"><input value="informatics" title="Informatics" id="ms-opt-103" type="checkbox">Informatics</label></li><li class=""><label for="ms-opt-104" style="padding-left: 23px;"><input value="information" title="Information" id="ms-opt-104" type="checkbox">Information</label></li><li class=""><label for="ms-opt-105" style="padding-left: 23px;"><input value="infrastructures" title="Infrastructures" id="ms-opt-105" type="checkbox">Infrastructures</label></li><li class=""><label for="ms-opt-106" style="padding-left: 23px;"><input value="inorganics" title="Inorganics" id="ms-opt-106" type="checkbox">Inorganics</label></li><li class=""><label for="ms-opt-107" style="padding-left: 23px;"><input value="insects" title="Insects" id="ms-opt-107" type="checkbox">Insects</label></li><li class=""><label for="ms-opt-108" style="padding-left: 23px;"><input value="instruments" title="Instruments" id="ms-opt-108" type="checkbox">Instruments</label></li><li class=""><label for="ms-opt-109" style="padding-left: 23px;"><input value="ijd" title="International Journal of Diabetology" id="ms-opt-109" type="checkbox">International Journal of Diabetology</label></li><li class=""><label for="ms-opt-110" style="padding-left: 23px;"><input value="ijerph" title="International Journal of Environmental Research and Public Health" id="ms-opt-110" type="checkbox">International Journal of Environmental Research and Public Health</label></li><li class=""><label for="ms-opt-111" style="padding-left: 23px;"><input value="ijfs" title="International Journal of Financial Studies" id="ms-opt-111" type="checkbox">International Journal of Financial Studies</label></li><li class=""><label for="ms-opt-112" style="padding-left: 23px;"><input value="ijms" title="International Journal of Molecular Sciences" id="ms-opt-112" type="checkbox">International Journal of Molecular Sciences</label></li><li class=""><label for="ms-opt-113" style="padding-left: 23px;"><input value="IJNS" title="International Journal of Neonatal Screening" id="ms-opt-113" type="checkbox">International Journal of Neonatal Screening</label></li><li class=""><label for="ms-opt-114" style="padding-left: 23px;"><input value="ijtpp" title="International Journal of Turbomachinery, Propulsion and Power" id="ms-opt-114" type="checkbox">International Journal of Turbomachinery, Propulsion and Power</label></li><li class=""><label for="ms-opt-115" style="padding-left: 23px;"><input value="inventions" title="Inventions" id="ms-opt-115" type="checkbox">Inventions</label></li><li class=""><label for="ms-opt-116" style="padding-left: 23px;"><input value="IoT" title="IoT" id="ms-opt-116" type="checkbox">IoT</label></li><li class=""><label for="ms-opt-117" style="padding-left: 23px;"><input value="ijgi" title="ISPRS International Journal of Geo-Information" id="ms-opt-117" type="checkbox">ISPRS International Journal of Geo-Information</label></li><li class=""><label for="ms-opt-118" style="padding-left: 23px;"><input value="J" title="J â€” Multidisciplinary Scientific Journal" id="ms-opt-118" type="checkbox">J â€” Multidisciplinary Scientific Journal</label></li><li class=""><label for="ms-opt-119" style="padding-left: 23px;"><input value="jcdd" title="Journal of Cardiovascular Development and Disease" id="ms-opt-119" type="checkbox">Journal of Cardiovascular Development and Disease</label></li><li class=""><label for="ms-opt-120" style="padding-left: 23px;"><input value="jcm" title="Journal of Clinical Medicine" id="ms-opt-120" type="checkbox">Journal of Clinical Medicine</label></li><li class=""><label for="ms-opt-121" style="padding-left: 23px;"><input value="jcs" title="Journal of Composites Science" id="ms-opt-121" type="checkbox">Journal of Composites Science</label></li><li class=""><label for="ms-opt-122" style="padding-left: 23px;"><input value="jdb" title="Journal of Developmental Biology" id="ms-opt-122" type="checkbox">Journal of Developmental Biology</label></li><li class=""><label for="ms-opt-123" style="padding-left: 23px;"><input value="jfb" title="Journal of Functional Biomaterials" id="ms-opt-123" type="checkbox">Journal of Functional Biomaterials</label></li><li class=""><label for="ms-opt-124" style="padding-left: 23px;"><input value="jfmk" title="Journal of Functional Morphology and Kinesiology" id="ms-opt-124" type="checkbox">Journal of Functional Morphology and Kinesiology</label></li><li class=""><label for="ms-opt-125" style="padding-left: 23px;"><input value="jof" title="Journal of Fungi" id="ms-opt-125" type="checkbox">Journal of Fungi</label></li><li class=""><label for="ms-opt-126" style="padding-left: 23px;"><input value="jimaging" title="Journal of Imaging" id="ms-opt-126" type="checkbox">Journal of Imaging</label></li><li class=""><label for="ms-opt-127" style="padding-left: 23px;"><input value="jintelligence" title="Journal of Intelligence" id="ms-opt-127" type="checkbox">Journal of Intelligence</label></li><li class=""><label for="ms-opt-128" style="padding-left: 23px;"><input value="jlpea" title="Journal of Low Power Electronics and Applications" id="ms-opt-128" type="checkbox">Journal of Low Power Electronics and Applications</label></li><li class=""><label for="ms-opt-129" style="padding-left: 23px;"><input value="jmmp" title="Journal of Manufacturing and Materials Processing" id="ms-opt-129" type="checkbox">Journal of Manufacturing and Materials Processing</label></li><li class=""><label for="ms-opt-130" style="padding-left: 23px;"><input value="jmse" title="Journal of Marine Science and Engineering" id="ms-opt-130" type="checkbox">Journal of Marine Science and Engineering</label></li><li class=""><label for="ms-opt-131" style="padding-left: 23px;"><input value="JOItmC" title="Journal of Open Innovation: Technology, Market, and Complexity" id="ms-opt-131" type="checkbox">Journal of Open Innovation: Technology, Market, and Complexity</label></li><li class=""><label for="ms-opt-132" style="padding-left: 23px;"><input value="jpm" title="Journal of Personalized Medicine" id="ms-opt-132" type="checkbox">Journal of Personalized Medicine</label></li><li class=""><label for="ms-opt-133" style="padding-left: 23px;"><input value="jrfm" title="Journal of Risk and Financial Management" id="ms-opt-133" type="checkbox">Journal of Risk and Financial Management</label></li><li class=""><label for="ms-opt-134" style="padding-left: 23px;"><input value="jsan" title="Journal of Sensor and Actuator Networks" id="ms-opt-134" type="checkbox">Journal of Sensor and Actuator Networks</label></li><li class=""><label for="ms-opt-135" style="padding-left: 23px;"><input value="land" title="Land" id="ms-opt-135" type="checkbox">Land</label></li><li class=""><label for="ms-opt-136" style="padding-left: 23px;"><input value="languages" title="Languages" id="ms-opt-136" type="checkbox">Languages</label></li><li class=""><label for="ms-opt-137" style="padding-left: 23px;"><input value="laws" title="Laws" id="ms-opt-137" type="checkbox">Laws</label></li><li class=""><label for="ms-opt-138" style="padding-left: 23px;"><input value="life" title="Life" id="ms-opt-138" type="checkbox">Life</label></li><li class=""><label for="ms-opt-139" style="padding-left: 23px;"><input value="logistics" title="Logistics" id="ms-opt-139" type="checkbox">Logistics</label></li><li class=""><label for="ms-opt-140" style="padding-left: 23px;"><input value="lubricants" title="Lubricants" id="ms-opt-140" type="checkbox">Lubricants</label></li><li class=""><label for="ms-opt-141" style="padding-left: 23px;"><input value="make" title="Machine Learning and Knowledge Extraction" id="ms-opt-141" type="checkbox">Machine Learning and Knowledge Extraction</label></li><li class=""><label for="ms-opt-142" style="padding-left: 23px;"><input value="machines" title="Machines" id="ms-opt-142" type="checkbox">Machines</label></li><li class=""><label for="ms-opt-143" style="padding-left: 23px;"><input value="magnetochemistry" title="Magnetochemistry" id="ms-opt-143" type="checkbox">Magnetochemistry</label></li><li class=""><label for="ms-opt-144" style="padding-left: 23px;"><input value="marinedrugs" title="Marine Drugs" id="ms-opt-144" type="checkbox">Marine Drugs</label></li><li class=""><label for="ms-opt-145" style="padding-left: 23px;"><input value="materials" title="Materials" id="ms-opt-145" type="checkbox">Materials</label></li><li class=""><label for="ms-opt-146" style="padding-left: 23px;"><input value="mca" title="Mathematical and Computational Applications" id="ms-opt-146" type="checkbox">Mathematical and Computational Applications</label></li><li class=""><label for="ms-opt-147" style="padding-left: 23px;"><input value="mathematics" title="Mathematics" id="ms-opt-147" type="checkbox">Mathematics</label></li><li class=""><label for="ms-opt-148" style="padding-left: 23px;"><input value="medsci" title="Medical Sciences" id="ms-opt-148" type="checkbox">Medical Sciences</label></li><li class=""><label for="ms-opt-149" style="padding-left: 23px;"><input value="medicina" title="Medicina" id="ms-opt-149" type="checkbox">Medicina</label></li><li class=""><label for="ms-opt-150" style="padding-left: 23px;"><input value="medicines" title="Medicines" id="ms-opt-150" type="checkbox">Medicines</label></li><li class=""><label for="ms-opt-151" style="padding-left: 23px;"><input value="membranes" title="Membranes" id="ms-opt-151" type="checkbox">Membranes</label></li><li class=""><label for="ms-opt-152" style="padding-left: 23px;"><input value="metabolites" title="Metabolites" id="ms-opt-152" type="checkbox">Metabolites</label></li><li class=""><label for="ms-opt-153" style="padding-left: 23px;"><input value="metals" title="Metals" id="ms-opt-153" type="checkbox">Metals</label></li><li class=""><label for="ms-opt-154" style="padding-left: 23px;"><input value="mps" title="Methods and Protocols" id="ms-opt-154" type="checkbox">Methods and Protocols</label></li><li class=""><label for="ms-opt-155" style="padding-left: 23px;"><input value="micromachines" title="Micromachines" id="ms-opt-155" type="checkbox">Micromachines</label></li><li class=""><label for="ms-opt-156" style="padding-left: 23px;"><input value="microorganisms" title="Microorganisms" id="ms-opt-156" type="checkbox">Microorganisms</label></li><li class=""><label for="ms-opt-157" style="padding-left: 23px;"><input value="minerals" title="Minerals" id="ms-opt-157" type="checkbox">Minerals</label></li><li class=""><label for="ms-opt-158" style="padding-left: 23px;"><input value="molbank" title="Molbank" id="ms-opt-158" type="checkbox">Molbank</label></li><li class=""><label for="ms-opt-159" style="padding-left: 23px;"><input value="molecules" title="Molecules" id="ms-opt-159" type="checkbox">Molecules</label></li><li class=""><label for="ms-opt-160" style="padding-left: 23px;"><input value="mti" title="Multimodal Technologies and Interaction" id="ms-opt-160" type="checkbox">Multimodal Technologies and Interaction</label></li><li class=""><label for="ms-opt-161" style="padding-left: 23px;"><input value="nanomaterials" title="Nanomaterials" id="ms-opt-161" type="checkbox">Nanomaterials</label></li><li class=""><label for="ms-opt-162" style="padding-left: 23px;"><input value="nitrogen" title="Nitrogen" id="ms-opt-162" type="checkbox">Nitrogen</label></li><li class=""><label for="ms-opt-163" style="padding-left: 23px;"><input value="ncrna" title="Non-Coding RNA" id="ms-opt-163" type="checkbox">Non-Coding RNA</label></li><li class=""><label for="ms-opt-164" style="padding-left: 23px;"><input value="nutrients" title="Nutrients" id="ms-opt-164" type="checkbox">Nutrients</label></li><li class=""><label for="ms-opt-165" style="padding-left: 23px;"><input value="oceans" title="Oceans" id="ms-opt-165" type="checkbox">Oceans</label></li><li class=""><label for="ms-opt-166" style="padding-left: 23px;"><input value="optics" title="Optics" id="ms-opt-166" type="checkbox">Optics</label></li><li class=""><label for="ms-opt-167" style="padding-left: 23px;"><input value="particles" title="Particles" id="ms-opt-167" type="checkbox">Particles</label></li><li class=""><label for="ms-opt-168" style="padding-left: 23px;"><input value="pathogens" title="Pathogens" id="ms-opt-168" type="checkbox">Pathogens</label></li><li class=""><label for="ms-opt-169" style="padding-left: 23px;"><input value="pharmaceuticals" title="Pharmaceuticals" id="ms-opt-169" type="checkbox">Pharmaceuticals</label></li><li class=""><label for="ms-opt-170" style="padding-left: 23px;"><input value="pharmaceutics" title="Pharmaceutics" id="ms-opt-170" type="checkbox">Pharmaceutics</label></li><li class=""><label for="ms-opt-171" style="padding-left: 23px;"><input value="pharmacy" title="Pharmacy" id="ms-opt-171" type="checkbox">Pharmacy</label></li><li class=""><label for="ms-opt-172" style="padding-left: 23px;"><input value="philosophies" title="Philosophies" id="ms-opt-172" type="checkbox">Philosophies</label></li><li class=""><label for="ms-opt-173" style="padding-left: 23px;"><input value="photonics" title="Photonics" id="ms-opt-173" type="checkbox">Photonics</label></li><li class=""><label for="ms-opt-174" style="padding-left: 23px;"><input value="physics" title="Physics" id="ms-opt-174" type="checkbox">Physics</label></li><li class=""><label for="ms-opt-175" style="padding-left: 23px;"><input value="plants" title="Plants" id="ms-opt-175" type="checkbox">Plants</label></li><li class=""><label for="ms-opt-176" style="padding-left: 23px;"><input value="plasma" title="Plasma" id="ms-opt-176" type="checkbox">Plasma</label></li><li class=""><label for="ms-opt-177" style="padding-left: 23px;"><input value="polymers" title="Polymers" id="ms-opt-177" type="checkbox">Polymers</label></li><li class=""><label for="ms-opt-178" style="padding-left: 23px;"><input value="proceedings" title="Proceedings" id="ms-opt-178" type="checkbox">Proceedings</label></li><li class=""><label for="ms-opt-179" style="padding-left: 23px;"><input value="processes" title="Processes" id="ms-opt-179" type="checkbox">Processes</label></li><li class=""><label for="ms-opt-180" style="padding-left: 23px;"><input value="prosthesis" title="Prosthesis" id="ms-opt-180" type="checkbox">Prosthesis</label></li><li class=""><label for="ms-opt-181" style="padding-left: 23px;"><input value="proteomes" title="Proteomes" id="ms-opt-181" type="checkbox">Proteomes</label></li><li class=""><label for="ms-opt-182" style="padding-left: 23px;"><input value="psych" title="Psych" id="ms-opt-182" type="checkbox">Psych</label></li><li class=""><label for="ms-opt-183" style="padding-left: 23px;"><input value="publications" title="Publications" id="ms-opt-183" type="checkbox">Publications</label></li><li class=""><label for="ms-opt-184" style="padding-left: 23px;"><input value="qubs" title="Quantum Beam Science" id="ms-opt-184" type="checkbox">Quantum Beam Science</label></li><li class=""><label for="ms-opt-185" style="padding-left: 23px;"><input value="quantumrep" title="Quantum Reports" id="ms-opt-185" type="checkbox">Quantum Reports</label></li><li class=""><label for="ms-opt-186" style="padding-left: 23px;"><input value="quaternary" title="Quaternary" id="ms-opt-186" type="checkbox">Quaternary</label></li><li class=""><label for="ms-opt-187" style="padding-left: 23px;"><input value="reactions" title="Reactions" id="ms-opt-187" type="checkbox">Reactions</label></li><li class=""><label for="ms-opt-188" style="padding-left: 23px;"><input value="recycling" title="Recycling" id="ms-opt-188" type="checkbox">Recycling</label></li><li class=""><label for="ms-opt-189" style="padding-left: 23px;"><input value="religions" title="Religions" id="ms-opt-189" type="checkbox">Religions</label></li><li class=""><label for="ms-opt-190" style="padding-left: 23px;"><input value="remotesensing" title="Remote Sensing" id="ms-opt-190" type="checkbox">Remote Sensing</label></li><li class=""><label for="ms-opt-191" style="padding-left: 23px;"><input value="reports" title="Reports â€” Medical Cases, Images, and Videos" id="ms-opt-191" type="checkbox">Reports â€” Medical Cases, Images, and Videos</label></li><li class=""><label for="ms-opt-192" style="padding-left: 23px;"><input value="reprodmed" title="Reproductive Medicine" id="ms-opt-192" type="checkbox">Reproductive Medicine</label></li><li class=""><label for="ms-opt-193" style="padding-left: 23px;"><input value="resources" title="Resources" id="ms-opt-193" type="checkbox">Resources</label></li><li class=""><label for="ms-opt-194" style="padding-left: 23px;"><input value="risks" title="Risks" id="ms-opt-194" type="checkbox">Risks</label></li><li class=""><label for="ms-opt-195" style="padding-left: 23px;"><input value="robotics" title="Robotics" id="ms-opt-195" type="checkbox">Robotics</label></li><li class=""><label for="ms-opt-196" style="padding-left: 23px;"><input value="safety" title="Safety" id="ms-opt-196" type="checkbox">Safety</label></li><li class=""><label for="ms-opt-197" style="padding-left: 23px;"><input value="sci" title="Sci" id="ms-opt-197" type="checkbox">Sci</label></li><li class=""><label for="ms-opt-198" style="padding-left: 23px;"><input value="scipharm" title="Scientia Pharmaceutica" id="ms-opt-198" type="checkbox">Scientia Pharmaceutica</label></li><li class=""><label for="ms-opt-199" style="padding-left: 23px;"><input value="sensors" title="Sensors" id="ms-opt-199" type="checkbox">Sensors</label></li><li class=""><label for="ms-opt-200" style="padding-left: 23px;"><input value="separations" title="Separations" id="ms-opt-200" type="checkbox">Separations</label></li><li class=""><label for="ms-opt-201" style="padding-left: 23px;"><input value="signals" title="Signals" id="ms-opt-201" type="checkbox">Signals</label></li><li class=""><label for="ms-opt-202" style="padding-left: 23px;"><input value="sinusitis" title="Sinusitis" id="ms-opt-202" type="checkbox">Sinusitis</label></li><li class=""><label for="ms-opt-203" style="padding-left: 23px;"><input value="smartcities" title="Smart Cities" id="ms-opt-203" type="checkbox">Smart Cities</label></li><li class=""><label for="ms-opt-204" style="padding-left: 23px;"><input value="socsci" title="Social Sciences" id="ms-opt-204" type="checkbox">Social Sciences</label></li><li class=""><label for="ms-opt-205" style="padding-left: 23px;"><input value="societies" title="Societies" id="ms-opt-205" type="checkbox">Societies</label></li><li class=""><label for="ms-opt-206" style="padding-left: 23px;"><input value="soilsystems" title="Soil Systems" id="ms-opt-206" type="checkbox">Soil Systems</label></li><li class=""><label for="ms-opt-207" style="padding-left: 23px;"><input value="sports" title="Sports" id="ms-opt-207" type="checkbox">Sports</label></li><li class=""><label for="ms-opt-208" style="padding-left: 23px;"><input value="stats" title="Stats" id="ms-opt-208" type="checkbox">Stats</label></li><li class=""><label for="ms-opt-209" style="padding-left: 23px;"><input value="surfaces" title="Surfaces" id="ms-opt-209" type="checkbox">Surfaces</label></li><li class=""><label for="ms-opt-210" style="padding-left: 23px;"><input value="surgeries" title="Surgeries" id="ms-opt-210" type="checkbox">Surgeries</label></li><li class=""><label for="ms-opt-211" style="padding-left: 23px;"><input value="sustainability" title="Sustainability" id="ms-opt-211" type="checkbox">Sustainability</label></li><li class=""><label for="ms-opt-212" style="padding-left: 23px;"><input value="symmetry" title="Symmetry" id="ms-opt-212" type="checkbox">Symmetry</label></li><li class=""><label for="ms-opt-213" style="padding-left: 23px;"><input value="systems" title="Systems" id="ms-opt-213" type="checkbox">Systems</label></li><li class=""><label for="ms-opt-214" style="padding-left: 23px;"><input value="technologies" title="Technologies" id="ms-opt-214" type="checkbox">Technologies</label></li><li class=""><label for="ms-opt-215" style="padding-left: 23px;"><input value="telecom" title="Telecom" id="ms-opt-215" type="checkbox">Telecom</label></li><li class=""><label for="ms-opt-216" style="padding-left: 23px;"><input value="toxics" title="Toxics" id="ms-opt-216" type="checkbox">Toxics</label></li><li class=""><label for="ms-opt-217" style="padding-left: 23px;"><input value="toxins" title="Toxins" id="ms-opt-217" type="checkbox">Toxins</label></li><li class=""><label for="ms-opt-218" style="padding-left: 23px;"><input value="tropicalmed" title="Tropical Medicine and Infectious Disease" id="ms-opt-218" type="checkbox">Tropical Medicine and Infectious Disease</label></li><li class=""><label for="ms-opt-219" style="padding-left: 23px;"><input value="universe" title="Universe" id="ms-opt-219" type="checkbox">Universe</label></li><li class=""><label for="ms-opt-220" style="padding-left: 23px;"><input value="urbansci" title="Urban Science" id="ms-opt-220" type="checkbox">Urban Science</label></li><li class=""><label for="ms-opt-221" style="padding-left: 23px;"><input value="vaccines" title="Vaccines" id="ms-opt-221" type="checkbox">Vaccines</label></li><li class=""><label for="ms-opt-222" style="padding-left: 23px;"><input value="vehicles" title="Vehicles" id="ms-opt-222" type="checkbox">Vehicles</label></li><li class=""><label for="ms-opt-223" style="padding-left: 23px;"><input value="vetsci" title="Veterinary Sciences" id="ms-opt-223" type="checkbox">Veterinary Sciences</label></li><li class=""><label for="ms-opt-224" style="padding-left: 23px;"><input value="vibration" title="Vibration" id="ms-opt-224" type="checkbox">Vibration</label></li><li class=""><label for="ms-opt-225" style="padding-left: 23px;"><input value="viruses" title="Viruses" id="ms-opt-225" type="checkbox">Viruses</label></li><li class=""><label for="ms-opt-226" style="padding-left: 23px;"><input value="vision" title="Vision" id="ms-opt-226" type="checkbox">Vision</label></li><li class=""><label for="ms-opt-227" style="padding-left: 23px;"><input value="water" title="Water" id="ms-opt-227" type="checkbox">Water</label></li><li class=""><label for="ms-opt-228" style="padding-left: 23px;"><input value="world" title="World" id="ms-opt-228" type="checkbox">World</label></li><li class=""><label for="ms-opt-229" style="padding-left: 23px;"><input value="wevj" title="World Electric Vehicle Journal" id="ms-opt-229" type="checkbox">World Electric Vehicle Journal</label></li></ul></div></div>
<input name="email" placeholder="Enter your email address..." required="required" type="email">
<button class="genericCaptcha button button--dark UA_FooterNewsletterSubscribeButton" type="submit">Subscribe</button>
</form>
</div>
</div>
</div>
<div id="footer-copyright">
<div class="row">
<div class="columns large-6 medium-6 small-12 text-left">
Â© 1996-2020 MDPI (Basel, Switzerland) unless otherwise stated
</div>
<div class="columns large-6 medium-6 small-12 small-text-left medium-text-right large-text-right">
<a data-dropdown="drop-view-disclaimer" aria-controls="drop-view-disclaimer" aria-expanded="false" data-options="align:top; is_hover:true; hover_timeout:2000;">
Disclaimer
</a>
<div id="drop-view-disclaimer" class="f-dropdown label__btn__dropdown label__btn__dropdown--wide text-left" data-dropdown-content="" aria-hidden="true" tabindex="-1">
The statements, opinions and data contained in the journals are solely
those of the individual authors and contributors and not of the publisher and the editor(s).
</div>
<a href="https://www.mdpi.com/about/terms-and-conditions">
Terms and Conditions
</a>
<a href="https://www.mdpi.com/about/privacy">
Privacy Policy
</a>
</div>
</div>
</div>
</div>
<div id="cookie-notification" class="js-allow-cookies" style="display: none;">
<div class="columns large-10 medium-10 small-12">
We use cookies on our website to ensure you get the best experience.<br class="show-for-medium-up">
Read more about our cookies <a href="https://www.mdpi.com/about/privacy">here</a>.
</div>
<div class="columns large-2 medium-2 small-12 small-only-text-left text-right">
<a class="button button--default" href="https://www.mdpi.com/accept_cookies">Accept</a>
</div>
</div>
<div id="cookie-notification" class="js-survey" style="display: none;">
<div class="columns large-10 medium-10 small-12">
<strong>We have just recently launched a new version of our website.</strong><br class="show-for-meidum-up">
<span style="font-weight: 400;">
Help us to further improve by taking part in this short 5 minute survey <span style="inline-block">
<span class="show-for-medium-up js-survey-link"><a href="https://www.surveymonkey.com/r/VWJZ6VW" target="_blank" rel="noopener noreferrer">here</a>.</span>
<span class="show-for-small-only js-survey-link"><a href="https://www.surveymonkey.com/r/VWMDSNQ" target="_blank" rel="noopener noreferrer">here</a>.</span>
</span>
</span>
</div>
<div class="columns large-2 medium-2 small-12 small-only-text-left text-right">
<a class="button button--default js-disable-survey-link" href="https://www.mdpi.com/disable_survey">Never show this again</a>
</div>
</div>
</div>
<div id="main-share-modal" class="reveal-modal reveal-modal-new reveal-modal-new--small" data-reveal="" aria-labelledby="modalTitle" aria-hidden="true" role="dialog">
<div class="row">
<div class="small-12 columns">
<h2 style="margin: 0;">Share Link</h2>
</div>
<div class="small-12 columns">
<div class="social-media-links UA_ShareModalLinks" style="text-align: left;">
<a href="mailto:?&amp;subject=From%20MDPI%3A%20%22Modelling%20Distributions%20of%20Rove%20Beetles%20in%20Mountainous%20Areas%20Using%20Remote%20Sensing%20Data&quot;&amp;body=https://www.mdpi.com/603188%3A%0A%0AModelling%20Distributions%20of%20Rove%20Beetles%20in%20Mountainous%20Areas%20Using%20Remote%20Sensing%20Data%0A%0AAbstract%3A%20Mountain%20ecosystems%20are%20biodiversity%20hotspots%20that%20are%20increasingly%20threatened%20by%20climate%20and%20land%20use%2Fland%20cover%20changes.%20Long-term%20biodiversity%20monitoring%20programs%20provide%20unique%20insights%20into%20resulting%20adverse%20impacts%20on%20plant%20and%20animal%20species%20distribution.%20Species%20distribution%20models%20%28SDMs%29%20in%20combination%20with%20satellite%20remote%20sensing%20%28SRS%29%20data%20offer%20the%20opportunity%20to%20analyze%20shifts%20of%20species%20distributions%20in%20response%20to%20these%20changes%20in%20a%20spatially%20explicit%20way.%20Here%2C%20we%20predicted%20the%20presence%20probability%20of%20three%20different%20rove%20beetles%20in%20a%20mountainous%20protected%20area%20%28Gran%20Paradiso%20National%20Park%2C%20GPNP%29%20using%20environmental%20variables%20derived%20from%20Landsat%20and%20Aster%20Global%20Digital%20Elevation%20Model%20data%20and%20an%20ensemble%20modelling%20approach%20based%20on%20five%20different%20model%20algorithms%20%28maximum%20entropy%2C%20random%20forest%2C%20generalized%20boosting%20models%2C%20generalized%20additive%20models%2C%20and%20generalized%20linear%20models%29.%20The%20objectives%20of%20the%20study%20were%20%281%29%20to%20evaluate%20the%20potential%20of%20SRS%20data%20for%20predicting%20the%20presence%20of%20species%20dependent%20on%20local-scale%20environmental%20parameters%20at%20two%20different%20time%20periods%2C%20%282%29%20to%20analyze%20shifts%20in%20species%20distributions%20between%20the%20years%2C%20and%20%283%29%20to%20identify%20the%20most%20important%20species-specific%20SRS[...]" title="Email">
<i class="fa fa-envelope-square" style="font-size: 30px;"></i>
</a>
<a href="https://twitter.com/intent/tweet?text=Modelling+Distributions+of+Rove+Beetles+in+Mountainous+Areas+Using+Remote+Sensing+Data++%40MDPIOpenAccess&amp;url=https%3A%2F%2Fwww.mdpi.com%2F603188" onclick="if (!window.__cfRLUnblockHandlers) return false; windowOpen(this.href,600,800); return false" title="Twitter" target="_blank">
<i class="fa fa-twitter-square" style="font-size: 30px;"></i>
</a>
<a href="http://www.linkedin.com/shareArticle?mini=true&amp;url=https%3A%2F%2Fwww.mdpi.com%2F603188&amp;title=Modelling%20Distributions%20of%20Rove%20Beetles%20in%20Mountainous%20Areas%20Using%20Remote%20Sensing%20Data%26source%3Dhttps%3A%2F%2Fwww.mdpi.com%26summary%3DMountain%20ecosystems%20are%20biodiversity%20hotspots%20that%20are%20increasingly%20threatened%20by%20climate%20and%20land%20use%2Fland%20cover%20changes.%20Long-term%20biodiversity%20monitoring%20programs%20provide%20unique%20insights%20into%20resulting%20adverse%20impacts%20on%20plant%20and%20animal%20species%20%5B...%5D" onclick="if (!window.__cfRLUnblockHandlers) return false; windowOpen(this.href,600,800); return false" title="LinkedIn" target="_blank">
<i class="fa fa-linkedin-square" style="font-size: 30px;"></i>
</a>
<a href="https://www.facebook.com/sharer.php?u=https://www.mdpi.com/603188" title="facebook" target="_blank">
<i class="fa fa-facebook-square" style="font-size: 30px;"></i>
</a>
<a href="https://www.reddit.com/submit?url=https://www.mdpi.com/603188" title="Reddit" target="_blank">
<i class="fa fa-reddit-square" style="font-size: 30px;"></i>
</a>
<a href="https://www.mendeley.com/import/?url=https://www.mdpi.com/603188" title="Mendeley" target="_blank">
<i class="fa fa-mendeley-square" style="font-size: 30px;"></i>
</a>
<a href="http://www.citeulike.org/posturl?url=https://www.mdpi.com/603188" title="CiteULike" target="_blank">
<i class="fa fa-citeulike-square" style="font-size: 30px;"></i>
</a>
</div>
</div>
<div class="small-9 columns">
<input id="js-clipboard-text" readonly="readonly" value="https://www.mdpi.com/603188" type="text">
</div>
<div class="small-3 columns text-left">
<a class="button button--color js-clipboard-copy" data-clipboard-target="#js-clipboard-text">Copy</a>
</div>
</div>
<a class="close-reveal-modal" aria-label="Close">
<i class="material-icons">clear</i>
</a>
</div>
<a href="#" class="back-to-top"><span class="show-for-medium-up">Back to Top</span><span class="show-for-small">Top</span></a>
<script src="modernizr-2.js" type="text/javascript"></script>
<script src="https://www.mdpi.com/assets/js/jquery-1.12.4.min.js?4f252523d4af0b47" type="text/javascript"></script>
<script src="https://www.mdpi.com/assets/js/foundation-5.5.3.min.js?6b2ec41c18b29054" type="text/javascript"></script>
<script src="https://www.mdpi.com/assets/js/foundation-5.5.3.equalizer.min.js?0f6c549b75ec554c" type="text/javascript"></script>
<script src="https://www.mdpi.com/assets/js/jquery.multiselect.js?0edd3998731d1091" type="text/javascript"></script>
<script src="https://www.mdpi.com/assets/js/jquery.cycle2.min.js?63413052928f97ee" type="text/javascript"></script>
<script type="text/javascript">
            $('select.foundation-select').multiselect({
                search: true,
                minHeight: 130,
                maxHeight: 130,
            });

            $(document).foundation({
                orbit: {
                    timer_speed: 4000,
                },
                reveal: {
                    animation: 'fadeAndPop',
                    animation_speed: 100,
                }
            });

            // old browser fix - this way the console log rows won't throw (silent) errors in browsers not supporting console log
            if (!window.console) window.console = {};
            if (!window.console.log) window.console.log = function () { };

            var currentJournalNameSystem = "remotesensing";

            $(document).ready(function()
            {
                $(".chosen-select").on('chosen:ready', function(event, data) {
                    var select = $(data.chosen.form_field);
                    if (select.attr('id') === 'journal-browser-volume') {
                        $(data.chosen.dropdown).addClass('UI_JournalBrowser_Volume_Options');
                    }
                    if (select.attr('id') === 'journal-browser-issue') {
                        $(data.chosen.dropdown).addClass('UI_JournalBrowser_Issue_Options');
                    }
                }).chosen({ disable_search_threshold: 7, width: "100%" });

                $(".toEncode").each(function(e) {
                    var oldHref = $(this).attr("href");
                    var newHref = oldHref.replace('.botdefense.please.enable.javaScript.','@');
                    $(this).attr("href", newHref);

                    if (!$(this).hasClass("emailCaptcha"))
                    {
                        $(this).html(newHref.replace('mailto:', ''));
                    }
                    
                                    });

                $(document).on('opened.fndtn.reveal', '[data-reveal]', function()
                {
                    $(document).foundation('equalizer', 'reflow');
                });

                // fix the images that have tag height / width defined
                // otherwise the default foundation styles overwrite the tag definitions
                $("img").each(function()
                {
                   if ($(this).attr('width') != undefined || $(this).attr('height') != undefined)
                   {
                       $(this).addClass("img-fixed");
                   }
                });

                $(".hide-show-desktop-option").click(function(e) 
                {
                    e.preventDefault();
                    var parentDiv = $(this).closest("div");

                    $.ajax({
                        url: $(this).attr('href'),
                        success: function(msg)
                        {
                            parentDiv.removeClass().hide();
                        }
                    });
                });

                /*
                * handle whole row as a link if the row contains only one visible link
                */
                $("table.new tr").hover(function()
                {
                   if ($(this).find("td:visible a").length == 1)
                   {
                       $(this).addClass("single-link");
                   }
                },
                function()
                {
                   $(this).removeClass("single-link");
                });

                $("table.new:not(.table-of-tables)").on("click", "tr.single-link", function(e)
                {
                    var target = $(e.target);
                    if (!e.ctrlKey && !target.is("a")) {
                        $(this).find("td:visible a")[0].click();
                    }
                });

                $(document).on("click", ".custom-accordion-for-small-screen-link", function(e) {
                    if ($(this).closest("#basic_search").length > 0) {
                        if ($(".search-container__advanced").first().is(":visible")) {
                            openAdvanced()
                        }
                    }

                    if (Foundation.utils.is_small_only()) {
                        if ($(this).hasClass("active")) {
                            $(this).removeClass("active");
                            $(this).next(".custom-accordion-for-small-screen-content").addClass("show-for-medium-up");
                        }
                        else {
                            $(this).addClass("active");
                            $(this).next(".custom-accordion-for-small-screen-content").removeClass("show-for-medium-up");
                            $(document).foundation('orbit', 'reflow');
                        }
                    }

                    if (undefined !== $(this).data("callback")) {
                        var customCallback = $(this).data("callback");
                        func = window[customCallback];
                        func();
                    }
                });

                $(document).on("click", ".js-open-small-search", function(e) {
                    e.preventDefault();  
                    $(this).toggleClass("active").closest(".tab-bar").toggleClass("active");
                    $(".search-container").toggleClass("hide-for-small-down");
                });

                $(document).on("click", ".js-survey-link", function(e) {
                    localStorage.setItem("mdpi_disable_survey", true);
                    window.location.href = "/disable_survey";
                });

                $(document).on("click", ".js-disable-survey-link", function(e) {
                    localStorage.setItem("mdpi_disable_survey", true);
                });

                $(document).on("click", ".js-open-menu", function(e) {
                    $(".search-container").addClass("hide-for-small-down");

                });

                $(window).on('resize', function() {
                    recalculate_main_browser_position();
                    recalculate_responsive_moving_containers();
                });

                checkCookiesAllowed();
                updateSearchLabelVisibilities();
                recalculate_main_browser_position();
                recalculate_responsive_moving_containers();
                
                
                if (window.document.documentMode == 11) {
                    $("<link/>", { rel: "stylesheet", type: "text/css", href: "https://fonts.googleapis.com/icon?family=Material+Icons"}).appendTo("head");
                }
            });

            function recalculate_main_browser_position() {
                if (Foundation.utils.is_small_only()) {
                    if ($("#js-main-top-container").parent("#js-large-main-top-container").length > 0) {
                        $("#js-main-top-container").appendTo($("#js-small-main-top-container"));
                    }
                }
                else {
                    if ($("#js-main-top-container").parent("#js-small-main-top-container").length > 0) {
                        $("#js-main-top-container").appendTo($("#js-large-main-top-container"));
                    }
                }
            }

            function recalculate_responsive_moving_containers() {
                $(".responsive-moving-container.large").each(function() {
                    var previousParent = $(".responsive-moving-container.active[data-id='"+$(this).data("id")+"']");
                    var movingContent  = previousParent.html();

                    if (Foundation.utils.is_small_only()) {
                        var currentParent = $(".responsive-moving-container.small[data-id='"+$(this).data("id")+"']");
                    }
                    else if (Foundation.utils.is_medium_only()) {
                        var currentParent = $(".responsive-moving-container.medium[data-id='"+$(this).data("id")+"']");
                    }
                    else {
                        var currentParent = $(".responsive-moving-container.large[data-id='"+$(this).data("id")+"']");
                    }

                    if (previousParent.attr("class") !== currentParent.attr("class")) {
                        currentParent.html(movingContent);
                        previousParent.html();
                        currentParent.addClass("active");
                        previousParent.removeClass("active");
                    }
                });
            }

            // cookies allowed is checked from a) local storage and b) from server separately so that the footer bar doesn't
            // get included in the custom page caches
            function checkCookiesAllowed() {
                var cookiesEnabled = localStorage.getItem("mdpi_cookies_enabled"); 

                if (null === cookiesEnabled) {
                    $.ajax({
                        url: "/ajax_cookie_value/mdpi_cookies_accepted",
                        success: function(data)
                        {
                            if (data.value) {
                                localStorage.setItem("mdpi_cookies_enabled", true);
                                checkDisplaySurvey();
                            }
                            else {
                                $(".js-allow-cookies").show();
                            }
                        }
                    });
                }
                else {
                    checkDisplaySurvey();
                }
            }

            // cookies allowed is checked from a) local storage and b) from server separately so that the footer bar doesn't
            // get included in the custom page caches
            //
            // NOTE! survey has been closed. Not used anymore (this code can be moved to somewhere for storage in case we need similar logic with other iterations)
            // 
            function checkDisplaySurvey() {
                return;

                var surveyDisabled = localStorage.getItem("mdpi_disable_survey"); 
                var viewCount      = localStorage.getItem("mdpi_new_layout_views"); 
                var viewLimit      = localStorage.getItem("mdpi_new_layout_limit");


                if (null !== surveyDisabled) {
                    return;
                }

                $('head').append('<script>(function(t,e,s,o){var n,a,c;t.SMCX=t.SMCX||[],e.getElementById(o)||(n=e.getElementsByTagName(s),a=n[n.length-1],c=e.createElement(s),c.type="text\/javascript",c.async=!0,c.id=o,c.src=["https:"===location.protocol?"https:\/\/":"http:\/\/","widget.surveymonkey.com\/collect\/website\/js\/tRaiETqnLgj758hTBazgd5A6lqMGTTGvz1r5Uxxx05XAcX_2B9_2Bt_2BCLTLkRxoplQbC.js"].join(""),a.parentNode.insertBefore(c,a))})(window,document,"script","smcx-sdk");<\/script>');

                return;
                // todo: the below part can be removed later on...

                if (null === viewLimit) {
                    localStorage.setItem("mdpi_new_layout_limit", Math.floor(Math.random() * 50));
                    localStorage.setItem("mdpi_new_layout_views", 0);
                }
                else {
                    viewCount++;
                    localStorage.setItem("mdpi_new_layout_views", viewCount);

                    if (viewCount >= viewLimit) {
                        $.ajax({
                            url: "/ajax_cookie_value/mdpi_disable_survey",
                            success: function(data)
                            {
                                if (data.value) {
                                    localStorage.setItem("mdpi_disable_survey", true);
                                }
                                else {
                                    $(".js-survey").show();
                                }
                            }
                        });
                    }
                }
            }
            
        </script>
<script src="https://www.mdpi.com/assets/js/lib.js?f0e39afa4d83e329" type="text/javascript"></script>
<script src="https://www.mdpi.com/assets/js/mdpi.js?c5e55b6509d6aab3" type="text/javascript"></script>
<script type="text/javascript">var banners_url = 'https://serve.mdpi.com';</script>
<script type="text/javascript" src="https://www.mdpi.com/assets/js/ifvisible.min.js?c621d19ecb761212"></script>
<script src="https://www.mdpi.com/assets/js/xmltohtml/affix.js?ac4ea55275297c15" type="text/javascript"></script>
<script src="https://www.mdpi.com/assets/js/clipboard.min.js?3f3688138a1b9fc4" type="text/javascript"></script>
<script type="text/javascript">
        $(document).ready(function()
        {
            var helpFunctions = $(".middle-column__help__fixed");
            var middleColumn  = $("#middle-column");

            helpFunctions.affix({
                offset: {
                    top: function() {
                        return middleColumn.offset().top - 8 - (Foundation.utils.is_medium_only() ? 30 : 0);
                    },
                    bottom: function() {
                        return $("#footer").innerHeight() + 56 + (Foundation.utils.is_medium_only() ? 0 : 0);
                    }
                }
            });

            new ClipboardJS('.js-clipboard-copy');
        });
    </script>
<script src="https://www.mdpi.com/assets/js/jquery-ui-1.12.0.min.js?4c2c52cc11e5a28d" type="text/javascript"></script>
<script type="text/javascript">
        $(document).ready(function()
        {
            
            $(".link-article-menu").click(function(e) {
                e.preventDefault();

                $(this).find('span').toggle();
                $(this).next("div").toggleClass("active");
            });

                    });
    </script>
<script src="https://www.mdpi.com/assets/js/xmltohtml/affix.js?ac4ea55275297c15" type="text/javascript"></script>
<script src="https://www.mdpi.com/assets/js/xmltohtml/storage.js?e9b262d3a3476d25" type="text/javascript"></script><!--<script src="https://www.mdpi.com/assets/js/xmltohtml/storage.js?e9b262d3a3476d25" type="text/javascript"></script>-->
<!--<script src="https://www.mdpi.com/assets/js/xmltohtml/jquery-scrollspy.js?09cbaec0dbb35a67" type="text/javascript"></script>-->
<!--<script src="https://www.mdpi.com/assets/js/xmltohtml/magnific-popup.js?4a09c18460afb26c" type="text/javascript"></script>-->
<!--<script src="https://www.mdpi.com/assets/js/xmltohtml/underscore.js?f893e294cde60c24" type="text/javascript"></script>-->
<!--<script type="text/javascript">
    $('document').ready(function(){
    //   if(1 == 0){
    //     var doi = $('.html-art-header').find('a');
    //     var text = doi.text();
    //     doi.remove();
    //     $('.html-art-header p').append(text);
    //     $('.html-art-header').html($('.html-art-header')[0].innerHTML.replace(/doi:\s+/,'doi:&nbsp;'))
    //   }

      $("#left-column").addClass("show-for-large-up");
      $("#middle-column").removeClass("medium-9").removeClass("left-bordered").addClass("medium-12");
    });
	$(document).on('DOMNodeInserted', function(e) {

        var element = $(e.target);

        if (element.hasClass('menu') && element.hasClass('html-nav') )
        {
            element.addClass("side-menu-ul");
                                            }
    });
  </script>-->
<!--<script src="https://www.mdpi.com/assets/js/jquery-ui-1.10.4.custom.min.js?9fb4d59ff745b497" type="text/javascript"></script>-->
<!--<script src="https://www.mdpi.com/assets/js/xmltohtml/articles.js?cf8ded7b859890d1" type="text/javascript"></script>-->
<!--<script type="text/javascript">
        repositionOpenSideBar = function() {
            $('#left-column').addClass("show-for-large-up show-for-medium-up").show();
            $('#middle-column').removeClass('large-12').removeClass('medium-12');
            $('#middle-column').addClass('large-9');
        }

        repositionCloseSideBar = function() {
            $('#left-column').removeClass("show-for-large-up show-for-medium-up").hide();
            $('#middle-column').removeClass('large-9');
            $('#middle-column').addClass('large-12').addClass('medium-12');
        }
    </script>-->
<!--<script type="text/javascript">
        $(document).ready(function() {

            $(document).on('keyup', function (e) {
                if (e.keyCode == 27) {

                    var hElem = $(this).find(".annotator-adder");

                    if (hElem.length){
                        hElem.css({'visibility':'hidden'});
                    } else {
                        document.querySelector("hypothesis-adder").shadowRoot.querySelector(".annotator-adder").style.visibility = "hidden";
                    }

                }
            });
        });
    </script>-->
<!--<script type="text/javascript">
            window.hypothesisConfig = function () {
                return {
                    sidebarAppUrl: 'https://commenting.mdpi.com/app.html',
                    showHighlights:   'whenSidebarOpen'  ,
                    openSidebar:  false  ,
                    assetRoot: 'https://commentingres.mdpi.com/hypothesis',
                    services: [{
                        apiUrl: 'https://commenting.mdpi.com/api/',
                        authority: 'sciforum',
                        grantToken: '',
                        doi: '10.3390/rs12010080'
                    }],
                };
            };
            MathJax . Hub . Startup . signal . Interest ( function ( message ) {
                if ( message == 'End' ){
                var hypoLink = document . getElementById ( "hypothesis_frame" )
                hypoLink . setAttribute ( "src" , "https://commenting.mdpi.com/embed.js" )
                }
});
        </script>-->
<!--<script async="" id="hypothesis_frame" type="text/javascript"></script>-->
<!--[if lt IE 9]>
                        <script src="https://www.mdpi.com/assets/js/ie8/ie8.js?6eef8fcbc831f5bd"></script>
            <script src="https://www.mdpi.com/assets/js/ie8/jquery.xdomainrequest.min.js?a945caca315782b0"></script>
        <![endif]-->

<!--<script type="text/javascript">
            !function(e,t,n,s,u,a){e.twq||(s=e.twq=function(){s.exe?s.exe.apply(s,arguments):s.queue.push(arguments);
            },s.version='1.1',s.queue=[],u=t.createElement(n),u.async=!0,u.src='//static.ads-twitter.com/uwt.js',
            a=t.getElementsByTagName(n)[0],a.parentNode.insertBefore(u,a))}(window,document,'script');
            // Insert Twitter Pixel ID and Standard Event data below
            twq('init','o2pip');
            twq('track','PageView');
        </script>-->




<script type="text/javascript" id="">!function(b,e,f,g,a,c,d){b.fbq||(a=b.fbq=function(){a.callMethod?a.callMethod.apply(a,arguments):a.queue.push(arguments)},b._fbq||(b._fbq=a),a.push=a,a.loaded=!0,a.version="2.0",a.queue=[],c=e.createElement(f),c.async=!0,c.src=g,d=e.getElementsByTagName(f)[0],d.parentNode.insertBefore(c,d))}(window,document,"script","https://connect.facebook.net/en_US/fbevents.js");fbq("init","1073306569680586");fbq("set","agent","tmgoogletagmanager","1073306569680586");fbq("track","PageView");</script>
<noscript><img height="1" width="1" style="display:none" src="https://www.facebook.com/tr?id=1073306569680586&amp;ev=PageView&amp;noscript=1"></noscript>

<script type="text/javascript" id="">(function(a,c,e,f,d,b){a.hj=a.hj||function(){(a.hj.q=a.hj.q||[]).push(arguments)};a._hjSettings={hjid:1635407,hjsv:6};d=c.getElementsByTagName("head")[0];b=c.createElement("script");b.async=1;b.src=e+a._hjSettings.hjid+f+a._hjSettings.hjsv;d.appendChild(b)})(window,document,"https://static.hotjar.com/c/hotjar-",".js?sv\x3d");</script></body></html>